{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": [],
      "authorship_tag": "ABX9TyNtOdL9xjz/qFvGcpkXxbVp",
      "include_colab_link": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    },
    "widgets": {
      "application/vnd.jupyter.widget-state+json": {
        "7460a2e51ddc49f3b44ba08c2f95e9df": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HBoxModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HBoxView",
            "box_style": "",
            "children": [
              "IPY_MODEL_302f2bce02fa4729a0001bb9cb47b68e",
              "IPY_MODEL_b4c2702de5524fb8a753776e8b1d3d9d"
            ],
            "layout": "IPY_MODEL_127d3d54a67641fead2e221e0076f5f3"
          }
        },
        "302f2bce02fa4729a0001bb9cb47b68e": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DropdownModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DropdownModel",
            "_options_labels": [
              "dcc_puccini",
              "dcc_vivaldi",
              "dcc_beethoven",
              "dcc_dvorak",
              "dcc_rossini",
              "dcc_schumann",
              "dcc_schubert",
              "dcc_brahms",
              "dcc_phe",
              "dcc_tara",
              "dcc_berlioz",
              "dcc_sibelius",
              "dcc_marc",
              "dcc_mozart",
              "dcc_bach",
              "dcc_verdi",
              "dcc_haydn",
              "dcc_chopin",
              "dcc_handel",
              "dcc_liszt",
              "dcc_wagner",
              "dcc_strauss",
              "dcc_ammons",
              "dcc_basie",
              "dcc_beard",
              "dcc_clark",
              "dcc_jamal",
              "dcc_pearson",
              "dcc_allison",
              "dcc_broadbent",
              "dcc_byard",
              "dcc_corea",
              "dcc_grusin",
              "dcc_korman",
              "dcc_hancock",
              "dcc_hines",
              "dcc_barron",
              "dcc_bley",
              "dcc_blake",
              "dcc_bromhead",
              "dcc_buckner",
              "dcc_denis",
              "dcc_drew",
              "dcc_green",
              "dcc_powell",
              "dcc_longo",
              "dcc_lightsey",
              "dcc_metagenome",
              "dcc_bonner",
              "dcc_cole",
              "dcc_ellington",
              "dcc_moran",
              "dcc_newborn",
              "dcc_benoit",
              "dcc_culbertson",
              "dcc_walton"
            ],
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "DropdownView",
            "description": "Datahub:",
            "description_tooltip": null,
            "disabled": false,
            "index": 4,
            "layout": "IPY_MODEL_6818bde579c74ba480f454791743bf1c",
            "style": "IPY_MODEL_cf779143e8114aafbca1062ca48ad09c"
          }
        },
        "b4c2702de5524fb8a753776e8b1d3d9d": {
          "model_module": "@jupyter-widgets/output",
          "model_name": "OutputModel",
          "model_module_version": "1.0.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/output",
            "_model_module_version": "1.0.0",
            "_model_name": "OutputModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/output",
            "_view_module_version": "1.0.0",
            "_view_name": "OutputView",
            "layout": "IPY_MODEL_779a03938c9c49d7b606761451802762",
            "msg_id": "",
            "outputs": [
              {
                "output_type": "stream",
                "name": "stdout",
                "text": [
                  "\n",
                  "DATAHUB: dcc_rossini\n",
                  "DESCRIPTION: dcc_rossini\n",
                  "PUBLIC DESCRIPTION: None\n",
                  "\n",
                  "current status: ACTIVE\n",
                  "reccommended status: DORMANT\n",
                  "\n",
                  "notes: Empty\n",
                  "\n",
                  "Datahub webin account: Webin-657\n",
                  "submission accounts: []\n",
                  "\n",
                  "            \n"
                ]
              }
            ]
          }
        },
        "127d3d54a67641fead2e221e0076f5f3": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "6818bde579c74ba480f454791743bf1c": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "cf779143e8114aafbca1062ca48ad09c": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "779a03938c9c49d7b606761451802762": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": "1px solid black",
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        }
      }
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/jas-mckin/datahubs-dormancy/blob/main/datahubs_dormancy_report.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "import ipywidgets as widgets\n",
        "from ipywidgets import interact\n",
        "from IPython.display import display, clear_output, Markdown\n",
        "\n",
        "from google.colab import data_table\n",
        "import pandas as pd\n",
        "import pickle\n",
        "\n",
        "class Datahub:\n",
        "    def __init__(self, dhub_name, desc, pub_desc, status, projects = [], submission_contact = [], latest_hold_date = '', run_count = 0 , latest_run = '', analysis_count = 0, latest_analysis = '', recc_status = '', notes = ''):\n",
        "        self.dhub_name = dhub_name\n",
        "        self.desc = desc\n",
        "        self.pub_desc = pub_desc\n",
        "        self.status = status\n",
        "        self.projects = projects\n",
        "        self.submission_contact = submission_contact\n",
        "        self.latest_hold_date = latest_hold_date\n",
        "        self.run_count = run_count\n",
        "        self.latest_run = latest_run\n",
        "        self.analysis_count = analysis_count\n",
        "        self.latest_analysis = latest_analysis\n",
        "        self.recc_status = recc_status\n",
        "        self.notes = notes\n",
        "    def __repr__(self):\n",
        "        description = f\"\"\"Object of class Datahub. Details...\n",
        "        meta_key:{self.dhub_name}, desc:{self.desc}, pub_desc:{self.pub_desc}, status:{self.status}, projects:{self.projects}\n",
        "        submission_contact:{self.submission_contact}, latest_hold_date:{self.latest_hold_date}, run_count:{self.run_count},\n",
        "        latest_run:{self.latest_run}, analysis_count:{self.analysis_count}, latest_analysis:{self.latest_analysis}, recc_status:{self.recc_status}\"\"\"\n",
        "        return description\n",
        "\n",
        "######################## read in datahubs information #################################\n",
        "with open('dhub_list.pkl', 'rb') as f:\n",
        "    dhub_list = pickle.load(f)\n",
        "\n",
        "# make list and dictionay of datahub info (for viewing in widget)\n",
        "dhub_name_list = []\n",
        "dhubs_list = []\n",
        "for dhub in dhub_list:\n",
        "  dhub_name_list.append(dhub.dhub_name)\n",
        "  dhub_dict = {}\n",
        "  dhub_dict['dname'] = dhub.dhub_name\n",
        "  dhub_dict['desc'] = dhub.desc\n",
        "  dhub_dict['pubdes'] = dhub.pub_desc\n",
        "  dhub_dict['status'] = dhub.status\n",
        "  dhub_dict['analysis_count'] = dhub.analysis_count\n",
        "  dhub_dict['run_count'] = dhub.run_count\n",
        "  dhub_dict['recc_status'] = dhub.recc_status\n",
        "  dhub_dict['notes'] = dhub.notes\n",
        "  dhub_dict['submission_contact'] = dhub.submission_contact\n",
        "  dhub_dict['parent_webin'] = dhub.parent_webin\n",
        "  dhubs_list.append(dhub_dict)\n",
        "\n",
        "\n",
        "#make a table with datahub statistics (number of projects, number of runs, )\n",
        "\n",
        "########################################## CREATING DROPDOWN SELECTIOR ###############################################\n",
        "# input is a list of the datahub names and a list of dictionaries with the datahub information\n",
        "# output is a plain text widget which changes based on datahub selected\n",
        "\n",
        "\n",
        "def dropdown_menu_widget():\n",
        "\n",
        "  # create the output widget and dropdown widget\n",
        "  output = widgets.Output(layout={'border': '1px solid black'})\n",
        "  w = widgets.Dropdown(options=dhub_name_list, value=None, description='Datahub:')\n",
        "\n",
        "  def dropdown_eventhandler(change):\n",
        "\n",
        "    ## get data for selected datahub\n",
        "    selected_dhub = change.new\n",
        "    for dhub in dhub_name_list:\n",
        "      if selected_dhub == dhub:\n",
        "        for info in dhubs_list:\n",
        "          if dhub == info['dname']:\n",
        "            info0 = info['dname']\n",
        "            info1 = info['desc']\n",
        "            info2 = info['pubdes']\n",
        "            info3 = info['status']\n",
        "            info4 = info['recc_status']\n",
        "            info7 = info['notes']\n",
        "            info8 = info['submission_contact']\n",
        "            info9 = info['parent_webin']\n",
        "            text_out = f'''\n",
        "DATAHUB: {info0}\n",
        "DESCRIPTION: {info1}\n",
        "PUBLIC DESCRIPTION: {info2}\n",
        "\n",
        "current status: {info3}\n",
        "reccommended status: {info4}\n",
        "\n",
        "notes: {info7}\n",
        "\n",
        "Datahub webin account: {info9}\n",
        "submission accounts: {info8}\n",
        "\n",
        "            '''\n",
        "\n",
        "    ## add to the output widget using context manager, try-except needed for the dropdown event handler - filler code block required under 'except' - this has no functionality\n",
        "    #out_text = Markdown(data=\"\"\"test text with **gobbledegook** inside\"\"\", url=None, filename=None, metadata=None)\n",
        "    #text_widg = widgets.HTML(value=f\"\"\"Hello <b>World</b> i am adding some info here with the info selected datahub name is {selected_dhub},\n",
        "    #  matching ls_dict {info1} {info2} and {info3} and status is <b>{info4}</b> \"\"\",placeholder='Some HTML',description='Some HTML')\n",
        "    try:\n",
        "      with output:\n",
        "        print(text_out)\n",
        "        # can add just the one display(..) item with widgets inside\n",
        "        #display(widgets.IntSlider())\n",
        "        #display(text_widg)\n",
        "        #display(out_text)\n",
        "    except KeyError:\n",
        "      print('hi I do not print anywhere')\n",
        "      ###if selected_dhub == None:\n",
        "\n",
        "    ## clears the output. with wait=True, it will wait for new input before clearing the output\n",
        "    output.clear_output(wait=True)\n",
        "\n",
        "  # use observe function to call dropdown_eventhandler, to get the output\n",
        "  w.observe(dropdown_eventhandler, names = 'value')\n",
        "\n",
        "  # display the widgets - test out formatting\n",
        "  #display(w)\n",
        "  #display(output)\n",
        "\n",
        "  # convert output widgets into tuple for displaying in HBox\n",
        "\n",
        "  stuff = (w, output)\n",
        "\n",
        "  display(widgets.HBox(stuff))\n"
      ],
      "metadata": {
        "id": "5sZdtElMlpAx"
      },
      "execution_count": 13,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "############################################## VIEW REPORT #############################################\n",
        "# display individual datahub information\n",
        "dropdown_menu_widget()\n",
        "# display information about datahubs projects using google colab data table formatter (e.g. can filter by datahub)\n",
        "project_df = pd.read_csv(\"projects.csv\")\n",
        "data_table.enable_dataframe_formatter()\n",
        "project_df"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1347,
          "referenced_widgets": [
            "7460a2e51ddc49f3b44ba08c2f95e9df",
            "302f2bce02fa4729a0001bb9cb47b68e",
            "b4c2702de5524fb8a753776e8b1d3d9d",
            "127d3d54a67641fead2e221e0076f5f3",
            "6818bde579c74ba480f454791743bf1c",
            "cf779143e8114aafbca1062ca48ad09c",
            "779a03938c9c49d7b606761451802762"
          ]
        },
        "id": "L95cHomVikI0",
        "outputId": "7b8f5f88-8fce-4021-9991-48e397f8a8b3"
      },
      "execution_count": 14,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "HBox(children=(Dropdown(description='Datahub:', options=('dcc_puccini', 'dcc_vivaldi', 'dcc_beethoven', 'dcc_d…"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "7460a2e51ddc49f3b44ba08c2f95e9df"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "         datahub       prj_id  prj_webin hold_date  status_id  run_count  \\\n",
              "0     dcc_grusin  PRJNA622290  Webin-842       NaN          4          0   \n",
              "1     dcc_grusin  PRJNA622520  Webin-842       NaN          4          3   \n",
              "2     dcc_grusin  PRJNA622652  Webin-842       NaN          4          6   \n",
              "3     dcc_grusin  PRJNA622691  Webin-842       NaN          4          2   \n",
              "4     dcc_grusin  PRJNA622817  Webin-842       NaN          4         23   \n",
              "...          ...          ...        ...       ...        ...        ...   \n",
              "1062  dcc_grusin  PRJNA614995  Webin-842       NaN          4     142844   \n",
              "1063  dcc_grusin  PRJNA615032  Webin-842       NaN          4        329   \n",
              "1064  dcc_grusin  PRJNA615319  Webin-842       NaN          4          6   \n",
              "1065  dcc_grusin  PRJNA616147  Webin-842       NaN          4          7   \n",
              "1066  dcc_grusin  PRJNA616446  Webin-842       NaN          4         10   \n",
              "\n",
              "               latest_run  analysis_count latest_analysis  \\\n",
              "0                     NaN               0             NaN   \n",
              "1     2022-01-17 01:47:52               0             NaN   \n",
              "2     2020-04-14 19:59:17               0             NaN   \n",
              "3     2020-08-13 01:39:02               0             NaN   \n",
              "4     2020-04-14 19:58:19               0             NaN   \n",
              "...                   ...             ...             ...   \n",
              "1062  2023-10-24 06:26:33               0             NaN   \n",
              "1063  2020-04-24 01:39:41               0             NaN   \n",
              "1064  2020-04-14 19:54:11               0             NaN   \n",
              "1065  2020-04-15 05:26:29               0             NaN   \n",
              "1066  2020-04-14 19:56:50               0             NaN   \n",
              "\n",
              "                                              prj_title  \n",
              "0     Physiologic RNA Targets and Refined Sequence S...  \n",
              "1     A Platform to Define Cellular Targets of SARS-...  \n",
              "2     RNA-seq-based whole transcriptome analysis of ...  \n",
              "3     Identification of A Drug Blocking SARS-CoV-2 I...  \n",
              "4     Severe acute respiratory syndrome coronavirus ...  \n",
              "...                                                 ...  \n",
              "1062                                     SARS-CoV-2 WGS  \n",
              "1063   Transcriptional response to SARS-CoV-2 infection  \n",
              "1064  Deep sequencing of recombinant SARS-CoV-2 (rSA...  \n",
              "1065  Severe acute respiratory syndrome coronavirus ...  \n",
              "1066                    Homo sapiens Raw sequence reads  \n",
              "\n",
              "[1067 rows x 10 columns]"
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-4f24b68e-3d3f-41d6-8550-cfc5005a3352\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>datahub</th>\n",
              "      <th>prj_id</th>\n",
              "      <th>prj_webin</th>\n",
              "      <th>hold_date</th>\n",
              "      <th>status_id</th>\n",
              "      <th>run_count</th>\n",
              "      <th>latest_run</th>\n",
              "      <th>analysis_count</th>\n",
              "      <th>latest_analysis</th>\n",
              "      <th>prj_title</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>dcc_grusin</td>\n",
              "      <td>PRJNA622290</td>\n",
              "      <td>Webin-842</td>\n",
              "      <td>NaN</td>\n",
              "      <td>4</td>\n",
              "      <td>0</td>\n",
              "      <td>NaN</td>\n",
              "      <td>0</td>\n",
              "      <td>NaN</td>\n",
              "      <td>Physiologic RNA Targets and Refined Sequence S...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>dcc_grusin</td>\n",
              "      <td>PRJNA622520</td>\n",
              "      <td>Webin-842</td>\n",
              "      <td>NaN</td>\n",
              "      <td>4</td>\n",
              "      <td>3</td>\n",
              "      <td>2022-01-17 01:47:52</td>\n",
              "      <td>0</td>\n",
              "      <td>NaN</td>\n",
              "      <td>A Platform to Define Cellular Targets of SARS-...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>dcc_grusin</td>\n",
              "      <td>PRJNA622652</td>\n",
              "      <td>Webin-842</td>\n",
              "      <td>NaN</td>\n",
              "      <td>4</td>\n",
              "      <td>6</td>\n",
              "      <td>2020-04-14 19:59:17</td>\n",
              "      <td>0</td>\n",
              "      <td>NaN</td>\n",
              "      <td>RNA-seq-based whole transcriptome analysis of ...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>dcc_grusin</td>\n",
              "      <td>PRJNA622691</td>\n",
              "      <td>Webin-842</td>\n",
              "      <td>NaN</td>\n",
              "      <td>4</td>\n",
              "      <td>2</td>\n",
              "      <td>2020-08-13 01:39:02</td>\n",
              "      <td>0</td>\n",
              "      <td>NaN</td>\n",
              "      <td>Identification of A Drug Blocking SARS-CoV-2 I...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>dcc_grusin</td>\n",
              "      <td>PRJNA622817</td>\n",
              "      <td>Webin-842</td>\n",
              "      <td>NaN</td>\n",
              "      <td>4</td>\n",
              "      <td>23</td>\n",
              "      <td>2020-04-14 19:58:19</td>\n",
              "      <td>0</td>\n",
              "      <td>NaN</td>\n",
              "      <td>Severe acute respiratory syndrome coronavirus ...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>...</th>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1062</th>\n",
              "      <td>dcc_grusin</td>\n",
              "      <td>PRJNA614995</td>\n",
              "      <td>Webin-842</td>\n",
              "      <td>NaN</td>\n",
              "      <td>4</td>\n",
              "      <td>142844</td>\n",
              "      <td>2023-10-24 06:26:33</td>\n",
              "      <td>0</td>\n",
              "      <td>NaN</td>\n",
              "      <td>SARS-CoV-2 WGS</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1063</th>\n",
              "      <td>dcc_grusin</td>\n",
              "      <td>PRJNA615032</td>\n",
              "      <td>Webin-842</td>\n",
              "      <td>NaN</td>\n",
              "      <td>4</td>\n",
              "      <td>329</td>\n",
              "      <td>2020-04-24 01:39:41</td>\n",
              "      <td>0</td>\n",
              "      <td>NaN</td>\n",
              "      <td>Transcriptional response to SARS-CoV-2 infection</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1064</th>\n",
              "      <td>dcc_grusin</td>\n",
              "      <td>PRJNA615319</td>\n",
              "      <td>Webin-842</td>\n",
              "      <td>NaN</td>\n",
              "      <td>4</td>\n",
              "      <td>6</td>\n",
              "      <td>2020-04-14 19:54:11</td>\n",
              "      <td>0</td>\n",
              "      <td>NaN</td>\n",
              "      <td>Deep sequencing of recombinant SARS-CoV-2 (rSA...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1065</th>\n",
              "      <td>dcc_grusin</td>\n",
              "      <td>PRJNA616147</td>\n",
              "      <td>Webin-842</td>\n",
              "      <td>NaN</td>\n",
              "      <td>4</td>\n",
              "      <td>7</td>\n",
              "      <td>2020-04-15 05:26:29</td>\n",
              "      <td>0</td>\n",
              "      <td>NaN</td>\n",
              "      <td>Severe acute respiratory syndrome coronavirus ...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1066</th>\n",
              "      <td>dcc_grusin</td>\n",
              "      <td>PRJNA616446</td>\n",
              "      <td>Webin-842</td>\n",
              "      <td>NaN</td>\n",
              "      <td>4</td>\n",
              "      <td>10</td>\n",
              "      <td>2020-04-14 19:56:50</td>\n",
              "      <td>0</td>\n",
              "      <td>NaN</td>\n",
              "      <td>Homo sapiens Raw sequence reads</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>1067 rows × 10 columns</p>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-4f24b68e-3d3f-41d6-8550-cfc5005a3352')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-4f24b68e-3d3f-41d6-8550-cfc5005a3352 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-4f24b68e-3d3f-41d6-8550-cfc5005a3352');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-329953b3-6504-435d-9d7b-b8a06473f314\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-329953b3-6504-435d-9d7b-b8a06473f314')\"\n",
              "            title=\"Suggest charts.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-329953b3-6504-435d-9d7b-b8a06473f314 button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.module+javascript": "\n      import \"https://ssl.gstatic.com/colaboratory/data_table/881c4a0d49046431/data_table.js\";\n\n      const table = window.createDataTable({\n        data: [[{\n            'v': 0,\n            'f': \"0\",\n        },\n\"dcc_grusin\",\n\"PRJNA622290\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Physiologic RNA Targets and Refined Sequence Specificity of Coronavirus EndoU\"],\n [{\n            'v': 1,\n            'f': \"1\",\n        },\n\"dcc_grusin\",\n\"PRJNA622520\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2022-01-17 01:47:52\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"A Platform to Define Cellular Targets of SARS-COV-2 Infection Using Human Pluripotent Stem Cell-Derived Cells and Organoids\"],\n [{\n            'v': 2,\n            'f': \"2\",\n        },\n\"dcc_grusin\",\n\"PRJNA622652\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 6,\n            'f': \"6\",\n        },\n\"2020-04-14 19:59:17\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNA-seq-based whole transcriptome analysis of IPEC-J2 cells during swine acute diarrhea syndrome coronavirus infection\"],\n [{\n            'v': 3,\n            'f': \"3\",\n        },\n\"dcc_grusin\",\n\"PRJNA622691\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2020-08-13 01:39:02\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Identification of A Drug Blocking SARS-CoV-2 Infection using Human Pluripotent Stem Cell-derived Colon Organoids\"],\n [{\n            'v': 4,\n            'f': \"4\",\n        },\n\"dcc_grusin\",\n\"PRJNA622817\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 23,\n            'f': \"23\",\n        },\n\"2020-04-14 19:58:19\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Raw sequence reads\"],\n [{\n            'v': 5,\n            'f': \"5\",\n        },\n\"dcc_grusin\",\n\"PRJNA622837\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 22215,\n            'f': \"22215\",\n        },\n\"2022-06-23 01:30:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 Patient Sequencing from the Broad Institute\"],\n [{\n            'v': 6,\n            'f': \"6\",\n        },\n\"dcc_grusin\",\n\"PRJNA623001\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2020-12-07 01:41:31\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARSCoV2_RNAseq\"],\n [{\n            'v': 7,\n            'f': \"7\",\n        },\n\"dcc_grusin\",\n\"PRJNA623285\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2021-01-17 03:29:56\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 direct RNA Nanopore sequencing\"],\n [{\n            'v': 8,\n            'f': \"8\",\n        },\n\"dcc_grusin\",\n\"PRJNA623683\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2021-07-12 01:25:26\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-COV2 Peruvian sequencing project\"],\n [{\n            'v': 9,\n            'f': \"9\",\n        },\n\"dcc_grusin\",\n\"PRJNA623797\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2020-08-21 01:52:02\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Canadian SARS-CoV-2 cultured isolates\"],\n [{\n            'v': 10,\n            'f': \"10\",\n        },\n\"dcc_grusin\",\n\"PRJNA623895\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-04-14 20:02:16\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia\"],\n [{\n            'v': 11,\n            'f': \"11\",\n        },\n\"dcc_grusin\",\n\"PRJNA624068\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2020-07-31 01:39:13\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Non-neuronal expression of SARS-CoV-2 entry genes in the olfactory system suggests mechanisms underlying COVID-19-associated anosmia (scRNA-seq: olfactory bulb)\"],\n [{\n            'v': 12,\n            'f': \"12\",\n        },\n\"dcc_grusin\",\n\"PRJNA624223\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 73,\n            'f': \"73\",\n        },\n\"2020-07-28 01:47:30\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Gut microbiome alterations and longitudinal kinetics in COVID-19 patients\"],\n [{\n            'v': 13,\n            'f': \"13\",\n        },\n\"dcc_grusin\",\n\"PRJNA624231\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-05-07 11:13:52\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Metatranscriptomic sequencing from the COVID-19 patient in Northern Germany\"],\n [{\n            'v': 14,\n            'f': \"14\",\n        },\n\"dcc_grusin\",\n\"PRJNA624358\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2020-04-14 20:21:06\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WGS of SARS-CoV-2 genomes from South Africa\"],\n [{\n            'v': 15,\n            'f': \"15\",\n        },\n\"dcc_grusin\",\n\"PRJNA624792\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2020-04-16 01:39:08\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 propagates in human lung epithelial cell line but not in immune cell populations\"],\n [{\n            'v': 16,\n            'f': \"16\",\n        },\n\"dcc_grusin\",\n\"PRJNA624801\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2020-06-10 01:40:37\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Murine B cell response to immunization with SARS-CoV-2\"],\n [{\n            'v': 17,\n            'f': \"17\",\n        },\n\"dcc_grusin\",\n\"PRJNA625551\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 28392,\n            'f': \"28392\",\n        },\n\"2023-10-08 13:12:07\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2_sequencing\"],\n [{\n            'v': 18,\n            'f': \"18\",\n        },\n\"dcc_grusin\",\n\"PRJNA625669\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12621,\n            'f': \"12621\",\n        },\n\"2023-09-30 06:50:42\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Novel Coronavirus genome sequence and its characterization from the patients in Gujarat during nCov-19 outbreak\"],\n [{\n            'v': 19,\n            'f': \"19\",\n        },\n\"dcc_grusin\",\n\"PRJNA626526\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-04-21 01:51:37\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-Cov-2 virus strain ISR_JP0320 genome sequencing\"],\n [{\n            'v': 20,\n            'f': \"20\",\n        },\n\"dcc_grusin\",\n\"PRJNA627100\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2020-04-30 01:50:51\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"The time-series integrative proteo-transcriptomics analysis to footprint the host cellular-response in SARS-CoV-2 infection in Huh7 cells\"],\n [{\n            'v': 21,\n            'f': \"21\",\n        },\n\"dcc_grusin\",\n\"PRJNA627229\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 69,\n            'f': \"69\",\n        },\n\"2020-04-24 01:40:26\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Genome sequencing\"],\n [{\n            'v': 22,\n            'f': \"22\",\n        },\n\"dcc_grusin\",\n\"PRJNA627286\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2020-05-07 11:14:00\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Whole genome sequencing of SARS-cov-2 samples from different patients\"],\n [{\n            'v': 23,\n            'f': \"23\",\n        },\n\"dcc_grusin\",\n\"PRJNA627354\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 7,\n            'f': \"7\",\n        },\n\"2020-04-25 01:31:46\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Complete genome sequence of SARS-CoV-2 from a U.S. zoological collection\"],\n [{\n            'v': 24,\n            'f': \"24\",\n        },\n\"dcc_grusin\",\n\"PRJNA627454\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 132,\n            'f': \"132\",\n        },\n\"2021-11-07 01:54:25\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 receptor ACE2 is an interferon-stimulated gene in human airway epithelial cells and is detected in specific cell subsets across tissues\"],\n [{\n            'v': 25,\n            'f': \"25\",\n        },\n\"dcc_grusin\",\n\"PRJNA627662\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 112,\n            'f': \"112\",\n        },\n\"2020-05-14 01:31:34\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Sequences of SARS-CoV-2\"],\n [{\n            'v': 26,\n            'f': \"26\",\n        },\n\"dcc_grusin\",\n\"PRJNA627860\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 90,\n            'f': \"90\",\n        },\n\"2020-04-30 01:51:08\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RV infections in asthmatics increase ACE2 expression and stimulate cytokine pathways implicated in COVID19\"],\n [{\n            'v': 27,\n            'f': \"27\",\n        },\n\"dcc_grusin\",\n\"PRJNA627977\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2020-04-25 01:32:17\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 Deep Sequencing Comparing Two Challenge Stocks\"],\n [{\n            'v': 28,\n            'f': \"28\",\n        },\n\"dcc_grusin\",\n\"PRJNA628043\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2020-05-29 01:29:57\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-coronavirus-2 replication in Vero E6 cells: replication kinetics, rapid adaptation and cytopathology\"],\n [{\n            'v': 29,\n            'f': \"29\",\n        },\n\"dcc_grusin\",\n\"PRJNA628125\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 54,\n            'f': \"54\",\n        },\n\"2020-09-05 01:38:52\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Human B cell clonal and convergent responses to acute SARS-CoV-2 infection\"],\n [{\n            'v': 30,\n            'f': \"30\",\n        },\n\"dcc_grusin\",\n\"PRJNA628628\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 22,\n            'f': \"22\",\n        },\n\"2020-05-07 11:14:08\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Bulk RNA sequencing of SARS-CoV and SARS-CoV-2 infected human intestinal organoids\"],\n [{\n            'v': 31,\n            'f': \"31\",\n        },\n\"dcc_grusin\",\n\"PRJNA628662\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 9946,\n            'f': \"9946\",\n        },\n\"2023-10-22 11:29:05\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 WGS\"],\n [{\n            'v': 32,\n            'f': \"32\",\n        },\n\"dcc_grusin\",\n\"PRJNA629752\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 20,\n            'f': \"20\",\n        },\n\"2020-07-07 01:37:50\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Immunophenotyping of COVID-19 and Influenza Underscores the Association of Type I IFN Response in Severe COVID-19\"],\n [{\n            'v': 33,\n            'f': \"33\",\n        },\n\"dcc_grusin\",\n\"PRJNA629889\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 43,\n            'f': \"43\",\n        },\n\"2020-05-29 01:30:11\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genome sequencing of Severe acute respiratory syndrome coronavirus 2 in California\"],\n [{\n            'v': 34,\n            'f': \"34\",\n        },\n\"dcc_grusin\",\n\"PRJNA629891\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2020-05-03 01:33:36\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) in Egypt\"],\n [{\n            'v': 35,\n            'f': \"35\",\n        },\n\"dcc_grusin\",\n\"PRJNA630455\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 88,\n            'f': \"88\",\n        },\n\"2020-07-17 01:43:23\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Comprehensive mapping of immune perturbations associated with severe COVID-19\"],\n [{\n            'v': 36,\n            'f': \"36\",\n        },\n\"dcc_faang\",\n\"PRJNA630504\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 181,\n            'f': \"181\",\n        },\n\"2020-10-01 14:29:29\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Mapping of transcription initiation in the bovine genome by RAMPAGE\"],\n [{\n            'v': 37,\n            'f': \"37\",\n        },\n\"dcc_grusin\",\n\"PRJNA630679\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2020-07-21 01:49:46\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Naturally occurring SARS-CoV-2 gene deletions close to the spike S1/S2 cleavage site in the viral quasispecies of COVID19 patients\"],\n [{\n            'v': 38,\n            'f': \"38\",\n        },\n\"dcc_grusin\",\n\"PRJNA630716\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 27,\n            'f': \"27\",\n        },\n\"2020-09-10 01:38:38\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Decoding SARS-CoV-2 coding capacity\"],\n [{\n            'v': 39,\n            'f': \"39\",\n        },\n\"dcc_grusin\",\n\"PRJNA631042\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 49,\n            'f': \"49\",\n        },\n\"2020-05-13 01:39:24\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 Raw sequence reads\"],\n [{\n            'v': 40,\n            'f': \"40\",\n        },\n\"dcc_grusin\",\n\"PRJNA631061\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4466,\n            'f': \"4466\",\n        },\n\"2021-02-13 13:33:08\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Genome sequencing\"],\n [{\n            'v': 41,\n            'f': \"41\",\n        },\n\"dcc_grusin\",\n\"PRJNA631753\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 88,\n            'f': \"88\",\n        },\n\"2020-08-14 01:45:20\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Spectrum of Viral Load and Host Response Seen in Autopsies of SARS-CoV-2 Infected Lungs\"],\n [{\n            'v': 42,\n            'f': \"42\",\n        },\n\"dcc_grusin\",\n\"PRJNA631969\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 6,\n            'f': \"6\",\n        },\n\"2020-06-21 01:35:19\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNA-Seq of Human iPSC-cardiomyocytes infected with SARS-CoV-2\"],\n [{\n            'v': 43,\n            'f': \"43\",\n        },\n\"dcc_grusin\",\n\"PRJNA632475\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 56,\n            'f': \"56\",\n        },\n\"2021-04-26 08:45:13\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Clinical benefit of remdesivir treatment in rhesus macaques infected with SARS-CoV-2\"],\n [{\n            'v': 44,\n            'f': \"44\",\n        },\n\"dcc_grusin\",\n\"PRJNA632678\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-05-15 01:36:56\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 isolate:Cali-01 | cultivar:Human Genome sequencing\"],\n [{\n            'v': 45,\n            'f': \"45\",\n        },\n\"dcc_grusin\",\n\"PRJNA633048\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-05-16 01:35:07\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 RNA raw sequence reads\"],\n [{\n            'v': 46,\n            'f': \"46\",\n        },\n\"dcc_grusin\",\n\"PRJNA633241\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-05-17 01:36:17\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genome sequences of SARS-CoV-2 from Bangladesh\"],\n [{\n            'v': 47,\n            'f': \"47\",\n        },\n\"dcc_grusin\",\n\"PRJNA633317\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 153,\n            'f': \"153\",\n        },\n\"2020-11-30 01:35:57\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Longitudinal high-throughput TCR repertoire profiling reveals the dynamics of T cell memory formation after mild COVID-19 infection\"],\n [{\n            'v': 48,\n            'f': \"48\",\n        },\n\"dcc_grusin\",\n\"PRJNA633393\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 13,\n            'f': \"13\",\n        },\n\"2020-05-19 01:35:30\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"A single-cell atlas of the peripheral immune response to severe COVID-19\"],\n [{\n            'v': 49,\n            'f': \"49\",\n        },\n\"dcc_grusin\",\n\"PRJNA633783\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2020-05-28 01:27:43\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Generation of human bronchial organoids for SARS-CoV-2 research\"],\n [{\n            'v': 50,\n            'f': \"50\",\n        },\n\"dcc_grusin\",\n\"PRJNA633852\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 6,\n            'f': \"6\",\n        },\n\"2020-06-13 01:30:22\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Generation of a Broadly Useful Model for COVID-19 Pathogenesis, Vaccination, and Treatment\"],\n [{\n            'v': 51,\n            'f': \"51\",\n        },\n\"dcc_grusin\",\n\"PRJNA633853\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 269,\n            'f': \"269\",\n        },\n\"2020-07-29 01:49:47\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"A gene signature differentiates SARS-CoV-2 from other respiratory viruses\"],\n [{\n            'v': 52,\n            'f': \"52\",\n        },\n\"dcc_grusin\",\n\"PRJNA633948\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 424,\n            'f': \"424\",\n        },\n\"2022-04-05 03:19:01\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Whole genome sequencing of SARS-CoV-2 clinical isolates\"],\n [{\n            'v': 53,\n            'f': \"53\",\n        },\n\"dcc_grusin\",\n\"PRJNA634119\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 49,\n            'f': \"49\",\n        },\n\"2020-06-18 01:36:41\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Genome sequencing\"],\n [{\n            'v': 54,\n            'f': \"54\",\n        },\n\"dcc_grusin\",\n\"PRJNA634167\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 6,\n            'f': \"6\",\n        },\n\"2020-09-01 01:52:33\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"A safe inhalational treatment prevents SARS\\u2013CoV-2 viral replication in human airway epithelial cells.\"],\n [{\n            'v': 55,\n            'f': \"55\",\n        },\n\"dcc_grusin\",\n\"PRJNA634194\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2020-05-22 01:46:01\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Broad inhibition of coronavirus entry by a lipid-conjugated peptide derived from the SARS-CoV-2 Spike Protein HRC Domain\"],\n [{\n            'v': 56,\n            'f': \"56\",\n        },\n\"dcc_grusin\",\n\"PRJNA634356\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 874,\n            'f': \"874\",\n        },\n\"2023-10-22 11:32:57\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 genome sequencing and assembly Emory Healthcare 2020\"],\n [{\n            'v': 57,\n            'f': \"57\",\n        },\n\"dcc_grusin\",\n\"PRJNA634377\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 20,\n            'f': \"20\",\n        },\n\"2020-05-23 01:39:55\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Coronavirus replication is regulated by both host and viral RNA N6\\u2013methyladenosine modifications\"],\n [{\n            'v': 58,\n            'f': \"58\",\n        },\n\"dcc_grusin\",\n\"PRJNA634396\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2021-06-08 06:36:00\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Genome sequencing\"],\n [{\n            'v': 59,\n            'f': \"59\",\n        },\n\"dcc_grusin\",\n\"PRJNA634489\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 15,\n            'f': \"15\",\n        },\n\"2020-07-01 01:41:00\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNA-seq of platelets from SARS-CoV-2 Covid-19\"],\n [{\n            'v': 60,\n            'f': \"60\",\n        },\n\"dcc_grusin\",\n\"PRJNA634718\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2020-05-30 01:32:49\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Effect of a nsp1 deletion in SARS-CoV-2 virus\"],\n [{\n            'v': 61,\n            'f': \"61\",\n        },\n\"dcc_grusin\",\n\"PRJNA634938\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 76,\n            'f': \"76\",\n        },\n\"2020-07-02 01:44:54\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Blocking of the CD80/86 axis as a therapeutic approach to prevent progression to more severe forms of COVID-19\"],\n [{\n            'v': 62,\n            'f': \"62\",\n        },\n\"dcc_grusin\",\n\"PRJNA634965\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2020-06-03 01:36:04\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genome sequences of SARS-CoV-2 in Hong Kong\"],\n [{\n            'v': 63,\n            'f': \"63\",\n        },\n\"dcc_grusin\",\n\"PRJNA635017\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-05-27 01:38:32\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-Cov-2 virus strain ISR_IT0320 genome sequencing\"],\n [{\n            'v': 64,\n            'f': \"64\",\n        },\n\"dcc_grusin\",\n\"PRJNA635443\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 23,\n            'f': \"23\",\n        },\n\"2021-01-15 03:33:34\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genome-wide mapping of SARS-CoV-2 RNA structures identifies therapeutically-relevant elements\"],\n [{\n            'v': 65,\n            'f': \"65\",\n        },\n\"dcc_grusin\",\n\"PRJNA635577\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8,\n            'f': \"8\",\n        },\n\"2020-07-31 01:39:19\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Non-neuronal expression of SARS-CoV-2 entry genes in the olfactory system suggests mechanisms underlying COVID-19-associated anosmia (scRNA-seq: olfactory mucosa)\"],\n [{\n            'v': 66,\n            'f': \"66\",\n        },\n\"dcc_grusin\",\n\"PRJNA635934\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-07-04 01:30:41\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Neutralizing Antibody and Soluble ACE2 Inhibition of a Replication-Competent VSV-SARS-CoV-2 and a Clinical Isolate of SARS-CoV-2\"],\n [{\n            'v': 67,\n            'f': \"67\",\n        },\n\"dcc_grusin\",\n\"PRJNA636004\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-05-31 01:28:39\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Isolation and characterization of severe acute respiratory syndrome coronavirus 2 in Turkey\"],\n [{\n            'v': 68,\n            'f': \"68\",\n        },\n\"dcc_grusin\",\n\"PRJNA636225\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 14,\n            'f': \"14\",\n        },\n\"2020-08-16 01:39:59\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Sequencing of SARS-CoV-2 from the Greater Geelong area, Victoria, Australia\"],\n [{\n            'v': 69,\n            'f': \"69\",\n        },\n\"dcc_grusin\",\n\"PRJNA636446\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 11,\n            'f': \"11\",\n        },\n\"2020-06-02 02:00:30\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Whole genome sequencing of SARS-CoV-2 using amplicon-based sequencing, random hexamers, and bait capture\"],\n [{\n            'v': 70,\n            'f': \"70\",\n        },\n\"dcc_grusin\",\n\"PRJNA636748\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 17118,\n            'f': \"17118\",\n        },\n\"2023-08-03 00:54:31\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"KRISP Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) virus sequencing in South-Africa\"],\n [{\n            'v': 71,\n            'f': \"71\",\n        },\n\"dcc_grusin\",\n\"PRJNA636824\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 84,\n            'f': \"84\",\n        },\n\"2020-08-04 02:07:10\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Alterations of the Gut Microbiota in Patients with COVID-19 or H1N1 Influenza\"],\n [{\n            'v': 72,\n            'f': \"72\",\n        },\n\"dcc_grusin\",\n\"PRJNA637143\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 56,\n            'f': \"56\",\n        },\n\"2020-09-03 02:02:37\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Two distinct immunopathological profiles in lungs of lethal COVID-19\"],\n [{\n            'v': 73,\n            'f': \"73\",\n        },\n\"dcc_grusin\",\n\"PRJNA637285\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2020-08-31 01:45:41\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Sequencing of SARS-Cov-2 from oropharyngeal swab and sputum\"],\n [{\n            'v': 74,\n            'f': \"74\",\n        },\n\"dcc_grusin\",\n\"PRJNA637303\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 21,\n            'f': \"21\",\n        },\n\"2020-06-13 01:30:25\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Identification of Lead Drug Candidates for COVID-19 based on Drug Screening Using Human Pluripotent Stem Cell-Derived Cells/Organoids\"],\n [{\n            'v': 75,\n            'f': \"75\",\n        },\n\"dcc_grusin\",\n\"PRJNA637515\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 32,\n            'f': \"32\",\n        },\n\"2020-06-12 01:32:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Multiple approaches for massively parallel sequencing of SARS-CoV-2 samples\"],\n [{\n            'v': 76,\n            'f': \"76\",\n        },\n\"dcc_grusin\",\n\"PRJNA637892\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-06-08 01:33:23\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"sars-cov2 sequencing\"],\n [{\n            'v': 77,\n            'f': \"77\",\n        },\n\"dcc_grusin\",\n\"PRJNA638039\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 16,\n            'f': \"16\",\n        },\n\"2021-05-11 12:59:31\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"multiplex RPA based Nanopore sequencing for real-time detection of virus\"],\n [{\n            'v': 78,\n            'f': \"78\",\n        },\n\"dcc_grusin\",\n\"PRJNA638211\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 10,\n            'f': \"10\",\n        },\n\"2020-08-01 01:41:10\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Epidemiological and Genomic Analysis of SARS-CoV-2 in Ten Patients from a Mid-sized City outside of Hubei, China in the Early Phase of the COVID-19 Outbreak\"],\n [{\n            'v': 79,\n            'f': \"79\",\n        },\n\"dcc_grusin\",\n\"PRJNA638224\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 19,\n            'f': \"19\",\n        },\n\"2020-06-10 01:42:02\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"BCR repertoire sequencing from COVID-19 patients\"],\n [{\n            'v': 80,\n            'f': \"80\",\n        },\n\"dcc_grusin\",\n\"PRJNA638251\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-06-10 01:42:04\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Mapping of SARS-CoV-2 Coronavirus Genome in COVID-19 Positive Patients in Kocaeli Region\"],\n [{\n            'v': 81,\n            'f': \"81\",\n        },\n\"dcc_grusin\",\n\"PRJNA638384\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2021-01-17 03:38:26\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Genome sequencing and assembly: Michigan Department of Health and Human Services\"],\n [{\n            'v': 82,\n            'f': \"82\",\n        },\n\"dcc_grusin\",\n\"PRJNA638614\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2020-10-20 02:38:04\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Alpaca nanobodies against SARS-CoV-2: Tyson\"],\n [{\n            'v': 83,\n            'f': \"83\",\n        },\n\"dcc_grusin\",\n\"PRJNA639066\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 19029,\n            'f': \"19029\",\n        },\n\"2023-10-22 11:32:37\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"TX SARS-CoV-2 Sequencing\"],\n [{\n            'v': 84,\n            'f': \"84\",\n        },\n\"dcc_grusin\",\n\"PRJNA639275\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 68,\n            'f': \"68\",\n        },\n\"2020-08-01 01:41:11\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Systems biological assessment of immunity to severe and mild COVID-19 infections\"],\n [{\n            'v': 85,\n            'f': \"85\",\n        },\n\"dcc_grusin\",\n\"PRJNA639367\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 16,\n            'f': \"16\",\n        },\n\"2020-06-16 01:40:13\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNA sequence of PBMCs of macaques infected with SARS-CoV-2\"],\n [{\n            'v': 86,\n            'f': \"86\",\n        },\n\"dcc_grusin\",\n\"PRJNA639551\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2020-06-18 01:37:05\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genomic RNA from SARS-Related Coronavirus 2\"],\n [{\n            'v': 87,\n            'f': \"87\",\n        },\n\"dcc_grusin\",\n\"PRJNA639580\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 78,\n            'f': \"78\",\n        },\n\"2020-10-09 10:26:52\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Imbalance of regulatory and cytotoxic SARS-CoV-2-reactive CD4+ T cells in COVID-19\"],\n [{\n            'v': 88,\n            'f': \"88\",\n        },\n\"dcc_grusin\",\n\"PRJNA639591\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2020-07-08 01:50:23\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 WGS: Innovative Genomics Institute, UC Berkeley\"],\n [{\n            'v': 89,\n            'f': \"89\",\n        },\n\"dcc_grusin\",\n\"PRJNA639864\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 46,\n            'f': \"46\",\n        },\n\"2020-06-26 01:29:53\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Deep-sequencing analysis of SARS-CoV-2 viruses\"],\n [{\n            'v': 90,\n            'f': \"90\",\n        },\n\"dcc_grusin\",\n\"PRJNA639956\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 446,\n            'f': \"446\",\n        },\n\"2022-04-27 02:52:51\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"deep mutational scanning of the SARS-CoV-2 RBD using yeast display of barcoded libraries\"],\n [{\n            'v': 91,\n            'f': \"91\",\n        },\n\"dcc_grusin\",\n\"PRJNA640656\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 88,\n            'f': \"88\",\n        },\n\"2020-08-07 11:04:49\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 genome sequencing\"],\n [{\n            'v': 92,\n            'f': \"92\",\n        },\n\"dcc_grusin\",\n\"PRJNA641813\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 23,\n            'f': \"23\",\n        },\n\"2020-06-26 01:30:24\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 is transmitted via contact and via the air between ferrets.\"],\n [{\n            'v': 93,\n            'f': \"93\",\n        },\n\"dcc_grusin\",\n\"PRJNA641908\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 9,\n            'f': \"9\",\n        },\n\"2020-09-22 02:12:25\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Global transcriptomic analyses of pluripotent stem cell-derived type II alveolar epithelial-like cells at 1 and 4 days after infection with SARS-CoV-2\"],\n [{\n            'v': 94,\n            'f': \"94\",\n        },\n\"dcc_grusin\",\n\"PRJNA641978\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 9,\n            'f': \"9\",\n        },\n\"2020-06-27 01:27:49\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"transcriptome sequencing of COVID-19 patients\"],\n [{\n            'v': 95,\n            'f': \"95\",\n        },\n\"dcc_grusin\",\n\"PRJNA642012\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 29,\n            'f': \"29\",\n        },\n\"2020-06-30 01:31:34\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Secondary pneumonia in critically ill ventilated patients with COVID-19\"],\n [{\n            'v': 96,\n            'f': \"96\",\n        },\n\"dcc_grusin\",\n\"PRJNA642727\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-07-01 01:41:11\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SAR-COV-2 Sequencing in Morocco\"],\n [{\n            'v': 97,\n            'f': \"97\",\n        },\n\"dcc_grusin\",\n\"PRJNA642869\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 7,\n            'f': \"7\",\n        },\n\"2021-03-16 04:56:39\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Sequencing of SARS-CoV-2 genomes from Morocco\"],\n [{\n            'v': 98,\n            'f': \"98\",\n        },\n\"dcc_grusin\",\n\"PRJNA642962\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2020-07-01 01:41:00\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"COVID-19 Single Cell VDJ sequencing\"],\n [{\n            'v': 99,\n            'f': \"99\",\n        },\n\"dcc_grusin\",\n\"PRJNA643530\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2020-11-18 01:49:44\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 Genome sequencing from UAMS patients\"],\n [{\n            'v': 100,\n            'f': \"100\",\n        },\n\"dcc_grusin\",\n\"PRJNA643574\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2020-07-03 01:35:11\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 Raw sequence reads which mapped to reference\"],\n [{\n            'v': 101,\n            'f': \"101\",\n        },\n\"dcc_grusin\",\n\"PRJNA643575\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 85,\n            'f': \"85\",\n        },\n\"2020-07-08 01:50:33\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 Raw sequence reads from ARCTIC V3 tiled amplicon approach\"],\n [{\n            'v': 102,\n            'f': \"102\",\n        },\n\"dcc_grusin\",\n\"PRJNA643654\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2020-07-03 01:35:11\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Whole genome sequencing of SARS-CoV-2 isolated from suspected cases among Bangladeshi patients.\"],\n [{\n            'v': 103,\n            'f': \"103\",\n        },\n\"dcc_grusin\",\n\"PRJNA643694\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2020-10-03 02:32:27\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 infection and replication in human fetal and pediatric gastric organoids\"],\n [{\n            'v': 104,\n            'f': \"104\",\n        },\n\"dcc_grusin\",\n\"PRJNA644357\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 15,\n            'f': \"15\",\n        },\n\"2021-03-15 02:38:18\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Structure of the full SARS-CoV-2 RNA genome in infected cells\"],\n [{\n            'v': 105,\n            'f': \"105\",\n        },\n\"dcc_grusin\",\n\"PRJNA644579\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 45,\n            'f': \"45\",\n        },\n\"2020-07-24 01:55:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severely ill COVID-19 patients display impaired exhaustion features in SARS-CoV-2-reactive CD8+ T cells\"],\n [{\n            'v': 106,\n            'f': \"106\",\n        },\n\"dcc_grusin\",\n\"PRJNA644588\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 6,\n            'f': \"6\",\n        },\n\"2020-07-21 01:50:04\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Gene expression of SARS-CoV-2-infected Vero E6 cells\"],\n [{\n            'v': 107,\n            'f': \"107\",\n        },\n\"dcc_grusin\",\n\"PRJNA644625\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2020-07-22 02:24:40\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Primary Human Airway Epithelial Cultures infected with SARS-CoV-2\"],\n [{\n            'v': 108,\n            'f': \"108\",\n        },\n\"dcc_grusin\",\n\"PRJNA644648\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 22,\n            'f': \"22\",\n        },\n\"2020-11-13 02:10:45\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNA secondary structome of SARS-CoV-2 and other 6 virus structures in the UTR regions\"],\n [{\n            'v': 109,\n            'f': \"109\",\n        },\n\"dcc_grusin\",\n\"PRJNA645048\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 21,\n            'f': \"21\",\n        },\n\"2020-07-10 10:58:53\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-Cov-2 Sequencing Paragon\"],\n [{\n            'v': 110,\n            'f': \"110\",\n        },\n\"dcc_grusin\",\n\"PRJNA645051\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 17,\n            'f': \"17\",\n        },\n\"2020-07-10 10:58:52\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-Cov-2 Sequencing NEB + Twist 2\"],\n [{\n            'v': 111,\n            'f': \"111\",\n        },\n\"dcc_grusin\",\n\"PRJNA645052\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 30,\n            'f': \"30\",\n        },\n\"2020-07-10 10:58:52\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-Cov-2 Sequencing NEB + Twist 1\"],\n [{\n            'v': 112,\n            'f': \"112\",\n        },\n\"dcc_grusin\",\n\"PRJNA645133\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 20,\n            'f': \"20\",\n        },\n\"2020-07-31 01:39:22\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 infection in the lungs of human ACE2 transgenic mice causes severe inflammation and immune cell infiltration, and compromised respiratory function\"],\n [{\n            'v': 113,\n            'f': \"113\",\n        },\n\"dcc_grusin\",\n\"PRJNA645209\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 15,\n            'f': \"15\",\n        },\n\"2020-10-16 01:44:00\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Targeting heparan sulfate proteoglycan-assisted endocytosis as a COVID-19 therapeutic option\"],\n [{\n            'v': 114,\n            'f': \"114\",\n        },\n\"dcc_grusin\",\n\"PRJNA645245\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 87,\n            'f': \"87\",\n        },\n\"2020-07-11 01:32:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Dynamics of B-cell repertoires and emergence of cross-reactive responses in COVID-19 patients with different disease severity\"],\n [{\n            'v': 115,\n            'f': \"115\",\n        },\n\"dcc_grusin\",\n\"PRJNA645342\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2021-01-12 12:03:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Complete in vivo SHAPE-MaP structure of the SARS-CoV-2 genome\"],\n [{\n            'v': 116,\n            'f': \"116\",\n        },\n\"dcc_grusin\",\n\"PRJNA645718\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2020-07-14 01:38:49\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 amplicon sequencing\"],\n [{\n            'v': 117,\n            'f': \"117\",\n        },\n\"dcc_grusin\",\n\"PRJNA645970\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 173,\n            'f': \"173\",\n        },\n\"2020-07-14 01:38:56\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genomic epidemiology of the early stages of SARS-CoV-2 outbreak in Russia\"],\n [{\n            'v': 118,\n            'f': \"118\",\n        },\n\"dcc_grusin\",\n\"PRJNA646224\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 38,\n            'f': \"38\",\n        },\n\"2020-10-31 02:31:02\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Transcriptional and proteomic insights into the host response in fatal COVID-19 cases\"],\n [{\n            'v': 119,\n            'f': \"119\",\n        },\n\"dcc_grusin\",\n\"PRJNA646535\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 10,\n            'f': \"10\",\n        },\n\"2020-07-17 01:45:03\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Mouse model of SARS-CoV-2 reveals inflammatory role of type I interferon signaling\"],\n [{\n            'v': 120,\n            'f': \"120\",\n        },\n\"dcc_grusin\",\n\"PRJNA646704\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 17,\n            'f': \"17\",\n        },\n\"2020-09-30 12:07:29\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Single cell RNAseq reveal covid19 patient immune response\"],\n [{\n            'v': 121,\n            'f': \"121\",\n        },\n\"dcc_grusin\",\n\"PRJNA646783\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 264,\n            'f': \"264\",\n        },\n\"2020-07-20 01:53:28\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Modulating the transcriptional landscape of SARS-CoV-2 as an effective method for developing antiviral compounds\"],\n [{\n            'v': 122,\n            'f': \"122\",\n        },\n\"dcc_grusin\",\n\"PRJNA647057\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-07-23 01:41:32\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"sequencing of SARS-Co-V2 isolates from Egypt\"],\n [{\n            'v': 123,\n            'f': \"123\",\n        },\n\"dcc_grusin\",\n\"PRJNA647390\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 9,\n            'f': \"9\",\n        },\n\"2020-10-09 10:21:17\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genome-wide CRISPR screen reveals host genes that regulate SARS-CoV-2 infection\"],\n [{\n            'v': 124,\n            'f': \"124\",\n        },\n\"dcc_grusin\",\n\"PRJNA647434\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 6,\n            'f': \"6\",\n        },\n\"2020-10-09 10:21:18\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genome-wide CRISPR screen reveals host genes that regulate SARS-CoV-2 infection [ATAC-seq]\"],\n [{\n            'v': 125,\n            'f': \"125\",\n        },\n\"dcc_grusin\",\n\"PRJNA647435\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8,\n            'f': \"8\",\n        },\n\"2020-10-09 10:21:18\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genome-wide CRISPR screen reveals host genes that regulate SARS-CoV-2 infection [RNA-seq]\"],\n [{\n            'v': 126,\n            'f': \"126\",\n        },\n\"dcc_grusin\",\n\"PRJNA647448\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 7,\n            'f': \"7\",\n        },\n\"2020-09-24 01:38:45\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Genome sequencing\"],\n [{\n            'v': 127,\n            'f': \"127\",\n        },\n\"dcc_grusin\",\n\"PRJNA647529\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 212,\n            'f': \"212\",\n        },\n\"2020-07-22 02:25:16\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Full genome viral sequences inform patterns of SARS-CoV-2 spread into and within Israel\"],\n [{\n            'v': 128,\n            'f': \"128\",\n        },\n\"dcc_grusin\",\n\"PRJNA647532\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2020-07-22 02:25:13\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"A human disease model of SARS-CoV-2-induced lung injury and immune responses with a microengineered organ chip\"],\n [{\n            'v': 129,\n            'f': \"129\",\n        },\n\"dcc_grusin\",\n\"PRJNA648120\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 14,\n            'f': \"14\",\n        },\n\"2020-10-03 02:32:38\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Transcriptional Profiling of Leukocytes in Critically Ill COVID19 Patients\"],\n [{\n            'v': 130,\n            'f': \"130\",\n        },\n\"dcc_grusin\",\n\"PRJNA648306\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 56,\n            'f': \"56\",\n        },\n\"2020-07-25 02:09:13\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"studies of microbial diversity in COVID-19 patients\"],\n [{\n            'v': 131,\n            'f': \"131\",\n        },\n\"dcc_grusin\",\n\"PRJNA648425\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2020-07-27 01:52:34\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Human dendritic cell and macrophage response to SARS-CoV-2 infection\"],\n [{\n            'v': 132,\n            'f': \"132\",\n        },\n\"dcc_grusin\",\n\"PRJNA648677\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 113,\n            'f': \"113\",\n        },\n\"2021-01-25 16:41:50\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Sequencing on B cell receptor repertoire of COVID-19 patients and bio-panning libraries to SARS-CoV-2\"],\n [{\n            'v': 133,\n            'f': \"133\",\n        },\n\"dcc_grusin\",\n\"PRJNA648821\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 13,\n            'f': \"13\",\n        },\n\"2020-08-14 01:45:52\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Favipiravir treatment in Vivo on Hamster following SARS Cov2 infection\"],\n [{\n            'v': 134,\n            'f': \"134\",\n        },\n\"dcc_grusin\",\n\"PRJNA649019\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2020-08-13 01:39:24\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Identification of Drugs Blocking SARS-CoV-2 Infection using hPSC-derived Lung and Colonic Organoids\"],\n [{\n            'v': 135,\n            'f': \"135\",\n        },\n\"dcc_grusin\",\n\"PRJNA649101\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 96,\n            'f': \"96\",\n        },\n\"2020-07-31 01:39:30\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Evaluation of NGS-based approaches for SARS-CoV-2 whole genome characterization\"],\n [{\n            'v': 136,\n            'f': \"136\",\n        },\n\"dcc_grusin\",\n\"PRJNA649246\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 26,\n            'f': \"26\",\n        },\n\"2020-09-24 01:37:58\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 infection is effectively treated and prevented by EIDD-2801\"],\n [{\n            'v': 137,\n            'f': \"137\",\n        },\n\"dcc_grusin\",\n\"PRJNA649525\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 48,\n            'f': \"48\",\n        },\n\"2020-07-31 01:39:34\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Transcriptomic profiles of whole blood from SARS-CoV-2 rhesus macaques\"],\n [{\n            'v': 138,\n            'f': \"138\",\n        },\n\"dcc_grusin\",\n\"PRJNA650037\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 270,\n            'f': \"270\",\n        },\n\"2020-08-26 02:04:13\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Johns Hopkins Viral Genomics of Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)\"],\n [{\n            'v': 139,\n            'f': \"139\",\n        },\n\"dcc_grusin\",\n\"PRJNA650134\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 22,\n            'f': \"22\",\n        },\n\"2020-08-02 01:57:11\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 Quasispecies Mediates Rapid Virus Evolution and Adaptation\"],\n [{\n            'v': 140,\n            'f': \"140\",\n        },\n\"dcc_grusin\",\n\"PRJNA650245\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 864,\n            'f': \"864\",\n        },\n\"2020-08-20 02:06:16\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"NYU Langone SARS-CoV-2 Sequencing Team\"],\n [{\n            'v': 141,\n            'f': \"141\",\n        },\n\"dcc_grusin\",\n\"PRJNA655201\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2021-01-04 13:54:53\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Sars-Cov-2 DCV Quasispecies\"],\n [{\n            'v': 142,\n            'f': \"142\",\n        },\n\"dcc_grusin\",\n\"PRJNA655427\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2021-03-29 02:07:39\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"The architecture of the SARS-CoV-2 RNA genome inside virion\"],\n [{\n            'v': 143,\n            'f': \"143\",\n        },\n\"dcc_grusin\",\n\"PRJNA655577\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1536,\n            'f': \"1536\",\n        },\n\"2020-08-12 02:01:12\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"High throughput detection and genetic epidemiology of SARS-CoV-2 using COVIDseq next generation sequencing\"],\n [{\n            'v': 144,\n            'f': \"144\",\n        },\n\"dcc_grusin\",\n\"PRJNA655740\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 144,\n            'f': \"144\",\n        },\n\"2020-08-10 01:56:04\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Systems biological assessment of immunity to severe and mild COVID-19 infections\"],\n [{\n            'v': 145,\n            'f': \"145\",\n        },\n\"dcc_grusin\",\n\"PRJNA655789\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1370,\n            'f': \"1370\",\n        },\n\"2023-10-13 00:17:13\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"US Air Force School of Aerospace Medicine SARS-CoV-2 genome sequencing\"],\n [{\n            'v': 146,\n            'f': \"146\",\n        },\n\"dcc_grusin\",\n\"PRJNA656060\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2020-08-19 02:13:54\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"A novel bat coronavirus closely related to SARS-CoV-2\"],\n [{\n            'v': 147,\n            'f': \"147\",\n        },\n\"dcc_grusin\",\n\"PRJNA656299\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 16,\n            'f': \"16\",\n        },\n\"2020-09-30 14:01:48\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 infection induces transcriptomic alterations specific to activation of innate immunity and cellular distress.\"],\n [{\n            'v': 148,\n            'f': \"148\",\n        },\n\"dcc_grusin\",\n\"PRJNA656534\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 13289,\n            'f': \"13289\",\n        },\n\"2023-10-22 11:25:49\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Genome sequencing\"],\n [{\n            'v': 149,\n            'f': \"149\",\n        },\n\"dcc_grusin\",\n\"PRJNA656695\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 171,\n            'f': \"171\",\n        },\n\"2020-08-13 01:39:40\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Philadelphia SARS-CoV-2 Genomes\"],\n [{\n            'v': 150,\n            'f': \"150\",\n        },\n\"dcc_grusin\",\n\"PRJNA656810\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 9,\n            'f': \"9\",\n        },\n\"2020-08-13 01:40:05\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"sewage metagenomics covid\"],\n [{\n            'v': 151,\n            'f': \"151\",\n        },\n\"dcc_grusin\",\n\"PRJNA657032\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 79,\n            'f': \"79\",\n        },\n\"2023-09-24 04:51:57\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 sequencing in cancer patients from Brazil\"],\n [{\n            'v': 152,\n            'f': \"152\",\n        },\n\"dcc_grusin\",\n\"PRJNA657893\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 20672,\n            'f': \"20672\",\n        },\n\"2023-06-24 10:14:18\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 - raw sequence reads\"],\n [{\n            'v': 153,\n            'f': \"153\",\n        },\n\"dcc_grusin\",\n\"PRJNA657938\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-08-22 01:50:40\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Complete Genome Sequencing of SARS-COV-2 Strains Isolated From Bangladesh\"],\n [{\n            'v': 154,\n            'f': \"154\",\n        },\n\"dcc_grusin\",\n\"PRJNA657985\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-08-20 02:06:32\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Complete Genome Sequencing of SARS-COV-2 Strains Isolated From Bangladesh\"],\n [{\n            'v': 155,\n            'f': \"155\",\n        },\n\"dcc_grusin\",\n\"PRJNA658211\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-08-21 01:51:48\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Complete Genome Sequencing of SARS-COV-2 Strains Isolated From Bangladesh\"],\n [{\n            'v': 156,\n            'f': \"156\",\n        },\n\"dcc_grusin\",\n\"PRJNA658242\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2021-01-23 04:09:53\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2/human/Giessen/IMVGi-1CCA/2020\"],\n [{\n            'v': 157,\n            'f': \"157\",\n        },\n\"dcc_grusin\",\n\"PRJNA658479\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 15,\n            'f': \"15\",\n        },\n\"2020-09-17 01:55:44\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Transposon mutagenesis screen identifies MHC class II transactivator (CIITA) as an inducer of cellular resistance to Ebola virus and SARS Coronavirus\"],\n [{\n            'v': 158,\n            'f': \"158\",\n        },\n\"dcc_grusin\",\n\"PRJNA658490\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2020-08-25 02:11:12\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genome Sequence of SARS-CoV-2 Siena-1 viral isolate\"],\n [{\n            'v': 159,\n            'f': \"159\",\n        },\n\"dcc_grusin\",\n\"PRJNA658711\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2020-08-24 01:55:39\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNA-Sequencing of intestinal eithelial and endothelial cells upon SARS-CoV-2 infection on a intestine chip\"],\n [{\n            'v': 160,\n            'f': \"160\",\n        },\n\"dcc_grusin\",\n\"PRJNA658795\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 246,\n            'f': \"246\",\n        },\n\"2020-10-21 02:54:53\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Vascular Disease and Thrombosis in SARS-CoV-2 Infected Humans and Rhesus Macaques\"],\n [{\n            'v': 161,\n            'f': \"161\",\n        },\n\"dcc_grusin\",\n\"PRJNA658974\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 40,\n            'f': \"40\",\n        },\n\"2020-09-22 02:12:36\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV2 infection of iPSC-derived cardiac cells predicts novel cytopathic features in COVID-19 patients\\u00a0\"],\n [{\n            'v': 162,\n            'f': \"162\",\n        },\n\"dcc_grusin\",\n\"PRJNA659293\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 23,\n            'f': \"23\",\n        },\n\"2020-08-28 01:59:59\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genome Sequences of Twenty-three SARS-CoV-2 Strains Isolated in the Philippines\"],\n [{\n            'v': 163,\n            'f': \"163\",\n        },\n\"dcc_grusin\",\n\"PRJNA660067\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 252,\n            'f': \"252\",\n        },\n\"2020-08-31 01:46:17\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Large-scale Multi-omic Analysis of COVID-19 Severity\"],\n [{\n            'v': 164,\n            'f': \"164\",\n        },\n\"dcc_grusin\",\n\"PRJNA660302\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 564,\n            'f': \"564\",\n        },\n\"2020-09-01 01:54:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Alterations of the Human Oral and Gut Microbiome and Lipid Metabonomics in COVID-19\"],\n [{\n            'v': 165,\n            'f': \"165\",\n        },\n\"dcc_grusin\",\n\"PRJNA660883\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 100,\n            'f': \"100\",\n        },\n\"2022-09-24 11:56:50\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"human stool Metagenome from acute COVID19+ subjects\"],\n [{\n            'v': 166,\n            'f': \"166\",\n        },\n\"dcc_grusin\",\n\"PRJNA661544\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 15,\n            'f': \"15\",\n        },\n\"2020-11-20 01:43:19\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 strain:Washington Raw sequence reads following adaptation in mice\"],\n [{\n            'v': 167,\n            'f': \"167\",\n        },\n\"dcc_grusin\",\n\"PRJNA661970\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 6,\n            'f': \"6\",\n        },\n\"2020-09-08 01:47:15\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Spermatogenesis damages in COVID-19 patients\"],\n [{\n            'v': 168,\n            'f': \"168\",\n        },\n\"dcc_grusin\",\n\"PRJNA662193\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 651,\n            'f': \"651\",\n        },\n\"2021-01-14 04:14:19\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 strain:SARS-CoV2 Raw sequence reads\"],\n [{\n            'v': 169,\n            'f': \"169\",\n        },\n\"dcc_grusin\",\n\"PRJNA662589\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 9,\n            'f': \"9\",\n        },\n\"2020-09-24 01:37:59\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Raw sequence reads of SARS-CoV-2 from a prolonged infection\"],\n [{\n            'v': 170,\n            'f': \"170\",\n        },\n\"dcc_grusin\",\n\"PRJNA662596\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 92,\n            'f': \"92\",\n        },\n\"2021-01-23 04:03:20\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) surveillance in wastewater samples in the USA\"],\n [{\n            'v': 171,\n            'f': \"171\",\n        },\n\"dcc_grusin\",\n\"PRJNA662684\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 33,\n            'f': \"33\",\n        },\n\"2020-09-11 01:44:57\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-Cov-2 from Brazilian Amazon\"],\n [{\n            'v': 172,\n            'f': \"172\",\n        },\n\"dcc_grusin\",\n\"PRJNA662905\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-09-30 14:01:52\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 in Patients with Severe Acute Respiratory Illness in Egypt\"],\n [{\n            'v': 173,\n            'f': \"173\",\n        },\n\"dcc_grusin\",\n\"PRJNA662965\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 9,\n            'f': \"9\",\n        },\n\"2020-09-24 01:38:00\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Human Pluripotent Stem Cell-Derived Neural Cells and Brain Organoids Reveal SARS-CoV-2 Neurotropism Predominates in Choroid Plexus Epithelium\"],\n [{\n            'v': 174,\n            'f': \"174\",\n        },\n\"dcc_grusin\",\n\"PRJNA663402\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 14,\n            'f': \"14\",\n        },\n\"2020-10-21 02:54:57\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WGS_SARS-CoV-2\"],\n [{\n            'v': 175,\n            'f': \"175\",\n        },\n\"dcc_grusin\",\n\"PRJNA663861\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2021-01-04 13:57:47\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genome-scale deconvolution of RNA structure ensembles\"],\n [{\n            'v': 176,\n            'f': \"176\",\n        },\n\"dcc_grusin\",\n\"PRJNA663917\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 460,\n            'f': \"460\",\n        },\n\"2021-03-19 06:00:03\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Targeted sequencing of hypermutable and conserved regions of the SARS-CoV-2 genome\"],\n [{\n            'v': 177,\n            'f': \"177\",\n        },\n\"dcc_grusin\",\n\"PRJNA664887\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 39,\n            'f': \"39\",\n        },\n\"2020-10-22 02:19:06\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Remdesivir targets a structurally analogous region of the Ebola virus and SARS-CoV-2 polymerases\"],\n [{\n            'v': 178,\n            'f': \"178\",\n        },\n\"dcc_grusin\",\n\"PRJNA665035\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 30,\n            'f': \"30\",\n        },\n\"2021-06-08 07:05:27\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genes with 5\\u2032 terminal oligopyrimidine tracts preferentially escape global suppression of translation by the SARS-CoV-2 NSP1 protein\"],\n [{\n            'v': 179,\n            'f': \"179\",\n        },\n\"dcc_faang\",\n\"PRJNA665193\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 48,\n            'f': \"48\",\n        },\n\"2020-09-30 12:07:34\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Comparative functional genome annotation of livestock species identifies core amniote regulatory features and avian enhancer versatility (RNA-Seq)\"],\n [{\n            'v': 180,\n            'f': \"180\",\n        },\n\"dcc_faang\",\n\"PRJNA665194\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 31,\n            'f': \"31\",\n        },\n\"2020-09-30 12:07:35\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Comparative functional genome annotation of livestock species identifies core amniote regulatory features and avian enhancer versatility (ATAC-Seq)\"],\n [{\n            'v': 181,\n            'f': \"181\",\n        },\n\"dcc_faang\",\n\"PRJNA665196\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 15,\n            'f': \"15\",\n        },\n\"2020-09-30 14:01:53\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Comparative functional genome annotation of livestock species identifies core amniote regulatory features and avian enhancer versatility (DNase-Hypersensitivity)\"],\n [{\n            'v': 182,\n            'f': \"182\",\n        },\n\"dcc_faang\",\n\"PRJNA665197\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 48,\n            'f': \"48\",\n        },\n\"2020-09-30 14:01:53\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Comparative functional genome annotation of livestock species identifies core amniote regulatory features and avian enhancer versatility (CTCF ChIP-Seq)\"],\n [{\n            'v': 183,\n            'f': \"183\",\n        },\n\"dcc_faang\",\n\"PRJNA665199\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 48,\n            'f': \"48\",\n        },\n\"2020-09-30 14:01:53\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Comparative functional genome annotation of livestock species identifies core amniote regulatory features and avian enhancer versatility (H3K27ac ChIP-Seq)\"],\n [{\n            'v': 184,\n            'f': \"184\",\n        },\n\"dcc_faang\",\n\"PRJNA665209\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 48,\n            'f': \"48\",\n        },\n\"2020-09-30 14:01:53\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Comparative functional genome annotation of livestock species identifies core amniote regulatory features and avian enhancer versatility (H3K27me3 ChIP-Seq)\"],\n [{\n            'v': 185,\n            'f': \"185\",\n        },\n\"dcc_faang\",\n\"PRJNA665214\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 48,\n            'f': \"48\",\n        },\n\"2020-09-30 14:01:53\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Comparative functional genome annotation of livestock species identifies core amniote regulatory features and avian enhancer versatility (H3K4me1 ChIP-Seq)\"],\n [{\n            'v': 186,\n            'f': \"186\",\n        },\n\"dcc_faang\",\n\"PRJNA665216\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 48,\n            'f': \"48\",\n        },\n\"2020-09-30 14:01:53\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Comparative functional genome annotation of livestock species identifies core amniote regulatory features and avian enhancer versatility (H3K4me3 ChIP-Seq)\"],\n [{\n            'v': 187,\n            'f': \"187\",\n        },\n\"dcc_grusin\",\n\"PRJNA665485\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8,\n            'f': \"8\",\n        },\n\"2021-01-04 13:58:01\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"UNAB/INDISA SARS-CoV-2 sequencing\"],\n [{\n            'v': 188,\n            'f': \"188\",\n        },\n\"dcc_grusin\",\n\"PRJNA665581\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 19,\n            'f': \"19\",\n        },\n\"2020-10-17 01:47:27\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 suppresses mRNA splicing, translation, and protein trafficking in a multipronged mechanism to evade host defenses\"],\n [{\n            'v': 189,\n            'f': \"189\",\n        },\n\"dcc_grusin\",\n\"PRJNA665692\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 73,\n            'f': \"73\",\n        },\n\"2020-10-17 01:47:28\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 suppresses mRNA splicing, translation, and protein trafficking in a multipronged mechanism to evade host defenses\"],\n [{\n            'v': 190,\n            'f': \"190\",\n        },\n\"dcc_grusin\",\n\"PRJNA666189\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-09-30 22:13:31\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)\"],\n [{\n            'v': 191,\n            'f': \"191\",\n        },\n\"dcc_grusin\",\n\"PRJNA666219\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 69,\n            'f': \"69\",\n        },\n\"2020-10-01 14:32:22\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 in sewage UK\"],\n [{\n            'v': 192,\n            'f': \"192\",\n        },\n\"dcc_grusin\",\n\"PRJNA666421\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 72,\n            'f': \"72\",\n        },\n\"2020-10-03 02:52:17\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Expression of SARS-CoV-2 receptor ACE2 and coincident host response signature varies by asthma inflammatory phenotype\"],\n [{\n            'v': 193,\n            'f': \"193\",\n        },\n\"dcc_grusin\",\n\"PRJNA666543\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1606,\n            'f': \"1606\",\n        },\n\"2023-05-06 00:03:43\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"NCSLPH SARS-CoV-2 Response Team Sequencing\"],\n [{\n            'v': 194,\n            'f': \"194\",\n        },\n\"dcc_grusin\",\n\"PRJNA666696\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 40,\n            'f': \"40\",\n        },\n\"2022-12-06 02:45:13\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 Deep Sequencing of viral challenge stocks produced by BIOQUAL labs\"],\n [{\n            'v': 195,\n            'f': \"195\",\n        },\n\"dcc_grusin\",\n\"PRJNA666926\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 97,\n            'f': \"97\",\n        },\n\"2021-05-12 12:45:19\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Transmission of SARS-CoV-2 in domestic cats\"],\n [{\n            'v': 196,\n            'f': \"196\",\n        },\n\"dcc_grusin\",\n\"PRJNA667180\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 102,\n            'f': \"102\",\n        },\n\"2020-10-05 01:57:24\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genomic surveillance of COVID-19 cases in Beijing\"],\n [{\n            'v': 197,\n            'f': \"197\",\n        },\n\"dcc_grusin\",\n\"PRJNA667434\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 10,\n            'f': \"10\",\n        },\n\"2020-10-09 10:28:41\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Genome sequencing and assembly\"],\n [{\n            'v': 198,\n            'f': \"198\",\n        },\n\"dcc_grusin\",\n\"PRJNA667798\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2020-10-09 10:28:19\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Establishment of 2019 Novel Coronavirus (SARS-CoV-2) Strain Genome Database in Malaysia\"],\n [{\n            'v': 199,\n            'f': \"199\",\n        },\n\"dcc_grusin\",\n\"PRJNA667921\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 16,\n            'f': \"16\",\n        },\n\"2020-10-10 01:55:12\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 infected cells present HLA-I peptides from canonical and out-of-frame ORFs\"],\n [{\n            'v': 200,\n            'f': \"200\",\n        },\n\"dcc_grusin\",\n\"PRJNA667999\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 9,\n            'f': \"9\",\n        },\n\"2020-10-10 01:55:15\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Development of a human ACE2-Fc fusion protein with potent prophylactic and therapeutic efficiency against SARS-CoV-2\"],\n [{\n            'v': 201,\n            'f': \"201\",\n        },\n\"dcc_grusin\",\n\"PRJNA668354\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 6,\n            'f': \"6\",\n        },\n\"2020-10-13 01:49:58\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Three-dimensional human alveolar stem cell culture models reveal infection response to SARS-CoV-2\"],\n [{\n            'v': 202,\n            'f': \"202\",\n        },\n\"dcc_grusin\",\n\"PRJNA668631\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 7,\n            'f': \"7\",\n        },\n\"2020-10-21 02:55:26\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"An engineered decoy receptor for SARS-CoV-2 broadly binds protein S sequence variants\"],\n [{\n            'v': 203,\n            'f': \"203\",\n        },\n\"dcc_grusin\",\n\"PRJNA668739\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 23,\n            'f': \"23\",\n        },\n\"2020-10-13 01:50:07\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Felis catus host response to feline coronavirus infection RNASeq reads\"],\n [{\n            'v': 204,\n            'f': \"204\",\n        },\n\"dcc_grusin\",\n\"PRJNA668889\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2021-01-04 13:58:39\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Proteo-genomic analysis of SARS-CoV-2\"],\n [{\n            'v': 205,\n            'f': \"205\",\n        },\n\"dcc_grusin\",\n\"PRJNA669043\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 474,\n            'f': \"474\",\n        },\n\"2022-01-24 02:56:55\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 genomic data and descriptive metadata from primary specimens collected in Manitoba, Canada during the COVID-19 outbreak\"],\n [{\n            'v': 206,\n            'f': \"206\",\n        },\n\"dcc_grusin\",\n\"PRJNA669459\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 15,\n            'f': \"15\",\n        },\n\"2020-10-21 02:55:52\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Genome sequencing\"],\n [{\n            'v': 207,\n            'f': \"207\",\n        },\n\"dcc_grusin\",\n\"PRJNA669553\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 352,\n            'f': \"352\",\n        },\n\"2021-01-04 14:00:39\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Replication and transmissibility of the SARS-CoV-2 spike D614G variant in multiple model systems\"],\n [{\n            'v': 208,\n            'f': \"208\",\n        },\n\"dcc_grusin\",\n\"PRJNA669833\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 16,\n            'f': \"16\",\n        },\n\"2020-10-21 02:55:39\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 Orf6 hijacks Nup98 to block STAT nuclear import and antagonize interferon signaling\"],\n [{\n            'v': 209,\n            'f': \"209\",\n        },\n\"dcc_grusin\",\n\"PRJNA669862\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8,\n            'f': \"8\",\n        },\n\"2021-01-04 13:58:59\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"The RNA architecture of the SARS-CoV-2 3-prime untranslated region\"],\n [{\n            'v': 210,\n            'f': \"210\",\n        },\n\"dcc_grusin\",\n\"PRJNA669945\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 69,\n            'f': \"69\",\n        },\n\"2020-10-27 03:14:00\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"69 Palestine SARS-CoV-2 isolates\"],\n [{\n            'v': 211,\n            'f': \"211\",\n        },\n\"dcc_grusin\",\n\"PRJNA670222\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2020-10-21 02:56:01\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 strain:Covid 19 | isolate:Nasopharyngeal (NP) swab Genome sequencing\"],\n [{\n            'v': 212,\n            'f': \"212\",\n        },\n\"dcc_faang\",\n\"PRJNA672728\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 85,\n            'f': \"85\",\n        },\n\"2020-10-30 02:36:57\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Mapping of transcription initiation in the swine genome by RAMPAGE\"],\n [{\n            'v': 213,\n            'f': \"213\",\n        },\n\"dcc_faang\",\n\"PRJNA672731\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 120,\n            'f': \"120\",\n        },\n\"2020-10-30 02:37:03\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Mapping of transcription initiation in the chicken genome by RAMPAGE\"],\n [{\n            'v': 214,\n            'f': \"214\",\n        },\n\"dcc_grusin\",\n\"PRJNA672811\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8,\n            'f': \"8\",\n        },\n\"2020-10-29 02:18:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Coding-Complete SARS-CoV-2 genome sequences originating from eight new Israeli patient isolates\"],\n [{\n            'v': 215,\n            'f': \"215\",\n        },\n\"dcc_grusin\",\n\"PRJNA673055\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2020-11-12 02:08:00\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genomic sequences and analysis of five SARS-CoV-2 variants obtained from patients in Lambayeque, Peru\"],\n [{\n            'v': 216,\n            'f': \"216\",\n        },\n\"dcc_grusin\",\n\"PRJNA673096\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 16047,\n            'f': \"16047\",\n        },\n\"2023-10-08 05:24:56\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"COVID-19\"],\n [{\n            'v': 217,\n            'f': \"217\",\n        },\n\"dcc_grusin\",\n\"PRJNA673341\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 7,\n            'f': \"7\",\n        },\n\"2020-11-02 02:13:37\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"KRISP0194\"],\n [{\n            'v': 218,\n            'f': \"218\",\n        },\n\"dcc_grusin\",\n\"PRJNA674624\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2021-01-12 12:03:33\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Amplicon Sequencing of SARS-CoV-2 for virus detection and sequence analyzes in humans and domestic animals\"],\n [{\n            'v': 219,\n            'f': \"219\",\n        },\n\"dcc_grusin\",\n\"PRJNA674796\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2020-11-06 02:36:11\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV2 sequencing of re-infection samples\"],\n [{\n            'v': 220,\n            'f': \"220\",\n        },\n\"dcc_grusin\",\n\"PRJNA675109\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 88,\n            'f': \"88\",\n        },\n\"2021-03-01 03:23:31\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 genomic data and descriptive metadata from primary specimens collected in New Brunswick, Canada during the COVID-19 outbreak.\"],\n [{\n            'v': 221,\n            'f': \"221\",\n        },\n\"dcc_grusin\",\n\"PRJNA675364\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 157,\n            'f': \"157\",\n        },\n\"2020-11-11 02:05:32\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Raw sequence reads\"],\n [{\n            'v': 222,\n            'f': \"222\",\n        },\n\"dcc_grusin\",\n\"PRJNA675370\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 152,\n            'f': \"152\",\n        },\n\"2021-12-11 02:39:46\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 infection in mammalian cell cultures\"],\n [{\n            'v': 223,\n            'f': \"223\",\n        },\n\"dcc_grusin\",\n\"PRJNA675840\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 66,\n            'f': \"66\",\n        },\n\"2020-11-14 01:59:03\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 genome sequencing\"],\n [{\n            'v': 224,\n            'f': \"224\",\n        },\n\"dcc_grusin\",\n\"PRJNA676016\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 249,\n            'f': \"249\",\n        },\n\"2022-12-01 02:08:10\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Integrative Genomics of COVID-19 Disease\"],\n [{\n            'v': 225,\n            'f': \"225\",\n        },\n\"dcc_grusin\",\n\"PRJNA678001\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 9,\n            'f': \"9\",\n        },\n\"2021-01-26 02:08:51\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Generation and Characterization of recombinant SARS-CoV-2 expressing reporter genes\"],\n [{\n            'v': 226,\n            'f': \"226\",\n        },\n\"dcc_grusin\",\n\"PRJNA678831\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 125,\n            'f': \"125\",\n        },\n\"2022-12-22 02:14:59\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Host Response during COVID-19\"],\n [{\n            'v': 227,\n            'f': \"227\",\n        },\n\"dcc_grusin\",\n\"PRJNA679456\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 7,\n            'f': \"7\",\n        },\n\"2020-11-20 01:44:04\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 in Ecuador\"],\n [{\n            'v': 228,\n            'f': \"228\",\n        },\n\"dcc_grusin\",\n\"PRJNA679460\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 208,\n            'f': \"208\",\n        },\n\"2020-11-20 01:44:09\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Mass molecular testing for COVID19 using NGS-based technology and a highly scalable workflow\"],\n [{\n            'v': 229,\n            'f': \"229\",\n        },\n\"dcc_grusin\",\n\"PRJNA679786\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-11-21 01:41:08\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"In-frame 12 nucleotide deletion within ORF3a in SARS-CoV-2 in a patient hospitalized with COVID-19\"],\n [{\n            'v': 230,\n            'f': \"230\",\n        },\n\"dcc_grusin\",\n\"PRJNA679980\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 6,\n            'f': \"6\",\n        },\n\"2020-12-08 01:48:09\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Raw sequence reads of SARS-CoV-2 from a case of donor derived infection by lung transplantation\"],\n [{\n            'v': 231,\n            'f': \"231\",\n        },\n\"dcc_grusin\",\n\"PRJNA680710\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2020-11-27 01:57:41\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Multiple SARS-CoV-2 Spike Variants Arising at Seroconversion during COVID-19\"],\n [{\n            'v': 232,\n            'f': \"232\",\n        },\n\"dcc_grusin\",\n\"PRJNA681020\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 57,\n            'f': \"57\",\n        },\n\"2021-01-04 14:03:35\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Genome sequencing\"],\n [{\n            'v': 233,\n            'f': \"233\",\n        },\n\"dcc_grusin\",\n\"PRJNA681038\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2020-12-03 01:59:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 genome sequencing\"],\n [{\n            'v': 234,\n            'f': \"234\",\n        },\n\"dcc_grusin\",\n\"PRJNA681192\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-12-01 01:58:41\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 stock sequencing for infection of nonhuman primates\"],\n [{\n            'v': 235,\n            'f': \"235\",\n        },\n\"dcc_grusin\",\n\"PRJNA681234\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 9,\n            'f': \"9\",\n        },\n\"2020-11-30 01:35:50\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Nasopharyngeal swabs from a chronically infected patient with SARS-CoV-2\"],\n [{\n            'v': 236,\n            'f': \"236\",\n        },\n\"dcc_grusin\",\n\"PRJNA681574\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 151,\n            'f': \"151\",\n        },\n\"2020-12-01 01:59:09\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Guidelines for accurate genotyping of SARS-CoV-2 using amplicon-based sequencing of clinical samples\"],\n [{\n            'v': 237,\n            'f': \"237\",\n        },\n\"dcc_grusin\",\n\"PRJNA681684\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2021-01-17 03:36:54\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Camel parainfluenza virus 3 and Middle East Respiratory Syndrome coronavirus Metagenomic assembly\"],\n [{\n            'v': 238,\n            'f': \"238\",\n        },\n\"dcc_grusin\",\n\"PRJNA682013\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 43,\n            'f': \"43\",\n        },\n\"2021-02-09 03:40:47\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Longitudinal SARS-CoV-2 Patient\"],\n [{\n            'v': 239,\n            'f': \"239\",\n        },\n\"dcc_grusin\",\n\"PRJNA682212\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 357,\n            'f': \"357\",\n        },\n\"2021-02-02 03:00:38\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Raw sequence reads of SARS-CoV-2 from hospitalized patients\"],\n [{\n            'v': 240,\n            'f': \"240\",\n        },\n\"dcc_grusin\",\n\"PRJNA682223\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 27,\n            'f': \"27\",\n        },\n\"2021-01-04 14:01:22\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genomic RNA elements drive phase separation of the SARS-CoV-2 nucleocapsid\"],\n [{\n            'v': 241,\n            'f': \"241\",\n        },\n\"dcc_grusin\",\n\"PRJNA682735\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 372,\n            'f': \"372\",\n        },\n\"2021-01-04 14:32:54\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Academic Consortium project on SARS-CoV-2 genome sequencing\"],\n [{\n            'v': 242,\n            'f': \"242\",\n        },\n\"dcc_grusin\",\n\"PRJNA683873\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 57,\n            'f': \"57\",\n        },\n\"2021-01-04 14:05:03\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Genome sequencing\"],\n [{\n            'v': 243,\n            'f': \"243\",\n        },\n\"dcc_faang\",\n\"PRJNA684563\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2021-01-04 14:02:24\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Interdependence between histone marks and steps in Pol II transcription [Antczak.submission]\"],\n [{\n            'v': 244,\n            'f': \"244\",\n        },\n\"dcc_grusin\",\n\"PRJNA685400\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 28,\n            'f': \"28\",\n        },\n\"2021-01-13 03:21:04\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Escape from neutralizing antibodies by SARS-CoV-2 spike protein variants\"],\n [{\n            'v': 245,\n            'f': \"245\",\n        },\n\"dcc_grusin\",\n\"PRJNA686081\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 180,\n            'f': \"180\",\n        },\n\"2021-02-03 02:31:30\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 genome sequencing Rio de Janeiro / Brazil\"],\n [{\n            'v': 246,\n            'f': \"246\",\n        },\n\"dcc_grusin\",\n\"PRJNA686083\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 81,\n            'f': \"81\",\n        },\n\"2021-01-04 14:03:32\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"DeepSpike\"],\n [{\n            'v': 247,\n            'f': \"247\",\n        },\n\"dcc_grusin\",\n\"PRJNA686984\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 179558,\n            'f': \"179558\",\n        },\n\"2023-09-15 02:39:58\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 Genome sequencing and assembly\"],\n [{\n            'v': 248,\n            'f': \"248\",\n        },\n\"dcc_grusin\",\n\"PRJNA687366\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 45,\n            'f': \"45\",\n        },\n\"2021-02-16 03:05:29\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 Whole Genome Sequencing and Analysis\"],\n [{\n            'v': 249,\n            'f': \"249\",\n        },\n\"dcc_grusin\",\n\"PRJNA688005\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 9,\n            'f': \"9\",\n        },\n\"2021-01-30 01:59:52\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"studies of SARS-CoV-2 single nucleotide polymorphisms\"],\n [{\n            'v': 250,\n            'f': \"250\",\n        },\n\"dcc_grusin\",\n\"PRJNA688208\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2021-01-04 14:32:02\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 sequencing\"],\n [{\n            'v': 251,\n            'f': \"251\",\n        },\n\"dcc_grusin\",\n\"PRJNA688999\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2021-01-04 15:08:18\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Whole genome sequencing of SARS-Cov-2 virus strain NITMA1086\"],\n [{\n            'v': 252,\n            'f': \"252\",\n        },\n\"dcc_grusin\",\n\"PRJNA689000\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2021-01-04 15:08:17\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Whole genome sequencing of SARS-Cov-2 virus strain NITMA1139\"],\n [{\n            'v': 253,\n            'f': \"253\",\n        },\n\"dcc_grusin\",\n\"PRJNA689203\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2021-01-21 04:39:15\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"First Whole Genome Sequence of a SARS-CoV-2 Isolate from Iraq\"],\n [{\n            'v': 254,\n            'f': \"254\",\n        },\n\"dcc_grusin\",\n\"PRJNA689621\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 492,\n            'f': \"492\",\n        },\n\"2021-01-27 02:03:47\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 genomic data and descriptive metadata from primary specimens collected in Ontario, Canada and sequenced and/or assembled by the McArthur Laboratory at McMaster University (Hamilton, Ontario, Canada) during the COVID-19 outbreak.\"],\n [{\n            'v': 255,\n            'f': \"255\",\n        },\n\"dcc_grusin\",\n\"PRJNA689713\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2021-01-10 14:11:08\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 related coronaviruses circulating in bats and pangolins in Southeast Asia\"],\n [{\n            'v': 256,\n            'f': \"256\",\n        },\n\"dcc_grusin\",\n\"PRJNA689811\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2021-01-05 22:03:54\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS CoV-2 WGS\"],\n [{\n            'v': 257,\n            'f': \"257\",\n        },\n\"dcc_grusin\",\n\"PRJNA690577\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 143,\n            'f': \"143\",\n        },\n\"2021-01-23 04:03:25\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Structural Variants of SARS-CoV-2 Genome and Transcriptomes\"],\n [{\n            'v': 258,\n            'f': \"258\",\n        },\n\"dcc_grusin\",\n\"PRJNA690890\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 434,\n            'f': \"434\",\n        },\n\"2021-03-15 02:37:27\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ACoRE: Accurate SARS-CoV-2 genome reconstruction for the characterization of intra-host 1 and inter-host viral diversity in clinical samples and for the evaluation of re-infections\"],\n [{\n            'v': 259,\n            'f': \"259\",\n        },\n\"dcc_grusin\",\n\"PRJNA691021\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 6,\n            'f': \"6\",\n        },\n\"2021-10-26 02:42:50\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Whole Genome Sequence of a SARS-CoV-2 Isolate from Dominican Republic\"],\n [{\n            'v': 260,\n            'f': \"260\",\n        },\n\"dcc_grusin\",\n\"PRJNA691537\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2021-01-13 03:22:04\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Sars-CoV-2 genome sequencing\"],\n [{\n            'v': 261,\n            'f': \"261\",\n        },\n\"dcc_grusin\",\n\"PRJNA691556\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 210,\n            'f': \"210\",\n        },\n\"2021-01-13 03:22:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 genome sequencing\"],\n [{\n            'v': 262,\n            'f': \"262\",\n        },\n\"dcc_grusin\",\n\"PRJNA691562\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2021-02-09 03:40:56\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 Antibodies Raw sequence reads\"],\n [{\n            'v': 263,\n            'f': \"263\",\n        },\n\"dcc_grusin\",\n\"PRJNA691606\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2021-03-13 02:51:22\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Raw sequence reads\"],\n [{\n            'v': 264,\n            'f': \"264\",\n        },\n\"dcc_grusin\",\n\"PRJNA691961\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2021-02-02 03:00:56\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Orally delivered MK-4482 inhibits SARS-CoV-2 replication in the Syrian hamster model\"],\n [{\n            'v': 265,\n            'f': \"265\",\n        },\n\"dcc_grusin\",\n\"PRJNA692472\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 198,\n            'f': \"198\",\n        },\n\"2022-10-13 02:22:40\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS CoV-2\"],\n [{\n            'v': 266,\n            'f': \"266\",\n        },\n\"dcc_grusin\",\n\"PRJNA692481\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2021-01-17 03:37:57\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genomic sequence analysis of SARS-COV-2\"],\n [{\n            'v': 267,\n            'f': \"267\",\n        },\n\"dcc_grusin\",\n\"PRJNA692653\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 10,\n            'f': \"10\",\n        },\n\"2021-01-21 04:39:49\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Whole genome sequencing of SARS-CoV-2 from clinical samples for diagnostic kit development\"],\n [{\n            'v': 268,\n            'f': \"268\",\n        },\n\"dcc_grusin\",\n\"PRJNA692972\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2021-01-28 01:40:00\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Complete genome sequence of SARS-CoV-2 strain B.1.1.7\"],\n [{\n            'v': 269,\n            'f': \"269\",\n        },\n\"dcc_grusin\",\n\"PRJNA694014\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 400,\n            'f': \"400\",\n        },\n\"2021-02-04 04:12:54\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Virus sequencing of the 501Y.V2 variant of SARS-CoV-2 by the Network for Genomics Surveillance in South-Africa (NGS-SA)\"],\n [{\n            'v': 270,\n            'f': \"270\",\n        },\n\"dcc_grusin\",\n\"PRJNA694097\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2021-01-29 02:16:54\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Raw sequence reads\"],\n [{\n            'v': 271,\n            'f': \"271\",\n        },\n\"dcc_grusin\",\n\"PRJNA694172\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 92,\n            'f': \"92\",\n        },\n\"2021-02-03 02:31:41\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 genome sequencing Rio Grande do Sul / Brazil, Dec 2020\"],\n [{\n            'v': 272,\n            'f': \"272\",\n        },\n\"dcc_grusin\",\n\"PRJNA694865\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2021-05-02 11:22:38\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Full genome sequencing and phylogenetic analysis of SARS-CoV-2 cases in Iran\"],\n [{\n            'v': 273,\n            'f': \"273\",\n        },\n\"dcc_grusin\",\n\"PRJNA698079\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2021-02-01 02:38:31\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"FAST-TRACK COVID\"],\n [{\n            'v': 274,\n            'f': \"274\",\n        },\n\"dcc_grusin\",\n\"PRJNA698267\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 91,\n            'f': \"91\",\n        },\n\"2021-02-04 04:12:15\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Intra-host Variation and Evolutionary Dynamics of SARS-CoV-2 Populations in COVID-19 Patients\"],\n [{\n            'v': 275,\n            'f': \"275\",\n        },\n\"dcc_grusin\",\n\"PRJNA698337\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 44,\n            'f': \"44\",\n        },\n\"2021-04-27 09:22:42\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-COV-2 viral transcriptome\"],\n [{\n            'v': 276,\n            'f': \"276\",\n        },\n\"dcc_grusin\",\n\"PRJNA698378\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 31,\n            'f': \"31\",\n        },\n\"2021-03-01 10:02:47\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 isolate sequencing\"],\n [{\n            'v': 277,\n            'f': \"277\",\n        },\n\"dcc_grusin\",\n\"PRJNA698478\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 46,\n            'f': \"46\",\n        },\n\"2021-02-03 02:33:17\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 reinfection in a cohort of 43,000 antibody-positive individuals followed for up to 35 weeks\"],\n [{\n            'v': 278,\n            'f': \"278\",\n        },\n\"dcc_grusin\",\n\"PRJNA699251\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2021-02-05 10:09:42\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Two prolonged viremic SARS-CoV-2 infections with conserved viral genome for two months\"],\n [{\n            'v': 279,\n            'f': \"279\",\n        },\n\"dcc_grusin\",\n\"PRJNA700531\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 7,\n            'f': \"7\",\n        },\n\"2021-02-09 05:45:52\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV and SARS-CoV-2 are transmitted through the air between ferrets over more than one meter distance\"],\n [{\n            'v': 280,\n            'f': \"280\",\n        },\n\"dcc_grusin\",\n\"PRJNA701254\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2021-02-11 04:44:44\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 genome Sequencing\"],\n [{\n            'v': 281,\n            'f': \"281\",\n        },\n\"dcc_grusin\",\n\"PRJNA701669\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2389,\n            'f': \"2389\",\n        },\n\"2023-01-15 01:43:42\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 WGS (Ohio Department of Health)\"],\n [{\n            'v': 282,\n            'f': \"282\",\n        },\n\"dcc_grusin\",\n\"PRJNA701790\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2021-02-17 02:56:58\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Demystifying SARS-CoV-2 from samples of COVID-19 patients of Chattogram, Bangladesh through whole genome sequencing\"],\n [{\n            'v': 283,\n            'f': \"283\",\n        },\n\"dcc_grusin\",\n\"PRJNA701872\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2021-02-17 02:57:44\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 genomic data and descriptive metadata collected during the COVID-19 outbreak in Dhaka, Bangladesh\"],\n [{\n            'v': 284,\n            'f': \"284\",\n        },\n\"dcc_grusin\",\n\"PRJNA702998\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2021-03-01 10:05:56\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genome Sequence of SARS CoV-2 variant B.1.1.7 501Y.V1 from Dhaka, Bangladesh\"],\n [{\n            'v': 285,\n            'f': \"285\",\n        },\n\"dcc_grusin\",\n\"PRJNA704235\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2021-03-01 10:08:31\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Brasilian P1 SARS-CoV-2 strain\"],\n [{\n            'v': 286,\n            'f': \"286\",\n        },\n\"dcc_grusin\",\n\"PRJNA704391\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 575,\n            'f': \"575\",\n        },\n\"2021-05-23 17:23:33\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 Genome sequencing\"],\n [{\n            'v': 287,\n            'f': \"287\",\n        },\n\"dcc_grusin\",\n\"PRJNA704499\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2021-03-05 03:14:03\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 genome sequencing\"],\n [{\n            'v': 288,\n            'f': \"288\",\n        },\n\"dcc_grusin\",\n\"PRJNA704627\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2021-05-18 15:04:48\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Structural basis of ribosomal frameshifting during translation of the SARS-CoV-2 RNA genome\"],\n [{\n            'v': 289,\n            'f': \"289\",\n        },\n\"dcc_grusin\",\n\"PRJNA707211\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 58,\n            'f': \"58\",\n        },\n\"2021-03-07 03:16:59\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Tiled-ClickSeq for targeted sequencing of Coronavirus genomes\"],\n [{\n            'v': 290,\n            'f': \"290\",\n        },\n\"dcc_grusin\",\n\"PRJNA707370\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 969,\n            'f': \"969\",\n        },\n\"2021-03-19 06:04:01\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Sars-CoV-2 Sequencing in Qatar\"],\n [{\n            'v': 291,\n            'f': \"291\",\n        },\n\"dcc_grusin\",\n\"PRJNA707397\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2021-03-12 03:12:29\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Whole Genome Sequencing of SARS-CoV-2\"],\n [{\n            'v': 292,\n            'f': \"292\",\n        },\n\"dcc_grusin\",\n\"PRJNA707404\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 16,\n            'f': \"16\",\n        },\n\"2021-04-23 01:17:18\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"A versatile reverse genetics platform for pandemic SARS-CoV-2 and other positive-strand RNA viruses.\"],\n [{\n            'v': 293,\n            'f': \"293\",\n        },\n\"dcc_grusin\",\n\"PRJNA707536\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2256,\n            'f': \"2256\",\n        },\n\"2023-06-12 02:13:52\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 WGS South Dakota Public Health Laboratory\"],\n [{\n            'v': 294,\n            'f': \"294\",\n        },\n\"dcc_grusin\",\n\"PRJNA707583\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 21,\n            'f': \"21\",\n        },\n\"2021-03-12 03:12:41\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genomic Epidemiology of SARS-CoV-2 in Esteio, Rio Grande do Sul, Brazil\"],\n [{\n            'v': 295,\n            'f': \"295\",\n        },\n\"dcc_grusin\",\n\"PRJNA708134\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 39,\n            'f': \"39\",\n        },\n\"2021-03-13 02:51:59\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Detection and genome sequencing of SARS-CoV-2 belonging to the B.1.1.7 lineage in the Philippines\"],\n [{\n            'v': 296,\n            'f': \"296\",\n        },\n\"dcc_grusin\",\n\"PRJNA713804\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 17549,\n            'f': \"17549\",\n        },\n\"2023-04-29 05:32:14\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 Surveillance Sequencing\"],\n [{\n            'v': 297,\n            'f': \"297\",\n        },\n\"dcc_grusin\",\n\"PRJNA713811\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2021-03-13 02:53:41\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Raw sequence reads\"],\n [{\n            'v': 298,\n            'f': \"298\",\n        },\n\"dcc_grusin\",\n\"PRJNA713860\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2021-03-13 02:53:43\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 Sequencing at Shenandoah University\"],\n [{\n            'v': 299,\n            'f': \"299\",\n        },\n\"dcc_grusin\",\n\"PRJNA714677\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 789,\n            'f': \"789\",\n        },\n\"2023-07-30 05:12:34\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"High-throughput surveys of SARS-related coronavirus RBDs using yeast display of barcoded libraries\"],\n [{\n            'v': 300,\n            'f': \"300\",\n        },\n\"dcc_grusin\",\n\"PRJNA714680\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 44,\n            'f': \"44\",\n        },\n\"2021-05-23 17:10:37\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 WGS: Los Alamos National Laboratory, Biosecurity and Public Health\"],\n [{\n            'v': 301,\n            'f': \"301\",\n        },\n\"dcc_grusin\",\n\"PRJNA714692\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 41,\n            'f': \"41\",\n        },\n\"2021-07-04 03:08:01\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 WGS: ISB and Swedish Medical Center\"],\n [{\n            'v': 302,\n            'f': \"302\",\n        },\n\"dcc_grusin\",\n\"PRJNA714744\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8,\n            'f': \"8\",\n        },\n\"2021-03-19 06:05:41\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 WGS: Oregon State Public Health Laboratory\"],\n [{\n            'v': 303,\n            'f': \"303\",\n        },\n\"dcc_grusin\",\n\"PRJNA715749\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 179550,\n            'f': \"179550\",\n        },\n\"2023-03-05 11:51:59\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Genome sequencing\"],\n [{\n            'v': 304,\n            'f': \"304\",\n        },\n\"dcc_grusin\",\n\"PRJNA716271\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 26,\n            'f': \"26\",\n        },\n\"2021-03-30 03:52:35\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Sequencing of SARS-CoV-2 Isolates from BEI Resources\"],\n [{\n            'v': 305,\n            'f': \"305\",\n        },\n\"dcc_grusin\",\n\"PRJNA716985\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 353273,\n            'f': \"353273\",\n        },\n\"2023-10-22 11:32:13\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 Genome sequencing and assembly - QUEST\"],\n [{\n            'v': 306,\n            'f': \"306\",\n        },\n\"dcc_grusin\",\n\"PRJNA717113\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1871,\n            'f': \"1871\",\n        },\n\"2023-02-27 21:15:32\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Raw sequence reads\"],\n [{\n            'v': 307,\n            'f': \"307\",\n        },\n\"dcc_grusin\",\n\"PRJNA717314\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 466,\n            'f': \"466\",\n        },\n\"2023-04-14 06:04:54\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 sequencing\"],\n [{\n            'v': 308,\n            'f': \"308\",\n        },\n\"dcc_grusin\",\n\"PRJNA717974\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 22,\n            'f': \"22\",\n        },\n\"2021-04-24 08:57:52\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Molecular Evidence of SARS-CoV-2 in New York Before the First Pandemic Wave\"],\n [{\n            'v': 309,\n            'f': \"309\",\n        },\n\"dcc_grusin\",\n\"PRJNA718073\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 20,\n            'f': \"20\",\n        },\n\"2021-04-01 03:57:23\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 GENOME SEQUECING ANALYSIS IN MINAS GERAIS, BRAZIL\"],\n [{\n            'v': 310,\n            'f': \"310\",\n        },\n\"dcc_grusin\",\n\"PRJNA718231\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 326,\n            'f': \"326\",\n        },\n\"2022-05-26 04:10:22\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 WGS\"],\n [{\n            'v': 311,\n            'f': \"311\",\n        },\n\"dcc_grusin\",\n\"PRJNA718341\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 300,\n            'f': \"300\",\n        },\n\"2021-04-05 05:36:04\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Wisconsin SARS-CoV-2 within host\"],\n [{\n            'v': 312,\n            'f': \"312\",\n        },\n\"dcc_grusin\",\n\"PRJNA718451\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1058,\n            'f': \"1058\",\n        },\n\"2022-05-08 04:38:04\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"KRISP Sequencing of SARS-CoV-2 in Mozambique\"],\n [{\n            'v': 313,\n            'f': \"313\",\n        },\n\"dcc_grusin\",\n\"PRJNA718818\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2021-04-23 01:33:39\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-coronavirus-2 expansion in Vero cells\"],\n [{\n            'v': 314,\n            'f': \"314\",\n        },\n\"dcc_grusin\",\n\"PRJNA719089\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2021-05-07 12:22:48\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Raw sequence reads\"],\n [{\n            'v': 315,\n            'f': \"315\",\n        },\n\"dcc_grusin\",\n\"PRJNA720050\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 196089,\n            'f': \"196089\",\n        },\n\"2023-10-22 11:31:29\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Sars-cov-2 samples from helix\"],\n [{\n            'v': 316,\n            'f': \"316\",\n        },\n\"dcc_grusin\",\n\"PRJNA720240\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2021-04-13 15:36:51\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Complete Genome Sequences of SARS-CoV-2 isolated from Egyptian Patients\"],\n [{\n            'v': 317,\n            'f': \"317\",\n        },\n\"dcc_grusin\",\n\"PRJNA720544\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 194,\n            'f': \"194\",\n        },\n\"2022-11-30 02:03:20\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 genome sequencing and assembly from COVID-19 autopsy tissue specimens\"],\n [{\n            'v': 318,\n            'f': \"318\",\n        },\n\"dcc_grusin\",\n\"PRJNA720687\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 116,\n            'f': \"116\",\n        },\n\"2021-04-23 01:32:38\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Wastewater study of severe acute respiratory syndrome coronavirus 2 in Newport Oregon\"],\n [{\n            'v': 319,\n            'f': \"319\",\n        },\n\"dcc_grusin\",\n\"PRJNA721287\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 25,\n            'f': \"25\",\n        },\n\"2021-04-13 16:08:38\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genomic Variation of Propagated SARS-2-CoV\"],\n [{\n            'v': 320,\n            'f': \"320\",\n        },\n\"dcc_grusin\",\n\"PRJNA721582\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2021-04-13 16:09:11\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2/human/Marburg/PUM-D2417/2021\"],\n [{\n            'v': 321,\n            'f': \"321\",\n        },\n\"dcc_grusin\",\n\"PRJNA721724\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2021-04-15 18:49:49\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 sequences from COVID19 patients in New York\"],\n [{\n            'v': 322,\n            'f': \"322\",\n        },\n\"dcc_grusin\",\n\"PRJNA722318\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2021-05-13 19:52:11\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Live Virus Neutralisation of the 501Y.V1 and 501Y.V2 SARS-CoV-2 Variants following INO-4800 Vaccination of Ferrets\"],\n [{\n            'v': 323,\n            'f': \"323\",\n        },\n\"dcc_grusin\",\n\"PRJNA722947\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 39,\n            'f': \"39\",\n        },\n\"2021-05-07 12:15:27\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Raw sequence reads\"],\n [{\n            'v': 324,\n            'f': \"324\",\n        },\n\"dcc_grusin\",\n\"PRJNA723329\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2021-04-23 01:34:46\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"EPICC SARS-CoV-2 Sequencing\"],\n [{\n            'v': 325,\n            'f': \"325\",\n        },\n\"dcc_grusin\",\n\"PRJNA723604\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 16,\n            'f': \"16\",\n        },\n\"2021-04-30 10:05:56\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 Capture Sequencing: Baylor College of Medicine\"],\n [{\n            'v': 326,\n            'f': \"326\",\n        },\n\"dcc_grusin\",\n\"PRJNA723901\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 118,\n            'f': \"118\",\n        },\n\"2021-04-24 09:01:20\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Whole genome sequencing of SARS-CoV-2 isolates from culture\"],\n [{\n            'v': 327,\n            'f': \"327\",\n        },\n\"dcc_grusin\",\n\"PRJNA725674\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1253,\n            'f': \"1253\",\n        },\n\"2021-06-08 07:54:33\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"KRISP SARS-CoV-2 Sequencing\"],\n [{\n            'v': 328,\n            'f': \"328\",\n        },\n\"dcc_grusin\",\n\"PRJNA726258\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2021-04-30 10:07:30\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 Sequencing\"],\n [{\n            'v': 329,\n            'f': \"329\",\n        },\n\"dcc_grusin\",\n\"PRJNA726840\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 11,\n            'f': \"11\",\n        },\n\"2021-05-12 12:45:10\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Coding-Complete Genome Sequences of 11 SARS-CoV-2 B.1.1.7 and B.1.351 variants from Metro Manila, Philippines\"],\n [{\n            'v': 330,\n            'f': \"330\",\n        },\n\"dcc_grusin\",\n\"PRJNA726871\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2021-05-04 12:20:28\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"The first confirmed SARS-CoV-2 B.1.617.2 variant sequence from Romania\"],\n [{\n            'v': 331,\n            'f': \"331\",\n        },\n\"dcc_grusin\",\n\"PRJNA726943\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2021-06-26 22:33:01\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 Genome Sequencing\"],\n [{\n            'v': 332,\n            'f': \"332\",\n        },\n\"dcc_grusin\",\n\"PRJNA727469\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2021-05-15 14:41:01\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genomic Sequencing of SARS-CoV-2\"],\n [{\n            'v': 333,\n            'f': \"333\",\n        },\n\"dcc_grusin\",\n\"PRJNA727735\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 661,\n            'f': \"661\",\n        },\n\"2022-04-01 17:59:26\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"KRISP Sequencing of SARS-CoV-2 in Zimbabwe\"],\n [{\n            'v': 334,\n            'f': \"334\",\n        },\n\"dcc_grusin\",\n\"PRJNA727811\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 31,\n            'f': \"31\",\n        },\n\"2021-05-09 12:03:19\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 raw sequence reads of emerging lineage C.37 in Peru\"],\n [{\n            'v': 335,\n            'f': \"335\",\n        },\n\"dcc_grusin\",\n\"PRJNA728463\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 813,\n            'f': \"813\",\n        },\n\"2021-05-10 12:44:58\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Evidence for increased breakthrough rates of SARS-CoV-2 variants of concern in BNT162b2 mRNA vaccinated individuals\"],\n [{\n            'v': 336,\n            'f': \"336\",\n        },\n\"dcc_grusin\",\n\"PRJNA728523\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 80,\n            'f': \"80\",\n        },\n\"2022-03-04 02:36:07\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"raw data of cut efficiency\"],\n [{\n            'v': 337,\n            'f': \"337\",\n        },\n\"dcc_grusin\",\n\"PRJNA729484\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 11,\n            'f': \"11\",\n        },\n\"2021-05-15 14:47:51\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Short Report: Early genomic detection of SARS-CoV-2 P.1 variant in Northeast Brazil\"],\n [{\n            'v': 338,\n            'f': \"338\",\n        },\n\"dcc_grusin\",\n\"PRJNA729878\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 11,\n            'f': \"11\",\n        },\n\"2021-05-16 14:07:50\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Short Plus Long-Amplicon Based Sequencing Approach Improves Genomic Coverage And Variant Detection In SARS-CoV-2 Genome\"],\n [{\n            'v': 339,\n            'f': \"339\",\n        },\n\"dcc_grusin\",\n\"PRJNA730334\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 312,\n            'f': \"312\",\n        },\n\"2021-05-18 16:00:43\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Tiled amplicon nanopore sequencing of SARS-CoV-2\"],\n [{\n            'v': 340,\n            'f': \"340\",\n        },\n\"dcc_grusin\",\n\"PRJNA731979\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 21,\n            'f': \"21\",\n        },\n\"2021-05-23 17:10:55\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 Genome Sequencing from Iraq\"],\n [{\n            'v': 341,\n            'f': \"341\",\n        },\n\"dcc_basie\",\n\"PRJNA733875\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1011,\n            'f': \"1011\",\n        },\n\"2021-06-08 07:52:37\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"POHEM\"],\n [{\n            'v': 342,\n            'f': \"342\",\n        },\n\"dcc_faang\",\n\"PRJNA762083\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 27,\n            'f': \"27\",\n        },\n\"2021-09-17 01:37:20\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Pig genome functional annotation RRBS datasets\"],\n [{\n            'v': 343,\n            'f': \"343\",\n        },\n\"dcc_grusin\",\n\"PRJDB11029\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 14,\n            'f': \"14\",\n        },\n\"2021-05-09 11:27:51\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 on asymptomatic carriers\"],\n [{\n            'v': 344,\n            'f': \"344\",\n        },\n\"dcc_grusin\",\n\"PRJDB11220\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2021-03-19 04:15:02\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Escape mutant screening of SARS-CoV-2 using two neutralizing antibody clones.\"],\n [{\n            'v': 345,\n            'f': \"345\",\n        },\n\"dcc_schubert\",\n\"PRJDB7087\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 247,\n            'f': \"247\",\n        },\n\"2018-08-15 02:16:16\",\n{\n            'v': 247,\n            'f': \"247\",\n        },\n\"2019-05-07 17:10:01\",\n\"A project of associating bacterial WGS and AST data\"],\n [{\n            'v': 346,\n            'f': \"346\",\n        },\n\"dcc_clark\",\n\"PRJEB10265\",\n\"Webin-40757\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 57,\n            'f': \"57\",\n        },\n\"2015-08-19 10:21:18\",\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2015-08-21 13:12:21\",\n\"Ebola virus sequencing Dutch Mobile Laboratories\"],\n [{\n            'v': 347,\n            'f': \"347\",\n        },\n\"dcc_clark\",\n\"PRJEB10571\",\n\"Webin-195\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 169,\n            'f': \"169\",\n        },\n\"2016-02-03 12:18:54\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"We present the complete sequences of 142 Ebola isolates sequenced in 148 runs in Guinea using a portable genomic surveillance laboratory based around the MinION nanopore sequencer. The small size of the instrument and minimal infrastructure required, meant the entire system could be transported into country as personal airline luggage. The equipment added sequencing capacity to an existing Ebola diagnostic laboratory and provided real-time genomic surveillance of new cases from March to October 2015.\"],\n [{\n            'v': 348,\n            'f': \"348\",\n        },\n\"dcc_marc\",\n\"PRJEB11007\",\n\"Webin-30\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 1660,\n            'f': \"1660\",\n        },\n\"2015-10-08 10:26:55\",\n\"Analysis study for MARC phase1\"],\n [{\n            'v': 349,\n            'f': \"349\",\n        },\n\"dcc_beethoven\",\n\"PRJEB11174\",\n\"Webin-509\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 291,\n            'f': \"291\",\n        },\n\"2015-09-30 19:31:16\",\n{\n            'v': 857,\n            'f': \"857\",\n        },\n\"2018-08-20 23:20:13\",\n\"Global genomic epidemiology of Salmonella Typhimurium DT104\"],\n [{\n            'v': 350,\n            'f': \"350\",\n        },\n\"dcc_basie\",\n\"PRJEB11474\",\n\"Webin-40224\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8,\n            'f': \"8\",\n        },\n\"2015-11-09 17:54:00\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Integrated \\u2018omics\\u2019, targeted metabolite and single-cell analyses of Arctic snow algae functionality and adaptability\"],\n [{\n            'v': 351,\n            'f': \"351\",\n        },\n\"dcc_dvorak\",\n\"PRJEB11543\",\n\"Webin-41866\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 520,\n            'f': \"520\",\n        },\n\"2015-10-28 23:13:35\",\n{\n            'v': 2060,\n            'f': \"2060\",\n        },\n\"2019-01-06 12:38:38\",\n\"Prospective whole genome sequencing for national surveillance of Listeria monocytogenes in Australia\"],\n [{\n            'v': 352,\n            'f': \"352\",\n        },\n\"dcc_marc\",\n\"PRJEB11690\",\n\"Webin-40359\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2015-11-06 14:47:26\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"P2a-Lab4-R1\"],\n [{\n            'v': 353,\n            'f': \"353\",\n        },\n\"dcc_marc\",\n\"PRJEB11691\",\n\"Webin-40359\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2015-11-06 14:47:51\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"P2a-Lab4-R2\"],\n [{\n            'v': 354,\n            'f': \"354\",\n        },\n\"dcc_marc\",\n\"PRJEB11693\",\n\"Webin-40352\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2015-11-06 14:55:16\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"MARC phase2a\"],\n [{\n            'v': 355,\n            'f': \"355\",\n        },\n\"dcc_marc\",\n\"PRJEB11703\",\n\"Webin-41903\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2015-11-09 10:03:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Aiming for ultra-long read lengths in Drosophila melanogaster to facilitate genome assembly and structural variation studies and tool development.\\n\\nIf you use this data, please cite: \\nSequencing ultra-long DNA molecules with the Oxford Nanopore MinION\\nJohn M Urban, Jacob Bliss, Charles E Lawrence, Susan A Gerbi\\ndoi: https://doi.org/10.1101/019281\\n\\nhttps://www.biorxiv.org/content/early/2015/05/13/019281\"],\n [{\n            'v': 356,\n            'f': \"356\",\n        },\n\"dcc_marc\",\n\"PRJEB11818\",\n\"Webin-42081\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 85,\n            'f': \"85\",\n        },\n\"2016-07-05 23:19:02\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ONT MinION sequencing of ERCC and HEK293 cDNA\"],\n [{\n            'v': 357,\n            'f': \"357\",\n        },\n\"dcc_verdi\",\n\"PRJEB11951\",\n\"Webin-24\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1205,\n            'f': \"1205\",\n        },\n\"2015-11-30 14:48:35\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Mouse embryonic RNA-seq\"],\n [{\n            'v': 358,\n            'f': \"358\",\n        },\n\"dcc_grusin\",\n\"PRJEB12126\",\n\"Webin-24\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 40,\n            'f': \"40\",\n        },\n\"2015-12-14 11:50:38\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Analysis of coronavirus and infected host-cell gene expression through RNA sequencing and ribosome profiling\"],\n [{\n            'v': 359,\n            'f': \"359\",\n        },\n\"dcc_marc\",\n\"PRJEB12439\",\n\"Webin-40031\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2016-01-19 16:07:43\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"P2a-Lab4-R3\"],\n [{\n            'v': 360,\n            'f': \"360\",\n        },\n\"dcc_marc\",\n\"PRJEB12440\",\n\"Webin-40031\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2016-01-19 16:09:20\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"P2a-Lab4-R3b\"],\n [{\n            'v': 361,\n            'f': \"361\",\n        },\n\"dcc_marc\",\n\"PRJEB12441\",\n\"Webin-40031\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2016-01-19 16:10:42\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"P2a-Lab4-R4\"],\n [{\n            'v': 362,\n            'f': \"362\",\n        },\n\"dcc_marc\",\n\"PRJEB12442\",\n\"Webin-40031\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2016-01-19 16:11:47\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"P2a-Lab4-R4b\"],\n [{\n            'v': 363,\n            'f': \"363\",\n        },\n\"dcc_sibelius\",\n\"PRJEB12582\",\n\"Webin-40759\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 15,\n            'f': \"15\",\n        },\n\"2018-12-13 13:42:03\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"COMPARE H5N8 Pilot Study\"],\n [{\n            'v': 364,\n            'f': \"364\",\n        },\n\"dcc_clark\",\n\"PRJEB12643\",\n\"Webin-39852\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2016-02-29 15:37:47\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNA/cDNA Hybrid RNA Virus Sequencing\"],\n [{\n            'v': 365,\n            'f': \"365\",\n        },\n\"dcc_marc\",\n\"PRJEB12684\",\n\"Webin-40352\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2016-02-09 21:42:57\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"MARC P2a-Lab5-R3\"],\n [{\n            'v': 366,\n            'f': \"366\",\n        },\n\"dcc_marc\",\n\"PRJEB12685\",\n\"Webin-40352\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2016-02-09 21:51:46\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"MARC P2a-Lab5-R4\"],\n [{\n            'v': 367,\n            'f': \"367\",\n        },\n\"dcc_wagner\",\n\"PRJEB12891\",\n\"Webin-42788\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 169,\n            'f': \"169\",\n        },\n\"2016-03-01 17:47:08\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"J-Line Chicken 21 Tissue RNA Expression\"],\n [{\n            'v': 368,\n            'f': \"368\",\n        },\n\"dcc_handel\",\n\"PRJEB13141\",\n\"Webin-43373\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2016-12-23 12:25:29\",\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2018-08-20 15:00:06\",\n\"RIVM protocol for nucleic acid processing and  non targeted NGS approach using NoV positive faeces samples\"],\n [{\n            'v': 369,\n            'f': \"369\",\n        },\n\"dcc_marc\",\n\"PRJEB13500\",\n\"Webin-43379\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2016-04-14 12:03:55\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"MARC P2a-Lab15\"],\n [{\n            'v': 370,\n            'f': \"370\",\n        },\n\"dcc_bromhead\",\n\"PRJEB13576\",\n\"Webin-43578\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2016-04-20 13:25:26\",\n{\n            'v': 6,\n            'f': \"6\",\n        },\n\"2019-01-06 12:38:50\",\n\"Prevalence of mcr-1 in Escherichia coli isolated from Great British pigs\"],\n [{\n            'v': 371,\n            'f': \"371\",\n        },\n\"dcc_phe\",\n\"PRJEB13587\",\n\"Webin-40432\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Collection of data for ESGLI WGS typing design group\"],\n [{\n            'v': 372,\n            'f': \"372\",\n        },\n\"dcc_beethoven\",\n\"PRJEB13610\",\n\"Webin-509\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 25,\n            'f': \"25\",\n        },\n\"2016-04-19 12:28:51\",\n{\n            'v': 66,\n            'f': \"66\",\n        },\n\"2018-07-16 19:00:09\",\n\"Investigating Salmonella Eko from various sources in Nigeria by whole genome sequencing to identify the source of human infections\"],\n [{\n            'v': 373,\n            'f': \"373\",\n        },\n\"dcc_phe\",\n\"PRJEB13666\",\n\"Webin-42890\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Collection of data for ESGLI WGS typing design\"],\n [{\n            'v': 374,\n            'f': \"374\",\n        },\n\"dcc_handel\",\n\"PRJEB13823\",\n\"Webin-40759\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2017-12-12 13:35:16\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"COMPARE Pilot study NoV\"],\n [{\n            'v': 375,\n            'f': \"375\",\n        },\n\"dcc_strauss\",\n\"PRJEB13833\",\n\"Webin-37120\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 46,\n            'f': \"46\",\n        },\n\"2019-09-02 08:25:04\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Surveillance of infectious diseases and antimicrobial resistance from sewage in Kibera (Nairobi, Kenya)\"],\n [{\n            'v': 376,\n            'f': \"376\",\n        },\n\"dcc_bromhead\",\n\"PRJEB13885\",\n\"Webin-37120\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 6,\n            'f': \"6\",\n        },\n\"2016-05-06 08:39:04\",\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2019-01-06 12:39:20\",\n\"Detection of mcr-1 encoding plasmid-mediated colistin-resistant Escherichia coli isolates from human bloodstream infection and imported chicken meat, Denmark 2015\"],\n [{\n            'v': 377,\n            'f': \"377\",\n        },\n\"dcc_basie\",\n\"PRJEB14001\",\n\"Webin-40224\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2016-05-25 10:01:33\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Linking microbial diversity and functionality of Arctic glacial surface habitats\"],\n [{\n            'v': 378,\n            'f': \"378\",\n        },\n\"dcc_handel\",\n\"PRJEB14042\",\n\"Webin-43561\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2017-02-09 12:30:14\",\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2018-11-29 10:45:05\",\n\"Ring trial stool sample analysis in Compare project\"],\n [{\n            'v': 379,\n            'f': \"379\",\n        },\n\"dcc_schumann\",\n\"PRJEB14042\",\n\"Webin-43561\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2017-02-09 12:30:14\",\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2018-11-29 10:45:05\",\n\"Ring trial stool sample analysis in Compare project\"],\n [{\n            'v': 380,\n            'f': \"380\",\n        },\n\"dcc_bromhead\",\n\"PRJEB14086\",\n\"Webin-37120\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 54,\n            'f': \"54\",\n        },\n\"2016-05-18 11:50:00\",\n{\n            'v': 149,\n            'f': \"149\",\n        },\n\"2019-01-06 12:42:23\",\n\"reference testing for ENGAGE E.coli (P160095)\"],\n [{\n            'v': 381,\n            'f': \"381\",\n        },\n\"dcc_schubert\",\n\"PRJEB14086\",\n\"Webin-37120\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 54,\n            'f': \"54\",\n        },\n\"2016-05-18 11:50:00\",\n{\n            'v': 149,\n            'f': \"149\",\n        },\n\"2019-01-06 12:42:23\",\n\"reference testing for ENGAGE E.coli (P160095)\"],\n [{\n            'v': 382,\n            'f': \"382\",\n        },\n\"dcc_marc\",\n\"PRJEB14105\",\n\"Webin-36089\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2016-05-19 15:56:41\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"\\u03c629 Treatment for MinION\"],\n [{\n            'v': 383,\n            'f': \"383\",\n        },\n\"dcc_faang\",\n\"PRJEB14330\",\n\"Webin-43198\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 433,\n            'f': \"433\",\n        },\n\"2020-02-14 17:23:39\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genome-wide Functional Annotation of Regulatory Elements in Livestock Species\"],\n [{\n            'v': 384,\n            'f': \"384\",\n        },\n\"dcc_handel\",\n\"PRJEB14393\",\n\"Webin-14569\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Testing human and food samples for Noroviruses\"],\n [{\n            'v': 385,\n            'f': \"385\",\n        },\n\"dcc_dvorak\",\n\"PRJEB14476\",\n\"Webin-44097\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 224,\n            'f': \"224\",\n        },\n\"2016-09-01 12:49:44\",\n{\n            'v': 895,\n            'f': \"895\",\n        },\n\"2019-01-06 13:13:33\",\n\"Danish L. monocytogenes cases years 2002-14\"],\n [{\n            'v': 386,\n            'f': \"386\",\n        },\n\"dcc_bromhead\",\n\"PRJEB14641\",\n\"Webin-37120\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 142,\n            'f': \"142\",\n        },\n\"2016-06-29 16:28:28\",\n{\n            'v': 308,\n            'f': \"308\",\n        },\n\"2019-01-06 12:50:35\",\n\"Reference testing - FVST, DTU, ENGAGE (E.coli, 2014-2015)\"],\n [{\n            'v': 387,\n            'f': \"387\",\n        },\n\"dcc_schubert\",\n\"PRJEB14641\",\n\"Webin-37120\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 142,\n            'f': \"142\",\n        },\n\"2016-06-29 16:28:28\",\n{\n            'v': 308,\n            'f': \"308\",\n        },\n\"2019-01-06 12:50:35\",\n\"Reference testing - FVST, DTU, ENGAGE (E.coli, 2014-2015)\"],\n [{\n            'v': 388,\n            'f': \"388\",\n        },\n\"dcc_vivaldi\",\n\"PRJEB14853\",\n\"Webin-44220\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 366,\n            'f': \"366\",\n        },\n\"2016-07-19 14:42:09\",\n{\n            'v': 507,\n            'f': \"507\",\n        },\n\"2019-01-06 12:52:00\",\n\"Investigating outbreaks of Salmonella enterica serovar Typhimurium and its monophasic variants in Denmark by whole-genome sequencing; getting ready for real-time surveillance\"],\n [{\n            'v': 389,\n            'f': \"389\",\n        },\n\"dcc_schubert\",\n\"PRJEB14981\",\n\"Webin-37120\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2016-08-01 16:14:58\",\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2016-12-15 16:12:03\",\n\"Extremely Drug-Resistant Salmonella enterica Serovar Senftenberg Infections in Patients in Zambia\"],\n [{\n            'v': 390,\n            'f': \"390\",\n        },\n\"dcc_marc\",\n\"PRJEB15077\",\n\"Webin-36089\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2016-11-14 14:48:24\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Global Rivers\"],\n [{\n            'v': 391,\n            'f': \"391\",\n        },\n\"dcc_vivaldi\",\n\"PRJEB15081\",\n\"Webin-43316\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2016-08-24 15:38:36\",\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2018-07-16 21:36:21\",\n\"Salmonella enterica serovar Enteritidis originating from returning travellers and locally acquired human infections in Europe.\"],\n [{\n            'v': 392,\n            'f': \"392\",\n        },\n\"dcc_puccini\",\n\"PRJEB15112\",\n\"Webin-44537\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 48,\n            'f': \"48\",\n        },\n\"2018-11-12 18:37:17\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Comparative Genomics of Intestinal Protozoa\"],\n [{\n            'v': 393,\n            'f': \"393\",\n        },\n\"dcc_vivaldi\",\n\"PRJEB15201\",\n\"Webin-37120\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 186,\n            'f': \"186\",\n        },\n\"2016-08-19 15:07:46\",\n{\n            'v': 573,\n            'f': \"573\",\n        },\n\"2019-01-06 12:53:19\",\n\"Salmonella for source attribution study. German samples.\"],\n [{\n            'v': 394,\n            'f': \"394\",\n        },\n\"dcc_marc\",\n\"PRJEB15431\",\n\"Webin-154\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 10,\n            'f': \"10\",\n        },\n\"2017-07-04 12:47:53\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Control Sequencing with E Coli for MinION Marc Consortium\"],\n [{\n            'v': 395,\n            'f': \"395\",\n        },\n\"dcc_marc\",\n\"PRJEB15636\",\n\"Webin-39878\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 16,\n            'f': \"16\",\n        },\n\"2017-07-04 12:47:53\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"UBC MARC E.coli Oxford Nanopore MinION R9 2D bridging experiment data\"],\n [{\n            'v': 396,\n            'f': \"396\",\n        },\n\"dcc_allison\",\n\"PRJEB15708\",\n\"Webin-41866\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 78,\n            'f': \"78\",\n        },\n\"2017-02-24 09:59:45\",\n{\n            'v': 312,\n            'f': \"312\",\n        },\n\"2018-12-31 12:56:15\",\n\"Emergence of ESBL-producing Salmonella Typhimurium in Australia due to a new clade harbouring blaCTX-M-9\"],\n [{\n            'v': 397,\n            'f': \"397\",\n        },\n\"dcc_schubert\",\n\"PRJEB16326\",\n\"Webin-43316\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1231,\n            'f': \"1231\",\n        },\n\"2018-10-19 07:57:30\",\n{\n            'v': 2936,\n            'f': \"2936\",\n        },\n\"2019-01-06 14:05:52\",\n\"This study will include priority foodborne pathogens and investigate different methods, based on NGS data, of analysing and clustering these for the purpose of improved surveillance.\"],\n [{\n            'v': 398,\n            'f': \"398\",\n        },\n\"dcc_vivaldi\",\n\"PRJEB16326\",\n\"Webin-43316\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1231,\n            'f': \"1231\",\n        },\n\"2018-10-19 07:57:30\",\n{\n            'v': 2936,\n            'f': \"2936\",\n        },\n\"2019-01-06 14:05:52\",\n\"This study will include priority foodborne pathogens and investigate different methods, based on NGS data, of analysing and clustering these for the purpose of improved surveillance.\"],\n [{\n            'v': 399,\n            'f': \"399\",\n        },\n\"dcc_basie\",\n\"PRJEB17781\",\n\"Webin-45293\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 252,\n            'f': \"252\",\n        },\n\"2016-11-11 13:59:57\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Sedimentary ancient DNA over a period of last ~14,500 years\"],\n [{\n            'v': 400,\n            'f': \"400\",\n        },\n\"dcc_mozart\",\n\"PRJEB1788\",\n\"Webin-9\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 85,\n            'f': \"85\",\n        },\n\"2014-09-05 08:47:32\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Shotgun Sequencing of Tara Oceans DNA samples corresponding to size fractions for  large DNA viruses\"],\n [{\n            'v': 401,\n            'f': \"401\",\n        },\n\"dcc_schubert\",\n\"PRJEB18042\",\n\"Webin-45346\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 55,\n            'f': \"55\",\n        },\n\"2020-05-01 13:41:49\",\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2016-12-15 15:58:58\",\n\"can metagenomics emulate culture for diagnosing urinary tract infections?\"],\n [{\n            'v': 402,\n            'f': \"402\",\n        },\n\"dcc_brahms\",\n\"PRJEB18065\",\n\"Webin-40759\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 34,\n            'f': \"34\",\n        },\n\"2018-12-18 15:06:06\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"COMPARE - Diagnostic metagenomics on clinical samples: Prediction of antibiotic resistance genes and pathogen discovery in shot-gun metagenomic data from swine feces\"],\n [{\n            'v': 403,\n            'f': \"403\",\n        },\n\"dcc_vivaldi\",\n\"PRJEB18442\",\n\"Webin-40760\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 329,\n            'f': \"329\",\n        },\n\"2017-11-30 16:42:07\",\n{\n            'v': 745,\n            'f': \"745\",\n        },\n\"2019-01-06 14:06:15\",\n\"Source attribution modelling of Salmonella Typhimurium and the monophasic variant.\"],\n [{\n            'v': 404,\n            'f': \"404\",\n        },\n\"dcc_bromhead\",\n\"PRJEB18587\",\n\"Webin-37120\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 53,\n            'f': \"53\",\n        },\n\"2016-12-12 15:14:09\",\n{\n            'v': 106,\n            'f': \"106\",\n        },\n\"2019-01-06 13:21:51\",\n\"E.coli and Salmonella colistin resistance\"],\n [{\n            'v': 405,\n            'f': \"405\",\n        },\n\"dcc_bromhead\",\n\"PRJEB18618\",\n\"Webin-37120\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 162,\n            'f': \"162\",\n        },\n\"2017-01-09 11:01:32\",\n{\n            'v': 401,\n            'f': \"401\",\n        },\n\"2019-01-06 13:30:57\",\n\"EURL reference testing\"],\n [{\n            'v': 406,\n            'f': \"406\",\n        },\n\"dcc_schubert\",\n\"PRJEB18618\",\n\"Webin-37120\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 162,\n            'f': \"162\",\n        },\n\"2017-01-09 11:01:32\",\n{\n            'v': 401,\n            'f': \"401\",\n        },\n\"2019-01-06 13:30:57\",\n\"EURL reference testing\"],\n [{\n            'v': 407,\n            'f': \"407\",\n        },\n\"dcc_blake\",\n\"PRJEB18998\",\n\"Webin-45636\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1161,\n            'f': \"1161\",\n        },\n\"2021-05-11 22:54:36\",\n{\n            'v': 3734,\n            'f': \"3734\",\n        },\n\"2018-12-31 14:21:54\",\n\"Salmonella WGS from Ireland\"],\n [{\n            'v': 408,\n            'f': \"408\",\n        },\n\"dcc_bley\",\n\"PRJEB18998\",\n\"Webin-45636\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1161,\n            'f': \"1161\",\n        },\n\"2021-05-11 22:54:36\",\n{\n            'v': 3734,\n            'f': \"3734\",\n        },\n\"2018-12-31 14:21:54\",\n\"Salmonella WGS from Ireland\"],\n [{\n            'v': 409,\n            'f': \"409\",\n        },\n\"dcc_faang\",\n\"PRJEB19199\",\n\"Webin-45927\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8096,\n            'f': \"8096\",\n        },\n\"2022-04-20 13:26:25\",\n{\n            'v': 426,\n            'f': \"426\",\n        },\n\"2019-03-22 13:46:52\",\n\"RNA sequencing of tissues and cell types from Texel x Scottish Blackface sheep for transcriptome annotation and expression analysis.\"],\n [{\n            'v': 410,\n            'f': \"410\",\n        },\n\"dcc_faang\",\n\"PRJEB19268\",\n\"Webin-31385\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2017-02-01 12:28:37\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Wageningen University & Research Animal Breeding and Genomics FAANG data from three boars.\"],\n [{\n            'v': 411,\n            'f': \"411\",\n        },\n\"dcc_beethoven\",\n\"PRJEB1927\",\n\"Webin-2\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 22,\n            'f': \"22\",\n        },\n\"2014-01-21 09:19:51\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Transcriptional_regulation_of_Treg_cells_differentiation_by_Foxp3___\"],\n [{\n            'v': 412,\n            'f': \"412\",\n        },\n\"dcc_faang\",\n\"PRJEB19386\",\n\"Webin-45137\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 305,\n            'f': \"305\",\n        },\n\"2017-02-10 09:50:41\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Pig transcriptome and gene expression atlas\"],\n [{\n            'v': 413,\n            'f': \"413\",\n        },\n\"dcc_basie\",\n\"PRJEB19488\",\n\"Webin-40965\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 99,\n            'f': \"99\",\n        },\n\"2017-03-29 14:04:15\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Metabarcoding of oomycete and cercozoan communities associated with Arabidopsis thaliana.\"],\n [{\n            'v': 414,\n            'f': \"414\",\n        },\n\"dcc_allison\",\n\"PRJEB2059\",\n\"Webin-2\",\n\"2012-10-18 17:42:01\",\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 390,\n            'f': \"390\",\n        },\n\"2011-09-06 09:06:50\",\n{\n            'v': 1436,\n            'f': \"1436\",\n        },\n\"2018-09-04 13:05:12\",\n\"Phylogeography of Salmonella Typhimurium\"],\n [{\n            'v': 415,\n            'f': \"415\",\n        },\n\"dcc_handel\",\n\"PRJEB20879\",\n\"Webin-43561\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 69,\n            'f': \"69\",\n        },\n\"2019-01-15 10:06:20\",\n{\n            'v': 118,\n            'f': \"118\",\n        },\n\"2020-03-02 08:30:58\",\n\"Bioaccumulation experiment\"],\n [{\n            'v': 416,\n            'f': \"416\",\n        },\n\"dcc_schumann\",\n\"PRJEB20879\",\n\"Webin-43561\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 69,\n            'f': \"69\",\n        },\n\"2019-01-15 10:06:20\",\n{\n            'v': 118,\n            'f': \"118\",\n        },\n\"2020-03-02 08:30:58\",\n\"Bioaccumulation experiment\"],\n [{\n            'v': 417,\n            'f': \"417\",\n        },\n\"dcc_basie\",\n\"PRJEB21047\",\n\"Webin-46909\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 73,\n            'f': \"73\",\n        },\n\"2017-05-26 12:17:26\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"The phases of microalgae succession in sea ice and water column in the Baltic Sea  from autumn to spring\"],\n [{\n            'v': 418,\n            'f': \"418\",\n        },\n\"dcc_schubert\",\n\"PRJEB21131\",\n\"Webin-46478\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 100,\n            'f': \"100\",\n        },\n\"2017-06-01 10:22:41\",\n{\n            'v': 100,\n            'f': \"100\",\n        },\n\"2017-06-02 16:39:09\",\n\"Investigation of antimicrobial resistance in Escherichia species - ENGAGE Project\"],\n [{\n            'v': 419,\n            'f': \"419\",\n        },\n\"dcc_faang\",\n\"PRJEB21180\",\n\"Webin-45927\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNA sequencing of tissues and cell types from water buffalo (Bubalus bubalis) for transcriptome annotation and expression analysis.\"],\n [{\n            'v': 420,\n            'f': \"420\",\n        },\n\"dcc_schubert\",\n\"PRJEB21201\",\n\"Webin-45520\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 103,\n            'f': \"103\",\n        },\n\"2017-06-19 14:02:15\",\n{\n            'v': 102,\n            'f': \"102\",\n        },\n\"2017-07-03 15:22:46\",\n\"A NGS Approach to Characterize Drug Resistance of E. coli Test\"],\n [{\n            'v': 421,\n            'f': \"421\",\n        },\n\"dcc_basie\",\n\"PRJEB21221\",\n\"Webin-44687\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 28,\n            'f': \"28\",\n        },\n\"2017-06-29 15:02:56\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Investigation of the community diversity in protists from the Vestfold Hills, Antarctica, using 454-pyrosequencing\"],\n [{\n            'v': 422,\n            'f': \"422\",\n        },\n\"dcc_basie\",\n\"PRJEB21491\",\n\"Webin-47123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 841,\n            'f': \"841\",\n        },\n\"2017-06-26 16:49:55\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Assembly and activity of microbial communities in the Pacific temperate rainforest\"],\n [{\n            'v': 423,\n            'f': \"423\",\n        },\n\"dcc_faang\",\n\"PRJEB21540\",\n\"Webin-41360\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 360,\n            'f': \"360\",\n        },\n\"2017-06-27 12:56:29\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Tissue-specific DNA methylation of the Chicken and Pig\"],\n [{\n            'v': 424,\n            'f': \"424\",\n        },\n\"dcc_bromhead\",\n\"PRJEB21546\",\n\"Webin-37120\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 192,\n            'f': \"192\",\n        },\n\"2017-11-13 12:21:07\",\n{\n            'v': 381,\n            'f': \"381\",\n        },\n\"2019-01-19 15:33:30\",\n\"EURL Reference Testing 2016\"],\n [{\n            'v': 425,\n            'f': \"425\",\n        },\n\"dcc_benoit\",\n\"PRJEB21631\",\n\"Webin-37120\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 78,\n            'f': \"78\",\n        },\n\"2017-07-03 17:12:02\",\n{\n            'v': 563,\n            'f': \"563\",\n        },\n\"2019-01-06 13:49:33\",\n\"COMPARE Salmonella isolates\"],\n [{\n            'v': 426,\n            'f': \"426\",\n        },\n\"dcc_vivaldi\",\n\"PRJEB21631\",\n\"Webin-37120\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 78,\n            'f': \"78\",\n        },\n\"2017-07-03 17:12:02\",\n{\n            'v': 563,\n            'f': \"563\",\n        },\n\"2019-01-06 13:49:33\",\n\"COMPARE Salmonella isolates\"],\n [{\n            'v': 427,\n            'f': \"427\",\n        },\n\"dcc_cole\",\n\"PRJEB21763\",\n\"Webin-45078\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2017-07-12 14:06:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Tick\"],\n [{\n            'v': 428,\n            'f': \"428\",\n        },\n\"dcc_schubert\",\n\"PRJEB21880\",\n\"Webin-43316\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 99,\n            'f': \"99\",\n        },\n\"2017-07-26 12:48:21\",\n{\n            'v': 314,\n            'f': \"314\",\n        },\n\"2019-01-06 13:51:03\",\n\"Verotoxin-producing E.coli genomes with AMR data\"],\n [{\n            'v': 429,\n            'f': \"429\",\n        },\n\"dcc_vivaldi\",\n\"PRJEB21880\",\n\"Webin-43316\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 99,\n            'f': \"99\",\n        },\n\"2017-07-26 12:48:21\",\n{\n            'v': 314,\n            'f': \"314\",\n        },\n\"2019-01-06 13:51:03\",\n\"Verotoxin-producing E.coli genomes with AMR data\"],\n [{\n            'v': 430,\n            'f': \"430\",\n        },\n\"dcc_basie\",\n\"PRJEB21950\",\n\"Webin-44434\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2017-07-26 17:14:01\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SSU rRNA amplicon of the Arctic Ocean during Winter-Spring Transition\"],\n [{\n            'v': 431,\n            'f': \"431\",\n        },\n\"dcc_schubert\",\n\"PRJEB21997\",\n\"Webin-42757\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 112,\n            'f': \"112\",\n        },\n\"2017-08-01 13:43:47\",\n{\n            'v': 334,\n            'f': \"334\",\n        },\n\"2018-12-06 12:37:31\",\n\"Zoonotic transmission of ESBL resistance genes from non-intensive poultry farms in Vietnam\"],\n [{\n            'v': 432,\n            'f': \"432\",\n        },\n\"dcc_vivaldi\",\n\"PRJEB22002\",\n\"Webin-37120\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 37,\n            'f': \"37\",\n        },\n\"2017-07-31 15:53:20\",\n{\n            'v': 75,\n            'f': \"75\",\n        },\n\"2019-01-06 13:51:22\",\n\"COMPARE Salmonella isolates project 1048\"],\n [{\n            'v': 433,\n            'f': \"433\",\n        },\n\"dcc_bromhead\",\n\"PRJEB22091\",\n\"Webin-37120\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 127,\n            'f': \"127\",\n        },\n\"2017-08-08 13:59:07\",\n{\n            'v': 379,\n            'f': \"379\",\n        },\n\"2019-01-19 15:26:50\",\n\"Danmap 2016\"],\n [{\n            'v': 434,\n            'f': \"434\",\n        },\n\"dcc_barron\",\n\"PRJEB2221\",\n\"Webin-2\",\n\"2012-10-19 10:29:36\",\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2119,\n            'f': \"2119\",\n        },\n\"2014-04-22 10:11:40\",\n{\n            'v': 387,\n            'f': \"387\",\n        },\n\"2015-03-10 09:54:08\",\n\"Mycobacterium tuberculosis pilot\"],\n [{\n            'v': 435,\n            'f': \"435\",\n        },\n\"dcc_faang\",\n\"PRJEB22535\",\n\"Webin-45927\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 34,\n            'f': \"34\",\n        },\n\"2018-08-31 14:27:47\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNA sequencing of cattle (Bos taurus) bone marrow derived macrophages (BMDMs), both unstimulated and 7 hours after stimulation with lipopolysaccharide (LPS).\"],\n [{\n            'v': 436,\n            'f': \"436\",\n        },\n\"dcc_faang\",\n\"PRJEB22536\",\n\"Webin-45927\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNA sequencing of pig (Sus scrofa) bone marrow derived macrophages (BMDMs), both unstimulated and 7 hours after stimulation with lipopolysaccharide (LPS).\"],\n [{\n            'v': 437,\n            'f': \"437\",\n        },\n\"dcc_faang\",\n\"PRJEB22537\",\n\"Webin-45927\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 42,\n            'f': \"42\",\n        },\n\"2017-09-08 14:27:07\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNA sequencing of horse (Equus caballus) bone marrow derived macrophages (BMDMs), both unstimulated and 7 hours after stimulation with lipopolysaccharide (LPS).\"],\n [{\n            'v': 438,\n            'f': \"438\",\n        },\n\"dcc_basie\",\n\"PRJEB23005\",\n\"Webin-40945\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 106,\n            'f': \"106\",\n        },\n\"2017-10-12 17:29:36\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Microbial diversity of the central Arctic Ocean\"],\n [{\n            'v': 439,\n            'f': \"439\",\n        },\n\"dcc_bromhead\",\n\"PRJEB23082\",\n\"Webin-37120\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 51,\n            'f': \"51\",\n        },\n\"2017-10-18 11:48:53\",\n{\n            'v': 101,\n            'f': \"101\",\n        },\n\"2019-01-19 15:32:55\",\n\"ENGAGE_Salmonella_various_serovars\"],\n [{\n            'v': 440,\n            'f': \"440\",\n        },\n\"dcc_faang\",\n\"PRJEB23119\",\n\"Webin-45927\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 42,\n            'f': \"42\",\n        },\n\"2017-10-19 17:17:26\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Duroc Pig Macrophages (+/-) LPS\"],\n [{\n            'v': 441,\n            'f': \"441\",\n        },\n\"dcc_faang\",\n\"PRJEB23196\",\n\"Webin-45927\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 168,\n            'f': \"168\",\n        },\n\"2017-10-26 09:59:56\",\n{\n            'v': 54,\n            'f': \"54\",\n        },\n\"2019-04-25 10:16:56\",\n\"Goat Gene Expression Atlas\"],\n [{\n            'v': 442,\n            'f': \"442\",\n        },\n\"dcc_barron\",\n\"PRJEB23245\",\n\"Webin-47619\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 508,\n            'f': \"508\",\n        },\n\"2022-03-02 20:27:24\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CRyPTIC. Universidad Peruana Cayetano Heredia, Peru. Peru-set1-Tandem\"],\n [{\n            'v': 443,\n            'f': \"443\",\n        },\n\"dcc_benoit\",\n\"PRJEB23266\",\n\"Webin-43316\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 10,\n            'f': \"10\",\n        },\n\"2017-10-31 10:45:39\",\n{\n            'v': 59,\n            'f': \"59\",\n        },\n\"2018-12-31 13:34:01\",\n\"Plasmid-borne colistin resistance gene mcr-3 in Salmonella isolates from human infections.\"],\n [{\n            'v': 444,\n            'f': \"444\",\n        },\n\"dcc_basie\",\n\"PRJEB23427\",\n\"Webin-48322\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2017-11-13 16:13:29\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Seasonal eukaryotic dynamics in Alaskan sea ice\"],\n [{\n            'v': 445,\n            'f': \"445\",\n        },\n\"dcc_basie\",\n\"PRJEB23471\",\n\"Webin-45653\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 10,\n            'f': \"10\",\n        },\n\"2017-11-24 17:42:48\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Metabarcoding analysis of 18S eukaryotic diversity in the Paran\\u00e1 River, Argentina (V4 region)\"],\n [{\n            'v': 446,\n            'f': \"446\",\n        },\n\"dcc_beard\",\n\"PRJEB23496\",\n\"Webin-40757\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 95,\n            'f': \"95\",\n        },\n\"2018-08-13 09:31:33\",\n{\n            'v': 133,\n            'f': \"133\",\n        },\n\"2018-12-01 05:15:09\",\n\"COMPARE global sewage snapshot virome sequencing part\"],\n [{\n            'v': 447,\n            'f': \"447\",\n        },\n\"dcc_basie\",\n\"PRJEB23549\",\n\"Webin-48375\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 122,\n            'f': \"122\",\n        },\n\"2017-12-21 10:00:57\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Protists in Rotated Core Experiment\"],\n [{\n            'v': 448,\n            'f': \"448\",\n        },\n\"dcc_basie\",\n\"PRJEB23558\",\n\"Webin-48378\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 100,\n            'f': \"100\",\n        },\n\"2018-02-13 01:02:34\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"High-throughput 18S V4 ribosomal DNA and RNA sequences could be used to calculate biotic indices for monitoring the impact of two Scotland salmon fish farms on Metazoa communities. The few species that could be assigned and incorporated in AMBI and ITI indices formulas yielded comparable results than with calculations based in morphological data. For both DNA and RNA extractions performed on 45 sediment samples and cDNA synthesis for RNA, we PCR amplified the 37f hypervariable region of the foramnifera using tagged PCR primers prior to library preparation (TruSeq) and MiSeq paired-end sequencing. The combinations of tagged primers are contained in the paried reads and could be used to assign the sequences the their sample of origin.\"],\n [{\n            'v': 449,\n            'f': \"449\",\n        },\n\"dcc_bonner\",\n\"PRJEB23584\",\n\"Webin-47977\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8,\n            'f': \"8\",\n        },\n\"2017-11-22 14:51:43\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"IMR90_konstanz\"],\n [{\n            'v': 450,\n            'f': \"450\",\n        },\n\"dcc_bonner\",\n\"PRJEB23588\",\n\"Webin-47977\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8,\n            'f': \"8\",\n        },\n\"2017-11-22 15:12:31\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"mc_iPSC cells Konstanz\"],\n [{\n            'v': 451,\n            'f': \"451\",\n        },\n\"dcc_bonner\",\n\"PRJEB23589\",\n\"Webin-47977\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2017-11-22 15:29:28\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"LUHMES baseline Konstanz\"],\n [{\n            'v': 452,\n            'f': \"452\",\n        },\n\"dcc_bonner\",\n\"PRJEB23590\",\n\"Webin-47977\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2017-11-22 15:36:01\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"PHH baseline samples\"],\n [{\n            'v': 453,\n            'f': \"453\",\n        },\n\"dcc_bonner\",\n\"PRJEB23591\",\n\"Webin-47977\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 16,\n            'f': \"16\",\n        },\n\"2017-11-22 15:45:35\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SH-SY5Y baseline cells from Swetox\"],\n [{\n            'v': 454,\n            'f': \"454\",\n        },\n\"dcc_handel\",\n\"PRJEB23606\",\n\"Webin-43373\",\nNaN,\n{\n            'v': 3,\n            'f': \"3\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2017-11-28 08:54:35\",\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2018-06-26 12:40:30\",\n\"non-targeted NGS on faecal samples of patients with viral gastro-enteritis\"],\n [{\n            'v': 455,\n            'f': \"455\",\n        },\n\"dcc_bonner\",\n\"PRJEB23619\",\n\"Webin-47977\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2018-01-18 14:22:01\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"U2O5 baseline samples from BDS\"],\n [{\n            'v': 456,\n            'f': \"456\",\n        },\n\"dcc_bonner\",\n\"PRJEB23620\",\n\"Webin-47977\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 9,\n            'f': \"9\",\n        },\n\"2017-11-24 09:45:03\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"iPSC baseline samples; differentiating into hepatocytes\"],\n [{\n            'v': 457,\n            'f': \"457\",\n        },\n\"dcc_bonner\",\n\"PRJEB23622\",\n\"Webin-47977\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 28,\n            'f': \"28\",\n        },\n\"2017-11-24 09:59:11\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"iPSC baseline cell from Innsbruck\"],\n [{\n            'v': 458,\n            'f': \"458\",\n        },\n\"dcc_basie\",\n\"PRJEB23628\",\n\"Webin-46909\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 9,\n            'f': \"9\",\n        },\n\"2017-11-24 14:27:56\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Additional samples in succession study\"],\n [{\n            'v': 459,\n            'f': \"459\",\n        },\n\"dcc_basie\",\n\"PRJEB23641\",\n\"Webin-178\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 428,\n            'f': \"428\",\n        },\n\"2018-01-30 10:43:03\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"The study compares the performance of five barcoding genes for ecological quality status inference in marine environments. The genes include one bacterial, one foraminiferal and three eukaryotes ones,\"],\n [{\n            'v': 460,\n            'f': \"460\",\n        },\n\"dcc_basie\",\n\"PRJEB23666\",\n\"Webin-48411\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 10,\n            'f': \"10\",\n        },\n\"2017-12-11 13:12:18\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"V4 region amplified with universal 18S primers from ten different samples taken around Uppsala, Sweden.\"],\n [{\n            'v': 461,\n            'f': \"461\",\n        },\n\"dcc_basie\",\n\"PRJEB23683\",\n\"Webin-48419\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2017-11-29 15:47:19\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Community analysis submission for UniEuk_EukBank\"],\n [{\n            'v': 462,\n            'f': \"462\",\n        },\n\"dcc_basie\",\n\"PRJEB23687\",\n\"Webin-48420\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2017-11-29 23:42:26\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Sandusky Bay ODNR1 2015-L 18S\"],\n [{\n            'v': 463,\n            'f': \"463\",\n        },\n\"dcc_basie\",\n\"PRJEB23729\",\n\"Webin-48261\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 83,\n            'f': \"83\",\n        },\n\"2017-12-04 20:58:54\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Seasonality of pico- and nano-eukaryotes inhabiting surface waters of a coastal upwelling system\"],\n [{\n            'v': 464,\n            'f': \"464\",\n        },\n\"dcc_basie\",\n\"PRJEB23771\",\n\"Webin-48454\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 179,\n            'f': \"179\",\n        },\n\"2017-12-14 12:18:31\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Picoplankton 18S rRNA gene sequences (V4 region, DNA and RNA) from Vertical Profiles (surface to 4000m depth) sampled during the Malaspina 2010 circumglobal expedition\"],\n [{\n            'v': 465,\n            'f': \"465\",\n        },\n\"dcc_basie\",\n\"PRJEB23785\",\n\"Webin-48456\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 191,\n            'f': \"191\",\n        },\n\"2017-12-22 13:32:50\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Amplicon data 18S Rothera research station\"],\n [{\n            'v': 466,\n            'f': \"466\",\n        },\n\"dcc_basie\",\n\"PRJEB23788\",\n\"Webin-48454\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 289,\n            'f': \"289\",\n        },\n\"2017-12-19 20:38:49\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Blanes Bay Microbial Observatory (BBMO) time-series 18S rRNA gene pico- and nano-plankton (10 years monthly samples)\"],\n [{\n            'v': 467,\n            'f': \"467\",\n        },\n\"dcc_basie\",\n\"PRJEB23800\",\n\"Webin-48473\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 10,\n            'f': \"10\",\n        },\n\"2017-12-07 13:37:14\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Amplicon sequencing-based analysis of eukaryotic communities in marsh soil.\"],\n [{\n            'v': 468,\n            'f': \"468\",\n        },\n\"dcc_basie\",\n\"PRJEB23841\",\n\"Webin-48485\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2017-12-14 11:46:09\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Cercozoan Diversity in forests of North America\"],\n [{\n            'v': 469,\n            'f': \"469\",\n        },\n\"dcc_basie\",\n\"PRJEB23864\",\n\"Webin-48496\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 60,\n            'f': \"60\",\n        },\n\"2017-12-15 20:01:22\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Eukaryotic Taxonomy\"],\n [{\n            'v': 470,\n            'f': \"470\",\n        },\n\"dcc_basie\",\n\"PRJEB23873\",\n\"Webin-48500\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 38,\n            'f': \"38\",\n        },\n\"2017-12-13 01:14:47\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Environmental MicroEukaryote sampling in Nago Bay, Okinawa\"],\n [{\n            'v': 471,\n            'f': \"471\",\n        },\n\"dcc_bromhead\",\n\"PRJEB23891\",\n\"Webin-37120\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 149,\n            'f': \"149\",\n        },\n\"2017-12-12 14:11:23\",\n{\n            'v': 294,\n            'f': \"294\",\n        },\n\"2019-01-19 15:48:44\",\n\"ENGAGE_Salmonella_Derby_Infantis_Napoli\"],\n [{\n            'v': 472,\n            'f': \"472\",\n        },\n\"dcc_basie\",\n\"PRJEB23901\",\n\"Webin-43870\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2017-12-13 18:18:33\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Thermal stress of Benthic foraminifera microbiome.\"],\n [{\n            'v': 473,\n            'f': \"473\",\n        },\n\"dcc_basie\",\n\"PRJEB23910\",\n\"Webin-48454\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 42,\n            'f': \"42\",\n        },\n\"2017-12-15 12:28:14\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Amundsen Sea Polynya International Research Expedition (ASPIRE)\"],\n [{\n            'v': 474,\n            'f': \"474\",\n        },\n\"dcc_basie\",\n\"PRJEB23911\",\n\"Webin-48516\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 82,\n            'f': \"82\",\n        },\n\"2017-12-19 09:32:43\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Eukaryotic diversity in the largest glacial lake of the Iberian Peninsula: a metabarcoding approach\"],\n [{\n            'v': 475,\n            'f': \"475\",\n        },\n\"dcc_basie\",\n\"PRJEB23913\",\n\"Webin-48454\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 124,\n            'f': \"124\",\n        },\n\"2017-12-16 15:06:22\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Picoplankton 18S rRNA genes from the tropical and sub-tropical global-ocean sampled during the Malaspina-2010 expedition\"],\n [{\n            'v': 476,\n            'f': \"476\",\n        },\n\"dcc_basie\",\n\"PRJEB23916\",\n\"Webin-48513\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 104,\n            'f': \"104\",\n        },\n\"2018-01-08 15:06:48\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Molecular inventory of the eukaryote diversity found in the street gutters of Paris, France.\"],\n [{\n            'v': 477,\n            'f': \"477\",\n        },\n\"dcc_basie\",\n\"PRJEB23918\",\n\"Webin-48518\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2018-01-09 21:46:57\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Survey of microbial eukaryotes from BATS\"],\n [{\n            'v': 478,\n            'f': \"478\",\n        },\n\"dcc_basie\",\n\"PRJEB23919\",\n\"Webin-48518\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2018-01-16 19:42:20\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Survey of microbial eukaryote from Hudson River, New York\"],\n [{\n            'v': 479,\n            'f': \"479\",\n        },\n\"dcc_basie\",\n\"PRJEB23920\",\n\"Webin-48518\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 9,\n            'f': \"9\",\n        },\n\"2018-01-16 19:56:42\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Survey of microbial eukaryotes from Ulsan Harbor, Korea\"],\n [{\n            'v': 480,\n            'f': \"480\",\n        },\n\"dcc_basie\",\n\"PRJEB23921\",\n\"Webin-48518\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2018-01-16 20:05:17\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Survey of microbial eukaryotes in Daegok dam, Korea\"],\n [{\n            'v': 481,\n            'f': \"481\",\n        },\n\"dcc_basie\",\n\"PRJEB23922\",\n\"Webin-48518\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2018-01-16 20:11:35\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Survey of microbial eukaryotes from Solidaridad Riviera Maya, Mexico\"],\n [{\n            'v': 482,\n            'f': \"482\",\n        },\n\"dcc_basie\",\n\"PRJEB23923\",\n\"Webin-48518\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2018-01-09 21:30:38\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Survey of microbial eukaryotes in White Lake of NJ, USA\"],\n [{\n            'v': 483,\n            'f': \"483\",\n        },\n\"dcc_basie\",\n\"PRJEB23924\",\n\"Webin-48518\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2018-01-16 20:16:05\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Survey of microbial eukaryotes from pitcher plants\"],\n [{\n            'v': 484,\n            'f': \"484\",\n        },\n\"dcc_basie\",\n\"PRJEB23943\",\n\"Webin-48375\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 96,\n            'f': \"96\",\n        },\n\"2017-12-15 12:01:47\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Changes in the diversity, composition and trophic groups of protists due to transformation of lowland rainforest into plantations.\"],\n [{\n            'v': 485,\n            'f': \"485\",\n        },\n\"dcc_basie\",\n\"PRJEB23950\",\n\"Webin-43276\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 168,\n            'f': \"168\",\n        },\n\"2017-12-21 16:45:14\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"18S rRNA gene V4 region amplicon sequencing of protists in NYC environmental samples\"],\n [{\n            'v': 486,\n            'f': \"486\",\n        },\n\"dcc_basie\",\n\"PRJEB23966\",\n\"Webin-48542\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 46,\n            'f': \"46\",\n        },\n\"2018-10-01 12:09:11\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Eukaryotic diversity in biofilms from a drinking water distribution network\"],\n [{\n            'v': 487,\n            'f': \"487\",\n        },\n\"dcc_basie\",\n\"PRJEB23971\",\n\"Webin-48543\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 35,\n            'f': \"35\",\n        },\n\"2017-12-20 21:54:20\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Diversity and ecology of nanophytoplankton in the Gulf of Gda\\u0144sk\"],\n [{\n            'v': 488,\n            'f': \"488\",\n        },\n\"dcc_basie\",\n\"PRJEB23975\",\n\"Webin-48497\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 10,\n            'f': \"10\",\n        },\n\"2018-01-16 15:37:07\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Narragansett Bay Eukaryotes - Sedimentary archived eukaryotic diversity in Narragansett Bay, RI, USA\"],\n [{\n            'v': 489,\n            'f': \"489\",\n        },\n\"dcc_basie\",\n\"PRJEB23984\",\n\"Webin-48552\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2017-12-21 01:09:25\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Dynamic shifts of phytoplankton community structures in the course of a dinoflagellate bloom\"],\n [{\n            'v': 490,\n            'f': \"490\",\n        },\n\"dcc_basie\",\n\"PRJEB23994\",\n\"Webin-41042\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 42,\n            'f': \"42\",\n        },\n\"2018-01-31 12:28:44\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"18S-V4 diversity in the English channel using an automated water sampler\"],\n [{\n            'v': 491,\n            'f': \"491\",\n        },\n\"dcc_basie\",\n\"PRJEB24000\",\n\"Webin-48565\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 91,\n            'f': \"91\",\n        },\n\"2017-12-30 05:15:18\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Intertidal transects of the microbial communities from sandy beaches of Calvert Island, British Columbia, Canada, 2014\"],\n [{\n            'v': 492,\n            'f': \"492\",\n        },\n\"dcc_basie\",\n\"PRJEB24002\",\n\"Webin-48498\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 14,\n            'f': \"14\",\n        },\n\"2018-01-17 17:15:41\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Global Lacustrine Eukarya - Eukaryotic diversity in lake water and sediments\"],\n [{\n            'v': 493,\n            'f': \"493\",\n        },\n\"dcc_basie\",\n\"PRJEB24017\",\n\"Webin-48589\",\n\"2023-11-16 00:00:00\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2017-12-19 17:41:24\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Eukaryotic diversity in the freshwater lake Lagunas de Ruidera (Spain)\"],\n [{\n            'v': 494,\n            'f': \"494\",\n        },\n\"dcc_basie\",\n\"PRJEB24025\",\n\"Webin-48381\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 6,\n            'f': \"6\",\n        },\n\"2017-12-19 20:30:16\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Pelagic eukaryote microbial diversity in Amundsen Gulf, from surface to bottom\"],\n [{\n            'v': 495,\n            'f': \"495\",\n        },\n\"dcc_basie\",\n\"PRJEB24049\",\n\"Webin-48590\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 24,\n            'f': \"24\",\n        },\n\"2017-12-31 18:27:57\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Microeukaryotes of undisturbed soil and treated areas of northern Alberta boreal forest.\"],\n [{\n            'v': 496,\n            'f': \"496\",\n        },\n\"dcc_basie\",\n\"PRJEB24051\",\n\"Webin-48511\",\n\"2023-11-16 00:00:00\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 6,\n            'f': \"6\",\n        },\n\"2017-12-21 14:49:16\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Diversity of protists in Marmara Sea sediments and cold seeps\"],\n [{\n            'v': 497,\n            'f': \"497\",\n        },\n\"dcc_basie\",\n\"PRJEB24054\",\n\"Webin-48381\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 142,\n            'f': \"142\",\n        },\n\"2017-12-20 17:46:25\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Protist distribution in Chukchi sea in spring 2014 to study under-ice blooms (SUBICE 2014) revealed by 18S rRNA high throughput sequencing\"],\n [{\n            'v': 498,\n            'f': \"498\",\n        },\n\"dcc_basie\",\n\"PRJEB24072\",\n\"Webin-48511\",\n\"2023-11-16 00:00:00\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2017-12-21 15:05:40\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Diversity of microbial eukaryotes from sediments of freshwater systems at the Natural Regional Park of the Haute Vallee de Chevreuse, France\"],\n [{\n            'v': 499,\n            'f': \"499\",\n        },\n\"dcc_basie\",\n\"PRJEB24074\",\n\"Webin-48381\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 39,\n            'f': \"39\",\n        },\n\"2017-12-21 15:43:00\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Microbial diversity across the Hudson Bay: a river to sea continuum\"],\n [{\n            'v': 500,\n            'f': \"500\",\n        },\n\"dcc_basie\",\n\"PRJEB24089\",\n\"Webin-48381\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 233,\n            'f': \"233\",\n        },\n\"2017-12-21 19:24:14\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Pelagic marine and freshwater protist diversity in Kimitkeot region (Cambridge Bay, Nunavut, Canada)\"],\n [{\n            'v': 501,\n            'f': \"501\",\n        },\n\"dcc_basie\",\n\"PRJEB24091\",\n\"Webin-48611\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 44,\n            'f': \"44\",\n        },\n\"2018-01-10 09:33:12\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"22 sediments/soils/mosses samples from natural granitic pools located in the Natural Park of Sierra de Guadarrama (near Madrid) Spain.\"],\n [{\n            'v': 502,\n            'f': \"502\",\n        },\n\"dcc_basie\",\n\"PRJEB24093\",\n\"Webin-48613\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2017-12-22 11:18:24\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Coastal microplankton of the northern Adriatic\"],\n [{\n            'v': 503,\n            'f': \"503\",\n        },\n\"dcc_basie\",\n\"PRJEB24094\",\n\"Webin-48614\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 6,\n            'f': \"6\",\n        },\n\"2017-12-22 11:40:37\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Open water microplankton of the northern Adriatic\"],\n [{\n            'v': 504,\n            'f': \"504\",\n        },\n\"dcc_basie\",\n\"PRJEB24098\",\n\"Webin-48617\",\n\"2023-12-07 00:00:00\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 94,\n            'f': \"94\",\n        },\n\"2018-01-03 15:13:54\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Effect of urine, faeces, blood and pig meat on soil micro-Eukaryotes\"],\n [{\n            'v': 505,\n            'f': \"505\",\n        },\n\"dcc_basie\",\n\"PRJEB24122\",\n\"Webin-48594\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 24,\n            'f': \"24\",\n        },\n\"2018-01-16 21:22:24\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Sydney Harbour microbial diversity study\"],\n [{\n            'v': 506,\n            'f': \"506\",\n        },\n\"dcc_basie\",\n\"PRJEB24127\",\n\"Webin-48518\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2018-01-16 20:20:11\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Survey of microbial eukaryotes from Sea off Jinha, South Korea\"],\n [{\n            'v': 507,\n            'f': \"507\",\n        },\n\"dcc_basie\",\n\"PRJEB24128\",\n\"Webin-48468\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2017-12-28 10:01:12\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Eukaryotic communities of saline water bodies in Orenburg region (Russia) studied with 18S metagenomic sequencing on the Illumina platform (Miseq)\"],\n [{\n            'v': 508,\n            'f': \"508\",\n        },\n\"dcc_basie\",\n\"PRJEB24133\",\n\"Webin-48632\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 10,\n            'f': \"10\",\n        },\n\"2018-01-03 14:01:36\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Plankton from 10 Pampean shallow lakes were investigated using MiSeq\"],\n [{\n            'v': 509,\n            'f': \"509\",\n        },\n\"dcc_basie\",\n\"PRJEB24138\",\n\"Webin-48500\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 24,\n            'f': \"24\",\n        },\n\"2017-12-28 09:26:18\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"MicroEukaryote Diversity in the East China Sea / Kuroshio Current\"],\n [{\n            'v': 510,\n            'f': \"510\",\n        },\n\"dcc_basie\",\n\"PRJEB24144\",\n\"Webin-48637\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 35,\n            'f': \"35\",\n        },\n\"2017-12-31 00:35:39\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Eukaryote sequences amplified from geothermal springs in Canada and New Zealand, and from saline springs from Canada.\"],\n [{\n            'v': 511,\n            'f': \"511\",\n        },\n\"dcc_basie\",\n\"PRJEB24153\",\n\"Webin-48645\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 46,\n            'f': \"46\",\n        },\n\"2018-02-12 20:10:54\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"V4 region sampling of the oil sands waste remediation site Base Mine Lake, in Northern Alberta.\"],\n [{\n            'v': 512,\n            'f': \"512\",\n        },\n\"dcc_basie\",\n\"PRJEB24154\",\n\"Webin-48646\",\n\"2024-02-01 00:00:00\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 20,\n            'f': \"20\",\n        },\n\"2018-01-02 11:18:55\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"2015 Phytoplankton assemblage in Northwest Pacific\"],\n [{\n            'v': 513,\n            'f': \"513\",\n        },\n\"dcc_basie\",\n\"PRJEB24157\",\n\"Webin-48647\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 29,\n            'f': \"29\",\n        },\n\"2018-02-15 08:09:42\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"A time series of 18S SSU rDNA V4 region meta-amplicons from Bellingham Bay, WA, USA.\"],\n [{\n            'v': 514,\n            'f': \"514\",\n        },\n\"dcc_basie\",\n\"PRJEB24158\",\n\"Webin-48648\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2018-02-09 15:28:20\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Drobak Sediment 18S V4 Amplicon\"],\n [{\n            'v': 515,\n            'f': \"515\",\n        },\n\"dcc_basie\",\n\"PRJEB24160\",\n\"Webin-48551\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 11,\n            'f': \"11\",\n        },\n\"2018-01-04 10:36:54\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Soils from different localisation in the Canary islands\"],\n [{\n            'v': 516,\n            'f': \"516\",\n        },\n\"dcc_basie\",\n\"PRJEB24162\",\n\"Webin-48590\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2017-12-31 19:18:51\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Microeukaryotes of a tailings pond sediments in Northeastern Alberta.\"],\n [{\n            'v': 517,\n            'f': \"517\",\n        },\n\"dcc_basie\",\n\"PRJEB24164\",\n\"Webin-48646\",\n\"2024-02-01 00:00:00\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 44,\n            'f': \"44\",\n        },\n\"2018-01-02 12:09:25\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Diel sampling of mesozooplankton collected by net tow at C6 and C9 stations of Taiwan Strait.\"],\n [{\n            'v': 518,\n            'f': \"518\",\n        },\n\"dcc_faang\",\n\"PRJEB24166\",\n\"Webin-48607\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 36,\n            'f': \"36\",\n        },\n\"2018-01-02 10:17:54\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Cattle transcriptome of macrophages\"],\n [{\n            'v': 519,\n            'f': \"519\",\n        },\n\"dcc_basie\",\n\"PRJEB24184\",\n\"Webin-48551\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 15,\n            'f': \"15\",\n        },\n\"2018-01-04 15:28:36\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Environmental DNA study of the V4 SSU genes of water tank of Bromeliads in Costa Rica\"],\n [{\n            'v': 520,\n            'f': \"520\",\n        },\n\"dcc_basie\",\n\"PRJEB24241\",\n\"Webin-48611\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 280,\n            'f': \"280\",\n        },\n\"2018-01-15 13:14:10\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Study of soil protists under pig cadaver\"],\n [{\n            'v': 521,\n            'f': \"521\",\n        },\n\"dcc_basie\",\n\"PRJEB24314\",\n\"Webin-48381\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 247,\n            'f': \"247\",\n        },\n\"2018-01-08 23:06:07\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Diversity of marine microbial eukaryotes in Nares Strait in summer 2014 and 2016\"],\n [{\n            'v': 522,\n            'f': \"522\",\n        },\n\"dcc_basie\",\n\"PRJEB24415\",\n\"Webin-48454\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 23,\n            'f': \"23\",\n        },\n\"2018-01-16 12:46:22\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Lake Baikal community 18S rRNA-gene sequences\"],\n [{\n            'v': 523,\n            'f': \"523\",\n        },\n\"dcc_basie\",\n\"PRJEB24428\",\n\"Webin-48518\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8,\n            'f': \"8\",\n        },\n\"2018-01-16 22:33:10\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Microbial Eukaryotes of Lagoons of the Solomon Islands\"],\n [{\n            'v': 524,\n            'f': \"524\",\n        },\n\"dcc_basie\",\n\"PRJEB24453\",\n\"Webin-43111\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 9,\n            'f': \"9\",\n        },\n\"2018-01-19 12:20:45\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Coral and human-associated micro-eukaryotes.\"],\n [{\n            'v': 525,\n            'f': \"525\",\n        },\n\"dcc_bonner\",\n\"PRJEB24464\",\n\"Webin-47977\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2018-01-18 10:00:46\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"HepG2 baseline samples\"],\n [{\n            'v': 526,\n            'f': \"526\",\n        },\n\"dcc_bonner\",\n\"PRJEB24466\",\n\"Webin-47977\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 51,\n            'f': \"51\",\n        },\n\"2018-11-01 14:52:45\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"HepG2 BAC and wild-type baseline cell\"],\n [{\n            'v': 527,\n            'f': \"527\",\n        },\n\"dcc_bonner\",\n\"PRJEB24481\",\n\"Webin-47977\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8,\n            'f': \"8\",\n        },\n\"2018-01-18 14:12:45\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"PBEC Baseline samples from Leiden\"],\n [{\n            'v': 528,\n            'f': \"528\",\n        },\n\"dcc_bonner\",\n\"PRJEB24482\",\n\"Webin-47977\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 39,\n            'f': \"39\",\n        },\n\"2018-11-01 15:03:47\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Primary 3D Liver micro tissues baseline samples\"],\n [{\n            'v': 529,\n            'f': \"529\",\n        },\n\"dcc_bonner\",\n\"PRJEB24484\",\n\"Webin-47977\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8,\n            'f': \"8\",\n        },\n\"2018-01-18 14:49:09\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"hu-PCLiS baseline samples\"],\n [{\n            'v': 530,\n            'f': \"530\",\n        },\n\"dcc_bonner\",\n\"PRJEB24485\",\n\"Webin-47977\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 48,\n            'f': \"48\",\n        },\n\"2018-11-01 14:40:45\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"mc-iPSC to H9 baseline samples from Konstanz\"],\n [{\n            'v': 531,\n            'f': \"531\",\n        },\n\"dcc_bonner\",\n\"PRJEB24487\",\n\"Webin-47977\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2018-01-18 15:06:52\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"HepaRG 3D baseline samples\"],\n [{\n            'v': 532,\n            'f': \"532\",\n        },\n\"dcc_bonner\",\n\"PRJEB24489\",\n\"Webin-47977\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 10,\n            'f': \"10\",\n        },\n\"2018-01-18 15:32:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"hu-PCLS baseline samples from ITEM\"],\n [{\n            'v': 533,\n            'f': \"533\",\n        },\n\"dcc_basie\",\n\"PRJEB24517\",\n\"Webin-35499\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 81,\n            'f': \"81\",\n        },\n\"2019-01-29 15:40:25\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Arctic microbiome along Svalbard Cross Shelf transects\"],\n [{\n            'v': 534,\n            'f': \"534\",\n        },\n\"dcc_basie\",\n\"PRJEB24538\",\n\"Webin-48500\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 54,\n            'f': \"54\",\n        },\n\"2018-01-23 02:56:56\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Intra-host symbiont diversity and extended symbiont maintenance in photosymbiotic Acantharea (clade F)\"],\n [{\n            'v': 535,\n            'f': \"535\",\n        },\n\"dcc_basie\",\n\"PRJEB24595\",\n\"Webin-31315\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8,\n            'f': \"8\",\n        },\n\"2018-01-25 13:29:53\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Temporal diversity and community structure of the planktonic protist assemblage at the Long Term Ecological Research (LTER) station MareChiara in the Gulf of Naples(Mediterranean Sea) assessed using V4 Illumina sequencing.\"],\n [{\n            'v': 536,\n            'f': \"536\",\n        },\n\"dcc_basie\",\n\"PRJEB24633\",\n\"Webin-31315\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2018-01-25 19:21:34\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Diversity of diatom resting stages in surface sediments using V4 fragment of 18S.\"],\n [{\n            'v': 537,\n            'f': \"537\",\n        },\n\"dcc_basie\",\n\"PRJEB24634\",\n\"Webin-48851\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 148,\n            'f': \"148\",\n        },\n\"2018-01-27 12:56:23\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"We applied different fertilizer strategy including chemical fertilizer, organic fertilizer, and bio-organic fertilizer (Trichoderma) under a crop rotation system (Maize-Cabbage-Potato) in a two years field experiment in Nantong city, China.\"],\n [{\n            'v': 538,\n            'f': \"538\",\n        },\n\"dcc_basie\",\n\"PRJEB24650\",\n\"Webin-48623\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 31,\n            'f': \"31\",\n        },\n\"2018-02-01 21:47:08\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"The parabasalid symbiont community of Heterotermes aureus: Molecular and morphological characterization of four new species and reestablishment of the genus Cononympha\"],\n [{\n            'v': 539,\n            'f': \"539\",\n        },\n\"dcc_basie\",\n\"PRJEB24677\",\n\"Webin-48869\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 9,\n            'f': \"9\",\n        },\n\"2018-02-01 02:04:41\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Eukaryotic diversity in a floodplain fluvial system\"],\n [{\n            'v': 540,\n            'f': \"540\",\n        },\n\"dcc_basie\",\n\"PRJEB24694\",\n\"Webin-35687\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8,\n            'f': \"8\",\n        },\n\"2018-01-31 09:13:04\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Food source analysis for freshwater pearl mussels\"],\n [{\n            'v': 541,\n            'f': \"541\",\n        },\n\"dcc_basie\",\n\"PRJEB24755\",\n\"Webin-48918\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 36,\n            'f': \"36\",\n        },\n\"2018-02-02 10:33:24\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Integrating quantitative morphological and qualitative\\nmolecular methods to analyze soil nematode community\\nresponses to plant range expansion\"],\n [{\n            'v': 542,\n            'f': \"542\",\n        },\n\"dcc_basie\",\n\"PRJEB24823\",\n\"Webin-48623\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 14,\n            'f': \"14\",\n        },\n\"2018-02-14 23:12:00\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Incomplete Co-cladogenesis Between Zootermopsis Termites and Their Associated Protists\"],\n [{\n            'v': 543,\n            'f': \"543\",\n        },\n\"dcc_basie\",\n\"PRJEB24876\",\n\"Webin-48648\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 6,\n            'f': \"6\",\n        },\n\"2018-02-09 13:16:23\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Bunnefjorden Sediment 18S V4 Amplicon\"],\n [{\n            'v': 544,\n            'f': \"544\",\n        },\n\"dcc_basie\",\n\"PRJEB24888\",\n\"Webin-48648\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8,\n            'f': \"8\",\n        },\n\"2018-02-12 08:37:27\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Pollevann Sediment 18S V4 Amplicon\"],\n [{\n            'v': 545,\n            'f': \"545\",\n        },\n\"dcc_basie\",\n\"PRJEB24912\",\n\"Webin-48981\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 59,\n            'f': \"59\",\n        },\n\"2018-02-11 04:32:28\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"A cross-taxa study using environmental DNA/RNA metabarcoding to measure biological impacts of offshore oil and gas drilling and production operations\"],\n [{\n            'v': 546,\n            'f': \"546\",\n        },\n\"dcc_basie\",\n\"PRJEB24918\",\n\"Webin-48990\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 469,\n            'f': \"469\",\n        },\n\"2018-02-12 12:36:35\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Soil nematode sequencing across Europe\"],\n [{\n            'v': 547,\n            'f': \"547\",\n        },\n\"dcc_faang\",\n\"PRJEB24920\",\n\"Webin-45927\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 42,\n            'f': \"42\",\n        },\n\"2018-02-12 13:34:48\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Horse Bone Marrow-Derived Macrophages (+/-) LPS\"],\n [{\n            'v': 548,\n            'f': \"548\",\n        },\n\"dcc_basie\",\n\"PRJEB24923\",\n\"Webin-48991\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-02-14 11:54:35\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"V4 18SrRNA HTP Metabarcoding of a Brazilian lagoon - Piratininga\"],\n [{\n            'v': 549,\n            'f': \"549\",\n        },\n\"dcc_basie\",\n\"PRJEB24972\",\n\"Webin-48468\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 14,\n            'f': \"14\",\n        },\n\"2018-02-22 14:40:51\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Eukaryotic communities of brackish and saline water bodies of Cis-Urals region (Russia) studied with 18S metagenomic sequencing on the Illumina platform (Miseq)\"],\n [{\n            'v': 550,\n            'f': \"550\",\n        },\n\"dcc_basie\",\n\"PRJEB24985\",\n\"Webin-35730\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2018-02-15 19:52:32\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Microbes' Microbiomes: characterization of prokaryotic microbiomes of single-celled protists\"],\n [{\n            'v': 551,\n            'f': \"551\",\n        },\n\"dcc_basie\",\n\"PRJEB24988\",\n\"Webin-48981\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 105,\n            'f': \"105\",\n        },\n\"2018-02-15 22:07:07\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Metabarcoding improves detection of eukaryotes from early biofouling communities: implications for pest monitoring and pathway management\"],\n [{\n            'v': 552,\n            'f': \"552\",\n        },\n\"dcc_basie\",\n\"PRJEB25005\",\n\"Webin-49030\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2018-02-16 17:05:43\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Metagenomic survey of microbial eukaryotes in a tidal sand/mud flat in the Pacific coast of British Columbia\"],\n [{\n            'v': 553,\n            'f': \"553\",\n        },\n\"dcc_basie\",\n\"PRJEB25010\",\n\"Webin-43333\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 696,\n            'f': \"696\",\n        },\n\"2018-02-20 21:47:15\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Microbial Ecology of Seaweed\"],\n [{\n            'v': 554,\n            'f': \"554\",\n        },\n\"dcc_basie\",\n\"PRJEB25011\",\n\"Webin-48981\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 45,\n            'f': \"45\",\n        },\n\"2018-02-16 06:36:30\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Wanted dead or alive? Using metabarcoding of environmental DNA and RNA to distinguish living assemblages for biosecurity applications\"],\n [{\n            'v': 555,\n            'f': \"555\",\n        },\n\"dcc_basie\",\n\"PRJEB25025\",\n\"Webin-49030\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 10,\n            'f': \"10\",\n        },\n\"2018-02-16 15:37:59\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Eukaryotic community associated with a marine microbial mat\"],\n [{\n            'v': 556,\n            'f': \"556\",\n        },\n\"dcc_basie\",\n\"PRJEB25034\",\n\"Webin-48623\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 17,\n            'f': \"17\",\n        },\n\"2018-02-23 16:45:13\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Composition of the protist community in the hindgut of Coptotermes gestroi and Coptotermes formosanus\"],\n [{\n            'v': 557,\n            'f': \"557\",\n        },\n\"dcc_basie\",\n\"PRJEB25036\",\n\"Webin-48981\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 119,\n            'f': \"119\",\n        },\n\"2018-02-17 03:54:16\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Advantageous complementarity of morpho-taxonomy and metabarcoding for passive surveillance of non-indigenous marine pests.\"],\n [{\n            'v': 558,\n            'f': \"558\",\n        },\n\"dcc_basie\",\n\"PRJEB25037\",\n\"Webin-48981\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 105,\n            'f': \"105\",\n        },\n\"2018-02-17 05:20:49\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Development and validation of a multi-trophic metabarcoding biotic index for benthic organic enrichment biomonitoring using a salmon farm case-study.\"],\n [{\n            'v': 559,\n            'f': \"559\",\n        },\n\"dcc_basie\",\n\"PRJEB25089\",\n\"Webin-46909\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 88,\n            'f': \"88\",\n        },\n\"2018-02-22 09:16:40\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"The phases of microalgae succession in sea ice and water column in the Baltic Sea  from autumn to spring\"],\n [{\n            'v': 560,\n            'f': \"560\",\n        },\n\"dcc_basie\",\n\"PRJEB25117\",\n\"Webin-49100\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 54,\n            'f': \"54\",\n        },\n\"2018-02-22 14:47:13\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Protist communities are more sensitive to nitrogen fertilization than other microorganisms in diverse agricultural soils\"],\n [{\n            'v': 561,\n            'f': \"561\",\n        },\n\"dcc_basie\",\n\"PRJEB25141\",\n\"Webin-48496\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 60,\n            'f': \"60\",\n        },\n\"2018-02-24 16:44:35\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"AWECS2 sea-ice and water samples\"],\n [{\n            'v': 562,\n            'f': \"562\",\n        },\n\"dcc_basie\",\n\"PRJEB25163\",\n\"Webin-48623\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 31,\n            'f': \"31\",\n        },\n\"2018-02-26 19:17:02\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Evolution, diversity and composition of the protist community in the hindgut of Heterotermes tenuis, a subterranean termite\"],\n [{\n            'v': 563,\n            'f': \"563\",\n        },\n\"dcc_basie\",\n\"PRJEB25169\",\n\"Webin-48981\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 90,\n            'f': \"90\",\n        },\n\"2018-02-26 19:34:31\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"A study investigating pelagic diversity and the detection of invasive species in the Auckland harbour\"],\n [{\n            'v': 564,\n            'f': \"564\",\n        },\n\"dcc_basie\",\n\"PRJEB25197\",\n\"Webin-49124\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2018-02-28 00:08:45\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Long-read metabarcoding of the eukaryotic rDNA operon to phylogenetically and taxonomically resolve environmental diversity\"],\n [{\n            'v': 565,\n            'f': \"565\",\n        },\n\"dcc_faang\",\n\"PRJEB25226\",\n\"Webin-45927\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2452,\n            'f': \"2452\",\n        },\n\"2018-02-27 15:51:20\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Water buffalo gene expression atlas\"],\n [{\n            'v': 566,\n            'f': \"566\",\n        },\n\"dcc_basie\",\n\"PRJEB25232\",\n\"Webin-49138\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2018-02-28 03:47:02\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Samples of eukaryotic plankton sequenced at the V4 rDNA region.\"],\n [{\n            'v': 567,\n            'f': \"567\",\n        },\n\"dcc_basie\",\n\"PRJEB25259\",\n\"Webin-49148\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 30,\n            'f': \"30\",\n        },\n\"2018-02-28 21:04:23\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Diplonemids in the Arctic marine plankton\"],\n [{\n            'v': 568,\n            'f': \"568\",\n        },\n\"dcc_basie\",\n\"PRJEB25267\",\n\"Webin-43333\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 307,\n            'f': \"307\",\n        },\n\"2020-11-12 09:54:44\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Seagrass Microbiome Survey 2015, Calvert Island, British Columbia\"],\n [{\n            'v': 569,\n            'f': \"569\",\n        },\n\"dcc_basie\",\n\"PRJEB25353\",\n\"Webin-49161\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2018-03-02 23:43:28\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Diversity of picoeukaryotes in Northern California coastal water\"],\n [{\n            'v': 570,\n            'f': \"570\",\n        },\n\"dcc_schubert\",\n\"PRJEB25629\",\n\"Webin-42757\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 31,\n            'f': \"31\",\n        },\n\"2018-12-12 10:24:16\",\n{\n            'v': 29,\n            'f': \"29\",\n        },\n\"2018-12-13 13:13:08\",\n\"Vancomycin-resistance Enterococcus faecium isolates were collected and sequenced during an outbreak occuring at the Academisch Medisch Centrum in Amsterdam in 2017.\"],\n [{\n            'v': 571,\n            'f': \"571\",\n        },\n\"dcc_faang\",\n\"PRJEB25677\",\n\"Webin-48466\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 438,\n            'f': \"438\",\n        },\n\"2018-03-22 01:22:07\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Bovine gene expression atlas\"],\n [{\n            'v': 572,\n            'f': \"572\",\n        },\n\"dcc_blake\",\n\"PRJEB26404\",\n\"Webin-49012\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2018-11-09 13:14:07\",\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2018-09-11 15:41:38\",\n\"Study for testing the EBI-ENA workspace system within the COMPARE pilot project.\"],\n [{\n            'v': 573,\n            'f': \"573\",\n        },\n\"dcc_bley\",\n\"PRJEB26404\",\n\"Webin-49012\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2018-11-09 13:14:07\",\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2018-09-11 15:41:38\",\n\"Study for testing the EBI-ENA workspace system within the COMPARE pilot project.\"],\n [{\n            'v': 574,\n            'f': \"574\",\n        },\n\"dcc_faang\",\n\"PRJEB26698\",\n\"Webin-49517\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2018-05-11 12:43:55\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Whole Genome Shotgun Sequence data for two female Thoroughbreds\"],\n [{\n            'v': 575,\n            'f': \"575\",\n        },\n\"dcc_blake\",\n\"PRJEB26717\",\n\"Webin-48389\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 187,\n            'f': \"187\",\n        },\n\"2018-09-03 14:39:29\",\n{\n            'v': 797,\n            'f': \"797\",\n        },\n\"2018-12-31 14:19:56\",\n\"ECDC Pilot 2018\"],\n [{\n            'v': 576,\n            'f': \"576\",\n        },\n\"dcc_bley\",\n\"PRJEB26717\",\n\"Webin-48389\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 187,\n            'f': \"187\",\n        },\n\"2018-09-03 14:39:29\",\n{\n            'v': 797,\n            'f': \"797\",\n        },\n\"2018-12-31 14:19:56\",\n\"ECDC Pilot 2018\"],\n [{\n            'v': 577,\n            'f': \"577\",\n        },\n\"dcc_faang\",\n\"PRJEB26787\",\n\"Webin-49517\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 104,\n            'f': \"104\",\n        },\n\"2018-05-16 17:35:09\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"FAANG Transcriptome Analysis of the horse\"],\n [{\n            'v': 578,\n            'f': \"578\",\n        },\n\"dcc_bromhead\",\n\"PRJEB26940\",\n\"Webin-43578\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 10,\n            'f': \"10\",\n        },\n\"2018-05-24 11:40:40\",\n{\n            'v': 20,\n            'f': \"20\",\n        },\n\"2019-01-19 15:49:56\",\n\"Molecular epidemiology of mcr-1 and mcr-3 plasmids from a pig farm in Great Britain over the course of 5 months\"],\n [{\n            'v': 579,\n            'f': \"579\",\n        },\n\"dcc_bromhead\",\n\"PRJEB26942\",\n\"Webin-43578\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 41,\n            'f': \"41\",\n        },\n\"2018-05-24 13:15:07\",\n{\n            'v': 81,\n            'f': \"81\",\n        },\n\"2019-01-19 15:55:02\",\n\"Molecular epidemiology of mcr-1 and mcr-3 plasmids from a pig farm in Great Britain over the course of 5 months\"],\n [{\n            'v': 580,\n            'f': \"580\",\n        },\n\"dcc_bley\",\n\"PRJEB27027\",\n\"Webin-49910\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 21,\n            'f': \"21\",\n        },\n\"2018-09-26 15:25:37\",\n{\n            'v': 21,\n            'f': \"21\",\n        },\n\"2018-09-30 15:50:23\",\n\"COMPARE_pilot_ISCIII\"],\n [{\n            'v': 581,\n            'f': \"581\",\n        },\n\"dcc_liszt\",\n\"PRJEB27054\",\n\"Webin-37120\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 235,\n            'f': \"235\",\n        },\n\"2018-05-31 13:35:14\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Global surveillance of antimicrobial resistance\"],\n [{\n            'v': 582,\n            'f': \"582\",\n        },\n\"dcc_faang\",\n\"PRJEB27111\",\n\"Webin-46224\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 38,\n            'f': \"38\",\n        },\n\"2018-06-04 08:21:33\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Chromatin accessibility profiling of liver and T cells in 4 livestock species by the FAANG pilot project FR-AgENCODE\"],\n [{\n            'v': 583,\n            'f': \"583\",\n        },\n\"dcc_cole\",\n\"PRJEB27238\",\n\"Webin-29236\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2018-06-11 13:57:58\",\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2019-03-15 04:30:09\",\n\"COMPARE Food Ring Trial - wet lab\"],\n [{\n            'v': 584,\n            'f': \"584\",\n        },\n\"dcc_cole\",\n\"PRJEB27260\",\n\"Webin-48113\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2018-06-15 12:25:10\",\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2019-03-15 04:45:07\",\n\"Compare Metagenomic Food Ring Trial-WET Lab\"],\n [{\n            'v': 585,\n            'f': \"585\",\n        },\n\"dcc_cole\",\n\"PRJEB27267\",\n\"Webin-48117\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2018-06-12 13:21:48\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"COMPARE Food Metagenomic Ring Trial 2018 - spiked salmon\"],\n [{\n            'v': 586,\n            'f': \"586\",\n        },\n\"dcc_cole\",\n\"PRJEB27316\",\n\"Webin-37120\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 16,\n            'f': \"16\",\n        },\n\"2018-06-15 11:45:28\",\n{\n            'v': 8,\n            'f': \"8\",\n        },\n\"2020-03-02 08:31:08\",\n\"COMPARE Food Ring Trial - wet\"],\n [{\n            'v': 587,\n            'f': \"587\",\n        },\n\"dcc_faang\",\n\"PRJEB27337\",\n\"Webin-49950\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 86,\n            'f': \"86\",\n        },\n\"2018-06-18 14:23:20\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Multi-tissue transcriptome associated with feed efficiency in Nellore cattle\"],\n [{\n            'v': 588,\n            'f': \"588\",\n        },\n\"dcc_bley\",\n\"PRJEB27350\",\n\"Webin-49186\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 25,\n            'f': \"25\",\n        },\n\"2018-06-19 08:53:28\",\n{\n            'v': 48,\n            'f': \"48\",\n        },\n\"2018-07-15 08:35:56\",\n\"Salmonella compare pilot\"],\n [{\n            'v': 589,\n            'f': \"589\",\n        },\n\"dcc_cole\",\n\"PRJEB27356\",\n\"Webin-45990\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-07-05 06:38:23\",\n{\n            'v': 13,\n            'f': \"13\",\n        },\n\"2018-07-03 07:55:14\",\n\"Detection of food borne pathogens via Metagenomics approach\"],\n [{\n            'v': 590,\n            'f': \"590\",\n        },\n\"dcc_faang\",\n\"PRJEB27364\",\n\"Webin-46224\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2018-06-19 13:12:07\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"High-throughput chromosome conformation capture of liver cells in 4 livestock species by the FAANG pilot project FR-AgENCODE\"],\n [{\n            'v': 591,\n            'f': \"591\",\n        },\n\"dcc_faang\",\n\"PRJEB27455\",\n\"Webin-46224\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 41,\n            'f': \"41\",\n        },\n\"2018-06-25 16:44:53\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Transcriptome profiling of liver and T cells in 4 livestock species by the FAANG pilot project FR-AgENCODE\"],\n [{\n            'v': 592,\n            'f': \"592\",\n        },\n\"dcc_blake\",\n\"PRJEB27555\",\n\"Webin-46611\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-06-29 09:00:46\",\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2018-12-31 14:09:30\",\n\"Automatic upload of a(n) Salmonella enterica subsp. enterica serovar Enteritidis isolate, strain: 16EP001290.\"],\n [{\n            'v': 593,\n            'f': \"593\",\n        },\n\"dcc_bley\",\n\"PRJEB27555\",\n\"Webin-46611\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-06-29 09:00:46\",\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2018-12-31 14:09:30\",\n\"Automatic upload of a(n) Salmonella enterica subsp. enterica serovar Enteritidis isolate, strain: 16EP001290.\"],\n [{\n            'v': 594,\n            'f': \"594\",\n        },\n\"dcc_blake\",\n\"PRJEB27556\",\n\"Webin-46611\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-06-29 09:01:13\",\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2018-12-31 14:09:36\",\n\"Automatic upload of a(n) Salmonella enterica subsp. enterica serovar Enteritidis isolate, strain: 16EP001384.\"],\n [{\n            'v': 595,\n            'f': \"595\",\n        },\n\"dcc_bley\",\n\"PRJEB27556\",\n\"Webin-46611\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-06-29 09:01:13\",\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2018-12-31 14:09:36\",\n\"Automatic upload of a(n) Salmonella enterica subsp. enterica serovar Enteritidis isolate, strain: 16EP001384.\"],\n [{\n            'v': 596,\n            'f': \"596\",\n        },\n\"dcc_blake\",\n\"PRJEB27557\",\n\"Webin-46611\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-06-29 09:01:29\",\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2018-12-31 14:09:42\",\n\"Automatic upload of a(n) Salmonella enterica subsp. enterica serovar Enteritidis isolate, strain: 16EP001008.\"],\n [{\n            'v': 597,\n            'f': \"597\",\n        },\n\"dcc_bley\",\n\"PRJEB27557\",\n\"Webin-46611\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-06-29 09:01:29\",\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2018-12-31 14:09:42\",\n\"Automatic upload of a(n) Salmonella enterica subsp. enterica serovar Enteritidis isolate, strain: 16EP001008.\"],\n [{\n            'v': 598,\n            'f': \"598\",\n        },\n\"dcc_blake\",\n\"PRJEB27558\",\n\"Webin-46611\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-06-29 09:01:52\",\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2018-12-31 14:09:48\",\n\"Automatic upload of a(n) Salmonella enterica subsp. enterica serovar Enteritidis isolate, strain: 16EP002245.\"],\n [{\n            'v': 599,\n            'f': \"599\",\n        },\n\"dcc_bley\",\n\"PRJEB27558\",\n\"Webin-46611\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-06-29 09:01:52\",\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2018-12-31 14:09:48\",\n\"Automatic upload of a(n) Salmonella enterica subsp. enterica serovar Enteritidis isolate, strain: 16EP002245.\"],\n [{\n            'v': 600,\n            'f': \"600\",\n        },\n\"dcc_blake\",\n\"PRJEB27559\",\n\"Webin-46611\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-06-29 09:02:36\",\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2018-12-31 14:09:59\",\n\"Automatic upload of a(n) Salmonella enterica subsp. enterica serovar Enteritidis isolate, strain: 16EP000866.\"],\n [{\n            'v': 601,\n            'f': \"601\",\n        },\n\"dcc_bley\",\n\"PRJEB27559\",\n\"Webin-46611\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-06-29 09:02:36\",\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2018-12-31 14:09:59\",\n\"Automatic upload of a(n) Salmonella enterica subsp. enterica serovar Enteritidis isolate, strain: 16EP000866.\"],\n [{\n            'v': 602,\n            'f': \"602\",\n        },\n\"dcc_blake\",\n\"PRJEB27560\",\n\"Webin-46611\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-06-29 09:02:41\",\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2018-12-31 14:10:05\",\n\"Automatic upload of a(n) Salmonella enterica subsp. enterica serovar Enteritidis isolate, strain: 16EP001503.\"],\n [{\n            'v': 603,\n            'f': \"603\",\n        },\n\"dcc_bley\",\n\"PRJEB27560\",\n\"Webin-46611\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-06-29 09:02:41\",\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2018-12-31 14:10:05\",\n\"Automatic upload of a(n) Salmonella enterica subsp. enterica serovar Enteritidis isolate, strain: 16EP001503.\"],\n [{\n            'v': 604,\n            'f': \"604\",\n        },\n\"dcc_liszt\",\n\"PRJEB27621\",\n\"Webin-37120\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 183,\n            'f': \"183\",\n        },\n\"2022-06-10 10:44:56\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Global surveillance of antimicrobial resistance - Batch June 2017\"],\n [{\n            'v': 605,\n            'f': \"605\",\n        },\n\"dcc_cole\",\n\"PRJEB27653\",\n\"Webin-40760\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2018-07-06 12:23:49\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"A sample of smoked salmon, spiked with a variety of microbes has been sent to a number of labs for analysis\"],\n [{\n            'v': 606,\n            'f': \"606\",\n        },\n\"dcc_blake\",\n\"PRJEB27692\",\n\"Webin-50183\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 29,\n            'f': \"29\",\n        },\n\"2018-07-10 11:51:21\",\n{\n            'v': 58,\n            'f': \"58\",\n        },\n\"2018-12-31 14:13:37\",\n\"EFSA-ECDC compare pilot submission test repeated data\"],\n [{\n            'v': 607,\n            'f': \"607\",\n        },\n\"dcc_bley\",\n\"PRJEB28105\",\n\"Webin-46611\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-08-07 14:49:03\",\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-08-15 04:05:10\",\n\"Automatic upload of a(n) Salmonella enterica isolate, strain: C090558.\"],\n [{\n            'v': 608,\n            'f': \"608\",\n        },\n\"dcc_bley\",\n\"PRJEB28106\",\n\"Webin-46611\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-08-07 14:49:03\",\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-08-15 04:05:20\",\n\"Automatic upload of a(n) Salmonella enterica isolate, strain: C072077.\"],\n [{\n            'v': 609,\n            'f': \"609\",\n        },\n\"dcc_bley\",\n\"PRJEB28107\",\n\"Webin-46611\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-08-07 14:49:08\",\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-08-20 15:30:17\",\n\"Automatic upload of a(n) Salmonella enterica isolate, strain: 14068727.\"],\n [{\n            'v': 610,\n            'f': \"610\",\n        },\n\"dcc_bley\",\n\"PRJEB28108\",\n\"Webin-46611\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-08-07 14:49:12\",\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-08-21 04:35:37\",\n\"Automatic upload of a(n) Salmonella enterica isolate, strain: 14068907.\"],\n [{\n            'v': 611,\n            'f': \"611\",\n        },\n\"dcc_bley\",\n\"PRJEB28109\",\n\"Webin-46611\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-08-07 14:49:17\",\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-08-21 04:25:16\",\n\"Automatic upload of a(n) Salmonella enterica isolate, strain: C100310.\"],\n [{\n            'v': 612,\n            'f': \"612\",\n        },\n\"dcc_bley\",\n\"PRJEB28110\",\n\"Webin-46611\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-08-07 14:49:21\",\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-08-15 03:55:12\",\n\"Automatic upload of a(n) Salmonella enterica isolate, strain: 14055199.\"],\n [{\n            'v': 613,\n            'f': \"613\",\n        },\n\"dcc_bley\",\n\"PRJEB28111\",\n\"Webin-46611\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-08-07 14:49:38\",\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-08-15 04:10:14\",\n\"Automatic upload of a(n) Salmonella enterica isolate, strain: 17007603.\"],\n [{\n            'v': 614,\n            'f': \"614\",\n        },\n\"dcc_bley\",\n\"PRJEB28113\",\n\"Webin-46611\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-08-07 15:11:11\",\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-08-21 04:35:43\",\n\"Automatic upload of a(n) Salmonella enterica isolate, strain: C080480.\"],\n [{\n            'v': 615,\n            'f': \"615\",\n        },\n\"dcc_bley\",\n\"PRJEB28114\",\n\"Webin-46611\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-08-07 15:12:16\",\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-08-21 04:20:10\",\n\"Automatic upload of a(n) Salmonella enterica isolate, strain: C090733.\"],\n [{\n            'v': 616,\n            'f': \"616\",\n        },\n\"dcc_basie\",\n\"PRJEB28140\",\n\"Webin-49778\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 94,\n            'f': \"94\",\n        },\n\"2018-08-10 10:26:10\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"La Selva Biological Station (Costa Rica) Neotropical lowland rainforest soil DNA metabarcoding of the 18S V4 rRNA\"],\n [{\n            'v': 617,\n            'f': \"617\",\n        },\n\"dcc_cole\",\n\"PRJEB28170\",\n\"Webin-43561\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2018-08-10 12:16:34\",\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2019-03-15 01:45:11\",\n\"Ring Trial COMPARE project\"],\n [{\n            'v': 618,\n            'f': \"618\",\n        },\n\"dcc_faang\",\n\"PRJEB28219\",\n\"Webin-48466\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 68,\n            'f': \"68\",\n        },\n\"2018-08-15 00:53:46\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Bovine small and micro RNA expression atlas\"],\n [{\n            'v': 619,\n            'f': \"619\",\n        },\n\"dcc_beethoven\",\n\"PRJEB2822\",\n\"Webin-2\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 606,\n            'f': \"606\",\n        },\n\"2015-05-12 09:14:25\",\n{\n            'v': 1722,\n            'f': \"1722\",\n        },\n\"2018-09-05 04:40:12\",\n\"Combining_VIDISCA_virus_discovery_with_Illumina__Deep_Sequencing\"],\n [{\n            'v': 620,\n            'f': \"620\",\n        },\n\"dcc_blake\",\n\"PRJEB28249\",\n\"Webin-49186\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 46,\n            'f': \"46\",\n        },\n\"2023-10-25 10:44:39\",\n{\n            'v': 66,\n            'f': \"66\",\n        },\n\"2018-08-28 09:10:10\",\n\"FOHM data share\"],\n [{\n            'v': 621,\n            'f': \"621\",\n        },\n\"dcc_bley\",\n\"PRJEB28249\",\n\"Webin-49186\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 46,\n            'f': \"46\",\n        },\n\"2023-10-25 10:44:39\",\n{\n            'v': 66,\n            'f': \"66\",\n        },\n\"2018-08-28 09:10:10\",\n\"FOHM data share\"],\n [{\n            'v': 622,\n            'f': \"622\",\n        },\n\"dcc_bley\",\n\"PRJEB28548\",\n\"Webin-49910\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 21,\n            'f': \"21\",\n        },\n\"2018-09-10 18:59:18\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"COMPARE_ISCIII_1\"],\n [{\n            'v': 623,\n            'f': \"623\",\n        },\n\"dcc_faang\",\n\"PRJEB28653\",\n\"Webin-50564\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 70,\n            'f': \"70\",\n        },\n\"2018-09-17 00:42:07\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Effect of maternal nutrition in late gestation in the blood and skeletal muscle transcriptome of calves\"],\n [{\n            'v': 624,\n            'f': \"624\",\n        },\n\"dcc_cole\",\n\"PRJEB28753\",\n\"Webin-49568\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-11-21 06:43:40\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"COMPARE food metagenomic ring trial\"],\n [{\n            'v': 625,\n            'f': \"625\",\n        },\n\"dcc_bromhead\",\n\"PRJEB29116\",\n\"Webin-49436\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 124,\n            'f': \"124\",\n        },\n\"2018-10-17 09:10:29\",\n\"Sequences of MCR postive isolates from veterinary samples in the Netherlands.\"],\n [{\n            'v': 626,\n            'f': \"626\",\n        },\n\"dcc_culbertson\",\n\"PRJEB30101\",\n\"Webin-23618\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2019-01-31 13:37:13\",\n\"Bovine Plasmidome - a novel class of infectious agents linked to common human cancers and chronic diseases\"],\n [{\n            'v': 627,\n            'f': \"627\",\n        },\n\"dcc_schumann\",\n\"PRJEB30665\",\n\"Webin-43561\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 32,\n            'f': \"32\",\n        },\n\"2019-01-28 16:19:07\",\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2019-03-20 11:47:11\",\n\"Enteric virus diversity in sewages collected during last decade\"],\n [{\n            'v': 628,\n            'f': \"628\",\n        },\n\"dcc_faang\",\n\"PRJEB31481\",\n\"Webin-42643\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 24,\n            'f': \"24\",\n        },\n\"2019-02-28 16:44:56\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNA-seq of pig alveolar macrophages stimulated with LPS and Poly (I:C)\"],\n [{\n            'v': 629,\n            'f': \"629\",\n        },\n\"dcc_faang\",\n\"PRJEB31482\",\n\"Webin-42643\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 20,\n            'f': \"20\",\n        },\n\"2019-02-28 17:21:43\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Pig histone modification alias\"],\n [{\n            'v': 630,\n            'f': \"630\",\n        },\n\"dcc_faang\",\n\"PRJEB31483\",\n\"Webin-42643\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 72,\n            'f': \"72\",\n        },\n\"2020-02-27 15:09:08\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Pig histone marks\"],\n [{\n            'v': 631,\n            'f': \"631\",\n        },\n\"dcc_buckner\",\n\"PRJEB32129\",\n\"Webin-52471\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2019-04-13 14:13:50\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"COMPARE FFPE Ring trial\"],\n [{\n            'v': 632,\n            'f': \"632\",\n        },\n\"dcc_buckner\",\n\"PRJEB32130\",\n\"Webin-52447\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2019-04-15 09:16:31\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"COMPARE Ring Trial 2019 on FFPE material\"],\n [{\n            'v': 633,\n            'f': \"633\",\n        },\n\"dcc_buckner\",\n\"PRJEB32136\",\n\"Webin-49086\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2019-04-23 08:47:36\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"COMPARE Metagenomic ring trial 2019 [Anses]\"],\n [{\n            'v': 634,\n            'f': \"634\",\n        },\n\"dcc_buckner\",\n\"PRJEB32138\",\n\"Webin-40757\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2019-04-12 08:54:34\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Comparison of the sequencing of 2 FFPE samples\"],\n [{\n            'v': 635,\n            'f': \"635\",\n        },\n\"dcc_buckner\",\n\"PRJEB32142\",\n\"Webin-14569\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2019-04-16 16:43:44\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Ring Trial 2019 on formalin-fixed paraffin-embedded (FFPE) material\"],\n [{\n            'v': 636,\n            'f': \"636\",\n        },\n\"dcc_buckner\",\n\"PRJEB32143\",\n\"Webin-29236\",\n\"2024-06-15 00:00:00\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 8,\n            'f': \"8\",\n        },\n\"2019-04-12 14:32:24\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"FLI COMPARE 2019 FFPE Proficiency Test\"],\n [{\n            'v': 637,\n            'f': \"637\",\n        },\n\"dcc_buckner\",\n\"PRJEB32144\",\n\"Webin-37120\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2019-04-12 11:36:15\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Formalin-fixed paraffin-embedded (FFPE) ring trial\"],\n [{\n            'v': 638,\n            'f': \"638\",\n        },\n\"dcc_buckner\",\n\"PRJEB32147\",\n\"Webin-40760\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2019-04-12 13:45:41\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Metagenomic sequencing of FFPE samples distributed to a number of labs for diganosis of veterinary pathogens\"],\n [{\n            'v': 639,\n            'f': \"639\",\n        },\n\"dcc_buckner\",\n\"PRJEB32356\",\n\"Webin-45990\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2019-04-30 14:23:26\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Ring Trial\"],\n [{\n            'v': 640,\n            'f': \"640\",\n        },\n\"dcc_corea\",\n\"PRJEB32420\",\n\"Webin-52639\",\nNaN,\n{\n            'v': 3,\n            'f': \"3\",\n        },\n{\n            'v': 60,\n            'f': \"60\",\n        },\n\"2019-05-04 10:43:00\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"TEST STUDY - Tara Pacific nanopore experiment - Data Hub Demonstrator\"],\n [{\n            'v': 641,\n            'f': \"641\",\n        },\n\"dcc_faang\",\n\"PRJEB32645\",\n\"Webin-49517\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 20,\n            'f': \"20\",\n        },\n\"2019-05-17 19:18:07\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"FAANG RRBS Analysis of the horse\"],\n [{\n            'v': 642,\n            'f': \"642\",\n        },\n\"dcc_bromhead\",\n\"PRJEB33169\",\n\"Webin-37120\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 399,\n            'f': \"399\",\n        },\n\"2019-06-24 14:46:22\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"EFSA_EURL-AR_Confirmatory_Testing_2017\"],\n [{\n            'v': 643,\n            'f': \"643\",\n        },\n\"dcc_faang\",\n\"PRJEB33476\",\n\"Webin-52795\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2019-07-11 12:54:09\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Adult Churra sheep abomasal lymphnode transcriptome\"],\n [{\n            'v': 644,\n            'f': \"644\",\n        },\n\"dcc_grusin\",\n\"PRJEB33797\",\n\"Webin-53556\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2019-07-31 17:20:02\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Nanopore long and direct RNA-seq of coronaviruses\"],\n [{\n            'v': 645,\n            'f': \"645\",\n        },\n\"dcc_byard\",\n\"PRJEB34029\",\n\"Webin-52819\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2019-08-15 16:31:00\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Viral metagenomics for pathogen detection in CSF and plasma\"],\n [{\n            'v': 646,\n            'f': \"646\",\n        },\n\"dcc_byard\",\n\"PRJEB34030\",\n\"Webin-52819\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2019-08-15 16:58:59\",\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2019-11-29 17:15:34\",\n\"The respiratory virome and exacerbations in patients with chronic obstructive pulmonary disease\"],\n [{\n            'v': 647,\n            'f': \"647\",\n        },\n\"dcc_byard\",\n\"PRJEB34031\",\n\"Webin-52819\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2019-08-15 17:11:53\",\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2019-11-29 17:15:56\",\n\"Metagenomic sequencing for combined detection of RNA and DNA viruses in respiratory samples from paediatric patients\"],\n [{\n            'v': 648,\n            'f': \"648\",\n        },\n\"dcc_grusin\",\n\"PRJEB34079\",\n\"Webin-41902\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2019-08-22 09:42:42\",\n{\n            'v': 23,\n            'f': \"23\",\n        },\n\"2019-09-20 16:55:55\",\n\"Swedish dogs with canine infectious respiratory disease highlight establishment of canine respiratory coronavirus in Sweden, and recent evolutionary history with bovine coronavirus\"],\n [{\n            'v': 649,\n            'f': \"649\",\n        },\n\"dcc_faang\",\n\"PRJEB34570\",\n\"Webin-43213\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 192,\n            'f': \"192\",\n        },\n\"2019-09-25 10:02:00\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Biological network approach for the identification of regulatory long non-coding RNAs associated with metabolic efficiency in cattle\"],\n [{\n            'v': 650,\n            'f': \"650\",\n        },\n\"dcc_faang\",\n\"PRJEB34864\",\n\"Webin-52302\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 56,\n            'f': \"56\",\n        },\n\"2019-10-14 13:57:59\",\n{\n            'v': 56,\n            'f': \"56\",\n        },\n\"2020-05-22 08:57:49\",\n\"CAGE-Sequencing of tissues from Benz 2616\"],\n [{\n            'v': 651,\n            'f': \"651\",\n        },\n\"dcc_faang\",\n\"PRJEB35127\",\n\"Webin-48466\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1536,\n            'f': \"1536\",\n        },\n\"2019-10-28 22:15:40\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Bovine gene expression atlas\"],\n [{\n            'v': 652,\n            'f': \"652\",\n        },\n\"dcc_faang\",\n\"PRJEB35292\",\n\"Webin-54321\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 32,\n            'f': \"32\",\n        },\n\"2019-12-11 18:18:04\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Benz2616 Rambouillet sheep mRNA sequencing\"],\n [{\n            'v': 653,\n            'f': \"653\",\n        },\n\"dcc_faang\",\n\"PRJEB35307\",\n\"Webin-54459\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 80,\n            'f': \"80\",\n        },\n\"2019-11-09 22:20:49\",\n{\n            'v': 204,\n            'f': \"204\",\n        },\n\"2022-11-07 11:31:11\",\n\"Functionally annotating regulatory elements in the equine genome using histone mark ChIP-Seq\"],\n [{\n            'v': 654,\n            'f': \"654\",\n        },\n\"dcc_drew\",\n\"PRJEB35420\",\n\"Webin-40757\",\n\"2025-10-17 00:00:00\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2019-11-28 11:52:09\",\n\"NoroNet data and visualisation\"],\n [{\n            'v': 655,\n            'f': \"655\",\n        },\n\"dcc_faang\",\n\"PRJEB36963\",\n\"Webin-42643\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Pig histone marks\"],\n [{\n            'v': 656,\n            'f': \"656\",\n        },\n\"dcc_faang\",\n\"PRJEB37103\",\n\"Webin-42643\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 20,\n            'f': \"20\",\n        },\n\"2020-03-05 22:16:36\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Pig fetal histone modification altas\"],\n [{\n            'v': 657,\n            'f': \"657\",\n        },\n\"dcc_grusin\",\n\"PRJEB37513\",\n\"Webin-55517\",\nNaN,\n{\n            'v': 5,\n            'f': \"5\",\n        },\n{\n            'v': 2010,\n            'f': \"2010\",\n        },\n\"2021-08-09 11:13:03\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WGS of SARS-CoV-2 circulating in Spain\"],\n [{\n            'v': 658,\n            'f': \"658\",\n        },\n\"dcc_faang\",\n\"PRJEB37735\",\n\"Webin-55550\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 77,\n            'f': \"77\",\n        },\n\"2020-06-11 20:16:07\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Identification of genome-wide regulatory elements of gut tissues in livestock species\"],\n [{\n            'v': 659,\n            'f': \"659\",\n        },\n\"dcc_grusin\",\n\"PRJEB37886\",\n\"Webin-55551\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2689118,\n            'f': \"2689118\",\n        },\n\"2023-10-24 12:18:53\",\n{\n            'v': 2804752,\n            'f': \"2804752\",\n        },\nNaN,\n\"COVID-19 Genomics UK (COG-UK) consortium\"],\n [{\n            'v': 660,\n            'f': \"660\",\n        },\n\"dcc_grusin\",\n\"PRJEB37966\",\n\"Webin-41583\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 752,\n            'f': \"752\",\n        },\n\"2020-11-08 13:53:31\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Corona virus sequencing in Denmark\"],\n [{\n            'v': 661,\n            'f': \"661\",\n        },\n\"dcc_grusin\",\n\"PRJEB38101\",\n\"Webin-51221\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 6,\n            'f': \"6\",\n        },\n\"2020-10-16 22:08:43\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Whole Genome Sequencing of SARS-CoV-2 virus isolated in Brescia (Italy)\"],\n [{\n            'v': 662,\n            'f': \"662\",\n        },\n\"dcc_grusin\",\n\"PRJEB38351\",\n\"Webin-49415\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 6,\n            'f': \"6\",\n        },\n\"2020-05-16 23:09:58\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) genome sequencing after in vitro culture\"],\n [{\n            'v': 663,\n            'f': \"663\",\n        },\n\"dcc_grusin\",\n\"PRJEB38369\",\n\"Webin-34550\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2020-06-16 11:39:30\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Comparing library preparation methods for SARS-CoV-2 multiplex amplicon sequencing on the Illumina MiSeq platform\"],\n [{\n            'v': 664,\n            'f': \"664\",\n        },\n\"dcc_grusin\",\n\"PRJEB38388\",\n\"Webin-40757\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 584,\n            'f': \"584\",\n        },\n\"2020-05-29 20:16:58\",\n{\n            'v': 584,\n            'f': \"584\",\n        },\n\"2020-08-21 22:32:27\",\n\"Rapid SARS-CoV-2 whole genome sequencing for informed public health decision making in the Netherlands\"],\n [{\n            'v': 665,\n            'f': \"665\",\n        },\n\"dcc_grusin\",\n\"PRJEB38459\",\n\"Webin-1344\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2020-05-23 17:36:13\",\n{\n            'v': 892,\n            'f': \"892\",\n        },\n\"2021-11-18 16:50:56\",\n\"Saudi SARS-CoV-2\"],\n [{\n            'v': 666,\n            'f': \"666\",\n        },\n\"dcc_grusin\",\n\"PRJEB38546\",\n\"Webin-46012\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 52,\n            'f': \"52\",\n        },\n\"2020-10-16 16:11:22\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 variant tracing within the first COVID-19 clusters in Northern Germany\"],\n [{\n            'v': 667,\n            'f': \"667\",\n        },\n\"dcc_faang\",\n\"PRJEB38599\",\n\"Webin-56073\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 16,\n            'f': \"16\",\n        },\n\"2020-06-03 21:12:31\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Transcriptome analysis in bursa tissue of chicken lines under NDV infection and heat stress\"],\n [{\n            'v': 668,\n            'f': \"668\",\n        },\n\"dcc_faang\",\n\"PRJEB38600\",\n\"Webin-56073\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 40,\n            'f': \"40\",\n        },\n\"2020-06-03 21:13:48\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Histone modification analysis in bursa tissue from two chicken lines under NDV infection and heat stress\"],\n [{\n            'v': 669,\n            'f': \"669\",\n        },\n\"dcc_grusin\",\n\"PRJEB38723\",\n\"Webin-55756\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 542,\n            'f': \"542\",\n        },\n\"2020-06-16 21:05:20\",\n{\n            'v': 540,\n            'f': \"540\",\n        },\n\"2020-06-18 11:05:22\",\n\"Large scale multiplexing of SARS-CoV-2 genomes using nanopore sequencing\"],\n [{\n            'v': 670,\n            'f': \"670\",\n        },\n\"dcc_grusin\",\n\"PRJEB38744\",\n\"Webin-47721\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 73,\n            'f': \"73\",\n        },\n\"2020-06-18 14:34:25\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Multi-level proteomics reveals host-perturbation strategies of SARS-CoV-2 and SARS-CoV.\"],\n [{\n            'v': 671,\n            'f': \"671\",\n        },\n\"dcc_grusin\",\n\"PRJEB38935\",\n\"Webin-24\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 20,\n            'f': \"20\",\n        },\n\"2020-06-18 05:03:28\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNA-seq of RBC depleted whole blood from COVID-19 patients and controls\"],\n [{\n            'v': 672,\n            'f': \"672\",\n        },\n\"dcc_walton\",\n\"PRJEB39014\",\n\"Webin-40757\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 10825,\n            'f': \"10825\",\n        },\n\"2023-09-14 14:57:17\",\n{\n            'v': 19650,\n            'f': \"19650\",\n        },\n\"2022-02-28 12:03:29\",\n\"Continued SARS-CoV-2 whole-genome sequencing for outbreak tracking and tracing in the Netherlands\"],\n [{\n            'v': 673,\n            'f': \"673\",\n        },\n\"dcc_faang\",\n\"PRJEB39178\",\n\"Webin-56219\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8,\n            'f': \"8\",\n        },\n\"2020-07-02 03:58:31\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Tissue specific whole genome bisulphite sequencing of Rambouillet sheep Benz2616\"],\n [{\n            'v': 674,\n            'f': \"674\",\n        },\n\"dcc_grusin\",\n\"PRJEB39337\",\n\"Webin-332\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-11-09 15:24:39\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"A single timepoint extraction of total RNA to allow direct mRNA sequencing of infected cells on a nanopore device\"],\n [{\n            'v': 675,\n            'f': \"675\",\n        },\n\"dcc_grusin\",\n\"PRJEB39487\",\n\"Webin-56512\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 339,\n            'f': \"339\",\n        },\n\"2020-07-22 18:13:48\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Evolution and epidemic spread of SARS-CoV-2 in Brazil\"],\n [{\n            'v': 676,\n            'f': \"676\",\n        },\n\"dcc_grusin\",\n\"PRJEB39632\",\n\"Webin-56529\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2020-07-29 14:50:09\",\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2020-08-07 09:42:08\",\n\"SARS-CoV-2 genomes from late April in Stockholm, Sweden\"],\n [{\n            'v': 677,\n            'f': \"677\",\n        },\n\"dcc_grusin\",\n\"PRJEB39737\",\n\"Webin-42222\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-08-11 13:07:59\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"In situ structural analysis of SARS-CoV-2 spike reveals flexibility mediated by three hinges\"],\n [{\n            'v': 678,\n            'f': \"678\",\n        },\n\"dcc_grusin\",\n\"PRJEB39849\",\n\"Webin-56698\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 420,\n            'f': \"420\",\n        },\n\"2020-10-23 09:46:19\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Deep whole-genome sequencing of SARS-CoV-2 patient samples from Austria.\"],\n [{\n            'v': 679,\n            'f': \"679\",\n        },\n\"dcc_grusin\",\n\"PRJEB39887\",\n\"Webin-38821\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 468,\n            'f': \"468\",\n        },\n\"2020-09-14 18:21:46\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 phylogeny during the early outbreak in the Basel area, Switzerland: import and spread dominated by a single B.1 lineage variant (C15324T)\"],\n [{\n            'v': 680,\n            'f': \"680\",\n        },\n\"dcc_grusin\",\n\"PRJEB40188\",\n\"Webin-56871\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2921,\n            'f': \"2921\",\n        },\n\"2022-02-02 17:45:06\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Latvian SARS-CoV-2 isolate genetic diveristy\"],\n [{\n            'v': 681,\n            'f': \"681\",\n        },\n\"dcc_tara\",\n\"PRJEB402\",\n\"Webin-261\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Tara-oceans samples barcoding and shotgun sequencing\"],\n [{\n            'v': 682,\n            'f': \"682\",\n        },\n\"dcc_grusin\",\n\"PRJEB40277\",\n\"Webin-56907\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 89904,\n            'f': \"89904\",\n        },\n\"2023-06-12 12:06:24\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Interinstitutional effort to sequence SARS-CoV-2 circulating genomes in Ireland during the Covid-19 global pandemic.\"],\n [{\n            'v': 683,\n            'f': \"683\",\n        },\n\"dcc_grusin\",\n\"PRJEB40394\",\n\"Webin-50425\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 48,\n            'f': \"48\",\n        },\n\"2020-09-30 12:22:10\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 spike cleavage and transmission\"],\n [{\n            'v': 684,\n            'f': \"684\",\n        },\n\"dcc_grusin\",\n\"PRJEB40443\",\n\"Webin-57030\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 346,\n            'f': \"346\",\n        },\n\"2020-09-24 12:24:33\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Deep sequencing and phylogenetic analysis of SARScov2 strains circulating in North Italy from February to April 2020\"],\n [{\n            'v': 685,\n            'f': \"685\",\n        },\n\"dcc_faang\",\n\"PRJEB40528\",\n\"Webin-56843\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2020-09-27 22:00:46\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ChIP-seq in sheep alveolar macrophages\"],\n [{\n            'v': 686,\n            'f': \"686\",\n        },\n\"dcc_faang\",\n\"PRJEB40576\",\n\"Webin-56947\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 65,\n            'f': \"65\",\n        },\n\"2020-09-30 14:30:19\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genome 3D structure in pig fetal muscle from Hi-C assays of Chromatin Conformation Capture\"],\n [{\n            'v': 687,\n            'f': \"687\",\n        },\n\"dcc_grusin\",\n\"PRJEB40692\",\n\"Webin-24\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 14,\n            'f': \"14\",\n        },\n\"2020-10-06 15:19:09\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Functional genomic screens in human cells to identify host factors for SARS-CoV-2 and common cold coronaviruses\"],\n [{\n            'v': 688,\n            'f': \"688\",\n        },\n\"dcc_grusin\",\n\"PRJEB40711\",\n\"Webin-56948\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 39,\n            'f': \"39\",\n        },\n\"2020-10-07 21:44:46\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Whole-genome sequencing of SARS-CoV-2 from Covid-19 patients from Belgium\"],\n [{\n            'v': 689,\n            'f': \"689\",\n        },\n\"dcc_grusin\",\n\"PRJEB40750\",\n\"Webin-55218\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 25,\n            'f': \"25\",\n        },\n\"2021-04-26 18:52:03\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"25 SARS-CoV-2 sequenced Romanian viral strains, from the first two months of the pandemic\"],\n [{\n            'v': 690,\n            'f': \"690\",\n        },\n\"dcc_grusin\",\n\"PRJEB40971\",\n\"Webin-27196\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 6,\n            'f': \"6\",\n        },\n\"2020-12-17 13:06:40\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genome sequencing of SARS-CoV-2 isolated in Malaysia\"],\n [{\n            'v': 691,\n            'f': \"691\",\n        },\n\"dcc_grusin\",\n\"PRJEB40972\",\n\"Webin-52935\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1369,\n            'f': \"1369\",\n        },\n\"2021-01-20 16:05:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Sub-genomic RNA identification in SARS-CoV-2 Genomic Sequencing Data\"],\n [{\n            'v': 692,\n            'f': \"692\",\n        },\n\"dcc_grusin\",\n\"PRJEB41002\",\n\"Webin-34\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1487,\n            'f': \"1487\",\n        },\n\"2020-10-29 14:41:25\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Evaluating the microbial context of SARS-CoV-2 in patients and the hospital built environment\"],\n [{\n            'v': 693,\n            'f': \"693\",\n        },\n\"dcc_bonner\",\n\"PRJEB41074\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 383,\n            'f': \"383\",\n        },\n\"2020-11-02 16:59:39\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS4_UL20_Stress_HepG2_WT_TempoSeq_24hTP raw data (EUT007, EUT008)\"],\n [{\n            'v': 694,\n            'f': \"694\",\n        },\n\"dcc_faang\",\n\"PRJEB41079\",\n\"Webin-57361\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 28,\n            'f': \"28\",\n        },\n\"2020-11-02 20:01:16\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CTCF ChIP-seq for equine FAANG project\"],\n [{\n            'v': 695,\n            'f': \"695\",\n        },\n\"dcc_grusin\",\n\"PRJEB41216\",\n\"Webin-57339\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-11-12 11:44:19\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 was grown by inoculation of VeroE6 cells with 200 \\u00b5l nasopharyngeal swab of a COVID-19 patient.\"],\n [{\n            'v': 696,\n            'f': \"696\",\n        },\n\"dcc_faang\",\n\"PRJEB41317\",\n\"Webin-57361\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 16,\n            'f': \"16\",\n        },\n\"2020-11-13 03:30:26\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Equine FAANG ATAC-seq Pilot\"],\n [{\n            'v': 697,\n            'f': \"697\",\n        },\n\"dcc_faang\",\n\"PRJEB41457\",\n\"Webin-54321\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 69,\n            'f': \"69\",\n        },\n\"2020-11-30 23:30:53\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ChIP-seq in sheep liver, spleen, and cerebellum\"],\n [{\n            'v': 698,\n            'f': \"698\",\n        },\n\"dcc_bonner\",\n\"PRJEB41470\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1154,\n            'f': \"1154\",\n        },\n\"2020-11-23 10:56:11\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WP9_UL48_TGX_PHH_50donor_S1500+v2_24hTP raw data (EUT024)\"],\n [{\n            'v': 699,\n            'f': \"699\",\n        },\n\"dcc_faang\",\n\"PRJEB41485\",\n\"Webin-45927\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 24,\n            'f': \"24\",\n        },\n\"2020-11-23 16:01:17\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Roslin Pig Muscle ATAC-seq Project\"],\n [{\n            'v': 700,\n            'f': \"700\",\n        },\n\"dcc_faang\",\n\"PRJEB41488\",\n\"Webin-45927\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 26,\n            'f': \"26\",\n        },\n\"2020-11-23 16:21:16\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Roslin Pig Muscle RNA-Seq Project\"],\n [{\n            'v': 701,\n            'f': \"701\",\n        },\n\"dcc_bonner\",\n\"PRJEB41491\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 345,\n            'f': \"345\",\n        },\n\"2020-11-24 09:52:15\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS4_VU3_MitoTox_RT_TempOSeq_24h raw data (EUT011b)\"],\n [{\n            'v': 702,\n            'f': \"702\",\n        },\n\"dcc_bonner\",\n\"PRJEB41499\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 234,\n            'f': \"234\",\n        },\n\"2020-11-24 12:37:23\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CSY/WP7_TempoSeq_Neuro2D_3D_iPSN_72h raw data (EUT042)\"],\n [{\n            'v': 703,\n            'f': \"703\",\n        },\n\"dcc_bonner\",\n\"PRJEB41566\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 764,\n            'f': \"764\",\n        },\n\"2020-11-27 10:55:41\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WP5_TempoSeq-BioSpyder raw data (EUT060)\"],\n [{\n            'v': 704,\n            'f': \"704\",\n        },\n\"dcc_faang\",\n\"PRJEB41576\",\n\"Webin-43213\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2020-11-27 15:26:20\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Single-cell RNA sequencing of milk cells and primary mammary epithelial cells\"],\n [{\n            'v': 705,\n            'f': \"705\",\n        },\n\"dcc_bonner\",\n\"PRJEB41582\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 144,\n            'f': \"144\",\n        },\n\"2020-11-27 18:06:29\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WP7_VU3d_WP5_RT_TempOSeq_24h raw data (EUT011a)\"],\n [{\n            'v': 706,\n            'f': \"706\",\n        },\n\"dcc_bonner\",\n\"PRJEB41589\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 341,\n            'f': \"341\",\n        },\n\"2020-11-30 11:13:25\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS1_VU3e_VPA_RT_TempOSeq_24h raw data (EUT027)\"],\n [{\n            'v': 707,\n            'f': \"707\",\n        },\n\"dcc_bonner\",\n\"PRJEB41627\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 363,\n            'f': \"363\",\n        },\n\"2020-12-01 10:04:12\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WP7_WP7_VU3c_Renal_RT_TempOSeq_24h raw data (EUT062)\"],\n [{\n            'v': 708,\n            'f': \"708\",\n        },\n\"dcc_bonner\",\n\"PRJEB41641\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 327,\n            'f': \"327\",\n        },\n\"2020-12-01 14:42:25\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CSY/WP7_Swetox8b_Neuro_SH_Diff_3d_TempO-Seq_Exp72h raw data (EUT026)\"],\n [{\n            'v': 709,\n            'f': \"709\",\n        },\n\"dcc_bonner\",\n\"PRJEB41670\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 380,\n            'f': \"380\",\n        },\n\"2020-12-02 12:43:12\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS4_Swetox8a_Neuro_SH_Diff_6d_TempO-Seq_Exp24h raw data (EUT013)\"],\n [{\n            'v': 710,\n            'f': \"710\",\n        },\n\"dcc_grusin\",\n\"PRJEB41708\",\n\"Webin-57654\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 28,\n            'f': \"28\",\n        },\n\"2021-02-12 14:04:59\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Determining the strain variability of the novel CoV-2 and underlying genetic predisposition of the host as a discovery tool for virulence and risk factors in COVID-19 pandemic\"],\n [{\n            'v': 711,\n            'f': \"711\",\n        },\n\"dcc_bonner\",\n\"PRJEB41710\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 360,\n            'f': \"360\",\n        },\n\"2020-12-03 09:55:55\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WP7_UL49_TGX_HepG2_BAC-GFP_CHOP_S1500+v2_24hTP raw data (EUT039)\"],\n [{\n            'v': 712,\n            'f': \"712\",\n        },\n\"dcc_bonner\",\n\"PRJEB41723\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 897,\n            'f': \"897\",\n        },\n\"2020-12-03 21:36:17\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS1/CS5_UL20_TGX_HepG2_WT_S1500+v2_24hTP raw data (EUT037)\"],\n [{\n            'v': 713,\n            'f': \"713\",\n        },\n\"dcc_bonner\",\n\"PRJEB41731\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 172,\n            'f': \"172\",\n        },\n\"2020-12-04 11:39:48\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS1_UL20_TGX_HepG2_WT_S1500+v2_24hTP raw data (EUT057)\"],\n [{\n            'v': 714,\n            'f': \"714\",\n        },\n\"dcc_bonner\",\n\"PRJEB41776\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 169,\n            'f': \"169\",\n        },\n\"2020-12-08 17:33:02\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS4_UL20_Stress_HepG2_WT_TempoSeq_24hTP raw data (EUT058)\"],\n [{\n            'v': 715,\n            'f': \"715\",\n        },\n\"dcc_bonner\",\n\"PRJEB41777\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 40,\n            'f': \"40\",\n        },\n\"2020-12-08 18:41:48\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WP6_HepG2-OX_HepG2_Mock-DMSO_conc0_TP7 raw data (EUT001)\"],\n [{\n            'v': 716,\n            'f': \"716\",\n        },\n\"dcc_green\",\n\"PRJEB41789\",\n\"Webin-54638\",\n\"2024-03-29 00:00:00\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 20,\n            'f': \"20\",\n        },\n\"2020-12-11 11:23:57\",\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2021-02-24 13:53:12\",\n\"SARS-CoV-2 viral data held in Spain, under the public project PRJEB37513.\"],\n [{\n            'v': 717,\n            'f': \"717\",\n        },\n\"dcc_faang\",\n\"PRJEB41822\",\n\"Webin-31385\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 63,\n            'f': \"63\",\n        },\n\"2020-12-09 17:14:44\",\n{\n            'v': 106,\n            'f': \"106\",\n        },\n\"2022-05-23 15:30:58\",\n\"GENE-SWITCH Pig DNA methylation with Reduced representation bisulfite sequencing (RRBS)\"],\n [{\n            'v': 718,\n            'f': \"718\",\n        },\n\"dcc_grusin\",\n\"PRJEB41826\",\n\"Webin-55756\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 100,\n            'f': \"100\",\n        },\n\"2020-12-09 18:46:24\",\n{\n            'v': 100,\n            'f': \"100\",\n        },\n\"2020-12-09 18:46:32\",\n\"Genomic epidemiology of SARS-CoV-2 epidemic in Zimbabwe: Role of regional migration and spread\"],\n [{\n            'v': 719,\n            'f': \"719\",\n        },\n\"dcc_faang\",\n\"PRJEB41829\",\n\"Webin-31385\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 67,\n            'f': \"67\",\n        },\n\"2022-01-24 14:01:39\",\n{\n            'v': 21,\n            'f': \"21\",\n        },\n\"2022-05-23 15:30:59\",\n\"GENE-SWITCH Chicken DNA methylation with Reduced representation bisulfite sequencing (RRBS)\"],\n [{\n            'v': 720,\n            'f': \"720\",\n        },\n\"dcc_bonner\",\n\"PRJEB41833\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 137,\n            'f': \"137\",\n        },\n\"2020-12-10 10:33:57\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WP6_UL20_TGX_HepG2_WT_S1500+v2_24hTP_01 raw data (EUT003)\"],\n [{\n            'v': 721,\n            'f': \"721\",\n        },\n\"dcc_bonner\",\n\"PRJEB41850\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 40,\n            'f': \"40\",\n        },\n\"2020-12-10 15:08:10\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WP6_UL78_TGX_PHH_OXsiRNA_S1500+v2_7hTP raw data (EUT001)\"],\n [{\n            'v': 722,\n            'f': \"722\",\n        },\n\"dcc_bonner\",\n\"PRJEB41872\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 59,\n            'f': \"59\",\n        },\n\"2020-12-11 13:19:39\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WP6_UL77_TGX_PHH_UPRsiRNA_S1500+v2_16hTP raw data (EUT001)\"],\n [{\n            'v': 723,\n            'f': \"723\",\n        },\n\"dcc_bonner\",\n\"PRJEB41929\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 58,\n            'f': \"58\",\n        },\n\"2020-12-11 16:16:27\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WP6_HepG2-UPR_HepG2_Mock-Tun_conc6_TP16 raw data (EUT001)\"],\n [{\n            'v': 724,\n            'f': \"724\",\n        },\n\"dcc_faang\",\n\"PRJEB41939\",\n\"Webin-48466\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 543,\n            'f': \"543\",\n        },\n\"2020-12-14 02:14:20\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Putative causal variants are enriched in annotated functional regions from 6 bovine tissues\"],\n [{\n            'v': 725,\n            'f': \"725\",\n        },\n\"dcc_faang\",\n\"PRJEB41970\",\n\"Webin-57578\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 84,\n            'f': \"84\",\n        },\n\"2020-12-14 15:29:45\",\n{\n            'v': 183,\n            'f': \"183\",\n        },\n\"2022-08-10 09:13:45\",\n\"GENE-SWITCH Pig transcriptome and gene expression atlas (mRNA-seq)\"],\n [{\n            'v': 726,\n            'f': \"726\",\n        },\n\"dcc_bonner\",\n\"PRJEB41973\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 103,\n            'f': \"103\",\n        },\n\"2021-03-29 14:31:43\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WP6_UL79_TGX_hiPSC-hep_S1500+v2_8hTP raw data (EUT001)\"],\n [{\n            'v': 727,\n            'f': \"727\",\n        },\n\"dcc_bonner\",\n\"PRJEB41977\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 137,\n            'f': \"137\",\n        },\n\"2020-12-14 17:25:28\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WP6_UL52_TGX_HepG2_WT_S1500+v2_8hTP_01 raw data (EUT003)\"],\n [{\n            'v': 728,\n            'f': \"728\",\n        },\n\"dcc_faang\",\n\"PRJEB42001\",\n\"Webin-57578\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 84,\n            'f': \"84\",\n        },\n\"2020-12-15 12:57:42\",\n{\n            'v': 420,\n            'f': \"420\",\n        },\n\"2022-03-16 16:17:43\",\n\"GENE-SWitCH Pig transcriptome and gene expression atlas (smallRNA-seq)\"],\n [{\n            'v': 729,\n            'f': \"729\",\n        },\n\"dcc_faang\",\n\"PRJEB42025\",\n\"Webin-57578\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 86,\n            'f': \"86\",\n        },\n\"2022-10-18 10:46:56\",\n{\n            'v': 183,\n            'f': \"183\",\n        },\n\"2022-11-07 09:11:24\",\n\"GENE-SWITCH Chicken transcriptome and gene expression atlas (mRNA-seq)\"],\n [{\n            'v': 730,\n            'f': \"730\",\n        },\n\"dcc_faang\",\n\"PRJEB42041\",\n\"Webin-57578\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 88,\n            'f': \"88\",\n        },\n\"2021-12-14 14:32:18\",\n{\n            'v': 420,\n            'f': \"420\",\n        },\n\"2022-04-04 14:44:32\",\n\"GENE-SWITCH Chicken transcriptome and gene expression atlas (smallRNA-seq)\"],\n [{\n            'v': 731,\n            'f': \"731\",\n        },\n\"dcc_bonner\",\n\"PRJEB42054\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1009,\n            'f': \"1009\",\n        },\n\"2020-12-16 17:50:17\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS1/CS5_UL51_TGX_PHH_10donorPool_S1500+v2_24hTP raw data (EUT038)\"],\n [{\n            'v': 732,\n            'f': \"732\",\n        },\n\"dcc_faang\",\n\"PRJEB42315\",\n\"Webin-54459\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 38,\n            'f': \"38\",\n        },\n\"2020-12-31 18:00:53\",\n{\n            'v': 82,\n            'f': \"82\",\n        },\n\"2021-01-20 17:53:36\",\n\"\\u201cAdopt-a-Tissue\\u201d Initiative Advances Efforts to Identify Tissue- Specific Histone Marks in the Mare\"],\n [{\n            'v': 733,\n            'f': \"733\",\n        },\n\"dcc_faang\",\n\"PRJEB42407\",\n\"Webin-51519\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2021-01-08 22:14:03\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Whole Genome Shotgun Sequence data for two male Thoroughbreds\"],\n [{\n            'v': 734,\n            'f': \"734\",\n        },\n\"dcc_faang\",\n\"PRJEB42490\",\n\"Webin-57578\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 225,\n            'f': \"225\",\n        },\n\"2021-01-14 20:37:20\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Pig, chicken, goat, cattle transcriptome and gene expression atlas by the FAANG pilot project FR-AgENCODE\"],\n [{\n            'v': 735,\n            'f': \"735\",\n        },\n\"dcc_grusin\",\n\"PRJEB42510\",\n\"Webin-57352\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 92,\n            'f': \"92\",\n        },\n\"2021-02-18 11:07:31\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"NGS sequencing of SARS CoV-2 samples in Ireland\"],\n [{\n            'v': 736,\n            'f': \"736\",\n        },\n\"dcc_grusin\",\n\"PRJEB42601\",\n\"Webin-57951\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 978,\n            'f': \"978\",\n        },\n\"2021-04-29 10:26:57\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"A novel method for the mass-scale genomic surveillance of SARS-CoV-2\"],\n [{\n            'v': 737,\n            'f': \"737\",\n        },\n\"dcc_grusin\",\n\"PRJEB42623\",\n\"Webin-2\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1187,\n            'f': \"1187\",\n        },\n\"2021-01-22 18:25:13\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Patterns of within-host diversity in 1,181 SARS-CoV-2 samples sequenced to high depth in duplicate\"],\n [{\n            'v': 738,\n            'f': \"738\",\n        },\n\"dcc_grusin\",\n\"PRJEB42647\",\n\"Webin-45990\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 80,\n            'f': \"80\",\n        },\n\"2021-03-18 16:56:04\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"AmpliCoV: Rapid Whole-Genome Sequencing Using Multiplex PCR Amplification and Real-Time Oxford Nanopore MinION Sequencing enables rapid variant identification of SARS-CoV-2\"],\n [{\n            'v': 739,\n            'f': \"739\",\n        },\n\"dcc_grusin\",\n\"PRJEB42699\",\n\"Webin-56948\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8,\n            'f': \"8\",\n        },\n\"2021-01-27 15:07:46\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 genome sequencing from post-mortem FFPE lung tissues\"],\n [{\n            'v': 740,\n            'f': \"740\",\n        },\n\"dcc_faang\",\n\"PRJEB42772\",\n\"Webin-31385\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 51,\n            'f': \"51\",\n        },\n\"2021-01-29 10:43:06\",\n{\n            'v': 20,\n            'f': \"20\",\n        },\n\"2022-05-23 15:21:58\",\n\"GENE-SWITCH Pig DNA methylation with Whole Genome bisulfite sequencing (WGBS)\"],\n [{\n            'v': 741,\n            'f': \"741\",\n        },\n\"dcc_faang\",\n\"PRJEB42775\",\n\"Webin-31385\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 50,\n            'f': \"50\",\n        },\n\"2021-01-29 11:47:44\",\n{\n            'v': 21,\n            'f': \"21\",\n        },\n\"2022-05-23 20:34:15\",\n\"GENE-SWITCH Chicken DNA methylation with Whole Genome bisulfite sequencing (WGBS)\"],\n [{\n            'v': 742,\n            'f': \"742\",\n        },\n\"dcc_grusin\",\n\"PRJEB42961\",\n\"Webin-58118\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 5740,\n            'f': \"5740\",\n        },\n\"2021-04-26 22:51:15\",\n{\n            'v': 1681,\n            'f': \"1681\",\n        },\n\"2021-06-22 10:21:12\",\n\"Whole genome sequencing of SARS-CoV-2 from Covid-19 patients from Estonia\"],\n [{\n            'v': 743,\n            'f': \"743\",\n        },\n\"dcc_grusin\",\n\"PRJEB42987\",\n\"Webin-56698\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 327,\n            'f': \"327\",\n        },\n\"2021-02-18 00:08:25\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Deep whole-genome sequencing of SARS-CoV-2 patient samples from Austria.\"],\n [{\n            'v': 744,\n            'f': \"744\",\n        },\n\"dcc_bonner\",\n\"PRJEB43004\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 846,\n            'f': \"846\",\n        },\n\"2021-02-09 11:44:41\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WP5_TempoSeq-BioSpyder raw data (EUT045)\"],\n [{\n            'v': 745,\n            'f': \"745\",\n        },\n\"dcc_grusin\",\n\"PRJEB43166\",\n\"Webin-56250\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 22504,\n            'f': \"22504\",\n        },\n\"2021-10-21 14:14:03\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WGS of SARS-CoV-2 circulating in Spain\"],\n [{\n            'v': 746,\n            'f': \"746\",\n        },\n\"dcc_grusin\",\n\"PRJEB43221\",\n\"Webin-48105\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2021-02-22 11:10:51\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Host genetic analysis should be mandatory for proper classification of COVID-19 reinfections\"],\n [{\n            'v': 747,\n            'f': \"747\",\n        },\n\"dcc_faang\",\n\"PRJEB43235\",\n\"Webin-56701\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 109,\n            'f': \"109\",\n        },\n\"2021-02-22 14:50:48\",\n{\n            'v': 109,\n            'f': \"109\",\n        },\n\"2021-02-23 15:52:29\",\n\"CAGE sequencing of BOVREG cattle tissues\"],\n [{\n            'v': 748,\n            'f': \"748\",\n        },\n\"dcc_korman\",\n\"PRJEB43235\",\n\"Webin-56701\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 109,\n            'f': \"109\",\n        },\n\"2021-02-22 14:50:48\",\n{\n            'v': 109,\n            'f': \"109\",\n        },\n\"2021-02-23 15:52:29\",\n\"CAGE sequencing of BOVREG cattle tissues\"],\n [{\n            'v': 749,\n            'f': \"749\",\n        },\n\"dcc_grusin\",\n\"PRJEB43269\",\n\"Webin-58252\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1130,\n            'f': \"1130\",\n        },\n\"2021-04-12 12:19:43\",\n{\n            'v': 1114,\n            'f': \"1114\",\n        },\n\"2021-04-29 16:51:38\",\n\"SARS-Cov2 Haut-de-France sequenced positive samples\"],\n [{\n            'v': 750,\n            'f': \"750\",\n        },\n\"dcc_grusin\",\n\"PRJEB43319\",\n\"Webin-55200\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1385,\n            'f': \"1385\",\n        },\n\"2021-03-18 12:37:37\",\n{\n            'v': 1385,\n            'f': \"1385\",\n        },\n\"2021-03-19 21:02:47\",\n\"Sequencing SARS-CoV-2 positive hospital staff and patient samples\"],\n [{\n            'v': 751,\n            'f': \"751\",\n        },\n\"dcc_powell\",\n\"PRJEB43435\",\n\"Webin-58389\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2021-03-08 17:49:18\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 Genomic Surveillance in Greece\"],\n [{\n            'v': 752,\n            'f': \"752\",\n        },\n\"dcc_grusin\",\n\"PRJEB43444\",\n\"Webin-58318\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8793,\n            'f': \"8793\",\n        },\n\"2022-01-11 13:47:37\",\n{\n            'v': 8571,\n            'f': \"8571\",\n        },\n\"2022-01-12 08:36:40\",\n\"Biomedical research PanClinCov is focused on usability of 2nd and 3rd generation sequencing methods in improving clinical management of people infected by SARS-CoV-2 as well as application of these methods to identify genomic variability of SARS-CoV-2.\"],\n [{\n            'v': 753,\n            'f': \"753\",\n        },\n\"dcc_faang\",\n\"PRJEB43513\",\n\"Webin-48466\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 14,\n            'f': \"14\",\n        },\n\"2021-03-09 01:45:15\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Identification and characterization of developmental, tissue, and evolutionary transcription start sites in Bos taurus indicus\"],\n [{\n            'v': 754,\n            'f': \"754\",\n        },\n\"dcc_grusin\",\n\"PRJEB43562\",\n\"Webin-55756\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 321,\n            'f': \"321\",\n        },\n\"2021-03-09 22:12:37\",\n{\n            'v': 321,\n            'f': \"321\",\n        },\n\"2021-03-09 23:05:19\",\n\"Genomic surveillance of SARS-CoV-2 in Zimbabwe reveals the dominance of variants of concern\"],\n [{\n            'v': 755,\n            'f': \"755\",\n        },\n\"dcc_bonner\",\n\"PRJEB43586\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 203,\n            'f': \"203\",\n        },\n\"2021-03-10 17:09:15\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WP9_ITEM3a_Liver_PCTS_TGX_exp_24h_01 raw data (EUT091)\"],\n [{\n            'v': 756,\n            'f': \"756\",\n        },\n\"dcc_bonner\",\n\"PRJEB43625\",\n\"Webin-54123\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 198,\n            'f': \"198\",\n        },\n\"2021-03-12 11:31:46\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS2_UKN1_DART_iPSC_Diff_6D raw data (EUT051)\"],\n [{\n            'v': 757,\n            'f': \"757\",\n        },\n\"dcc_grusin\",\n\"PRJEB43731\",\n\"Webin-49415\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 192,\n            'f': \"192\",\n        },\n\"2021-03-17 18:33:29\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) experimental evolution\"],\n [{\n            'v': 758,\n            'f': \"758\",\n        },\n\"dcc_faang\",\n\"PRJEB43826\",\n\"Webin-58299\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 22,\n            'f': \"22\",\n        },\n\"2021-03-21 18:23:07\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Transcriptional landscape of porcine circulating immune cells\"],\n [{\n            'v': 759,\n            'f': \"759\",\n        },\n\"dcc_grusin\",\n\"PRJEB43882\",\n\"Webin-58181\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 17,\n            'f': \"17\",\n        },\n\"2021-03-27 15:26:11\",\n{\n            'v': 17,\n            'f': \"17\",\n        },\n\"2021-03-28 21:18:04\",\n\"Test datasets for impuSARS, a tool for the imputation of SARS-CoV-2 whole genome sequences from incomplete or partial sequences.\"],\n [{\n            'v': 760,\n            'f': \"760\",\n        },\n\"dcc_longo\",\n\"PRJEB44141\",\n\"Webin-58484\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8368,\n            'f': \"8368\",\n        },\n\"2023-10-20 13:56:11\",\n{\n            'v': 1163,\n            'f': \"1163\",\n        },\n\"2021-07-16 13:53:04\",\n\"SARS-CoV-2 sequencing data, generated at INAB-CERTH, and retrieved from Human samples\"],\n [{\n            'v': 761,\n            'f': \"761\",\n        },\n\"dcc_mozart\",\n\"PRJEB4419\",\n\"Webin-9\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 141,\n            'f': \"141\",\n        },\n\"2018-08-22 13:04:36\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Shotgun Sequencing of Tara Oceans DNA samples corresponding to size fractions for small DNA viruses.\"],\n [{\n            'v': 762,\n            'f': \"762\",\n        },\n\"dcc_faang\",\n\"PRJEB44198\",\n\"Webin-57578\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2021-04-08 13:21:59\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Pig promoter Capture Hi-C on liver cells\"],\n [{\n            'v': 763,\n            'f': \"763\",\n        },\n\"dcc_grusin\",\n\"PRJEB44303\",\n\"Webin-47017\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 253,\n            'f': \"253\",\n        },\n\"2021-04-18 11:32:25\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Nanopore Sequencing of SARS-CoV-2: Comparison of Short and Long PCR-tiling Amplicon Protocols\"],\n [{\n            'v': 764,\n            'f': \"764\",\n        },\n\"dcc_grusin\",\n\"PRJEB44396\",\n\"Webin-58181\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 29035,\n            'f': \"29035\",\n        },\n\"2023-08-18 11:52:33\",\n{\n            'v': 781,\n            'f': \"781\",\n        },\n\"2021-05-11 09:45:50\",\n\"Whole genome sequencing of SARS-CoV-2 for epidemiological surveillance in Andalusia (Spain)\"],\n [{\n            'v': 765,\n            'f': \"765\",\n        },\n\"dcc_faang\",\n\"PRJEB44441\",\n\"Webin-57578\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8,\n            'f': \"8\",\n        },\n\"2021-04-21 11:33:06\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Pig and chicken whole genome sequencing by the FAANG pilot project FR-AgENCODE\"],\n [{\n            'v': 766,\n            'f': \"766\",\n        },\n\"dcc_faang\",\n\"PRJEB44468\",\n\"Webin-57578\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 336,\n            'f': \"336\",\n        },\n\"2021-04-22 09:01:18\",\n{\n            'v': 212,\n            'f': \"212\",\n        },\n\"2022-05-17 10:20:05\",\n\"GENE-SWitCH Pig chromatin accessibility profiling by ATAC-seq\"],\n [{\n            'v': 767,\n            'f': \"767\",\n        },\n\"dcc_faang\",\n\"PRJEB44486\",\n\"Webin-57578\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 15,\n            'f': \"15\",\n        },\n\"2021-04-23 09:50:15\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"GENE-SWitCH pig promoter Capture Hi-C\"],\n [{\n            'v': 768,\n            'f': \"768\",\n        },\n\"dcc_faang\",\n\"PRJEB44547\",\n\"Webin-58714\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 106,\n            'f': \"106\",\n        },\n\"2021-04-26 15:35:38\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Transcriptome and metabolome analysis of the post-hatch broiler chicken\"],\n [{\n            'v': 769,\n            'f': \"769\",\n        },\n\"dcc_faang\",\n\"PRJEB44817\",\n\"Webin-59030\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 22,\n            'f': \"22\",\n        },\n\"2021-05-21 08:13:39\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Identification and characterization of developmental, tissue, and evolutionary transcription start sites in Bos taurus indicus\"],\n [{\n            'v': 770,\n            'f': \"770\",\n        },\n\"dcc_grusin\",\n\"PRJEB44932\",\n\"Webin-57088\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1829,\n            'f': \"1829\",\n        },\n\"2022-06-20 00:05:51\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Surveillance of SARS-CoV-2 genomic variants in wastewater in Switzerland\"],\n [{\n            'v': 771,\n            'f': \"771\",\n        },\n\"dcc_faang\",\n\"PRJEB45945\",\n\"Webin-57578\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 336,\n            'f': \"336\",\n        },\n\"2021-06-28 12:13:41\",\n{\n            'v': 106,\n            'f': \"106\",\n        },\n\"2022-05-20 14:25:39\",\n\"GENE-SWitCH Chicken chromatin accessibility profiling by ATAC-seq\"],\n [{\n            'v': 772,\n            'f': \"772\",\n        },\n\"dcc_faang\",\n\"PRJEB46047\",\n\"Webin-58454\",\nNaN,\n{\n            'v': 5,\n            'f': \"5\",\n        },\n{\n            'v': 170,\n            'f': \"170\",\n        },\n\"2021-08-02 22:35:27\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Functional Annotation of Bovine Genome\"],\n [{\n            'v': 773,\n            'f': \"773\",\n        },\n\"dcc_faang\",\n\"PRJEB46166\",\n\"Webin-40564\",\nNaN,\n{\n            'v': 5,\n            'f': \"5\",\n        },\n{\n            'v': 209,\n            'f': \"209\",\n        },\n\"2021-09-07 17:38:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ChIPseq for Atlantic Salmon tissues\"],\n [{\n            'v': 774,\n            'f': \"774\",\n        },\n\"dcc_faang\",\n\"PRJEB46340\",\n\"Webin-40564\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ChIPseq for Atlantic Salmon tissues\"],\n [{\n            'v': 775,\n            'f': \"775\",\n        },\n\"dcc_faang\",\n\"PRJEB46769\",\n\"Webin-58454\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Functional Annotation of Bovine Genome\"],\n [{\n            'v': 776,\n            'f': \"776\",\n        },\n\"dcc_moran\",\n\"PRJEB47132\",\n\"Webin-56719\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Demonstration COVID-19 Data Hub Project\"],\n [{\n            'v': 777,\n            'f': \"777\",\n        },\n\"dcc_faang\",\n\"PRJEB47408\",\n\"Webin-40564\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 60,\n            'f': \"60\",\n        },\n\"2021-09-07 17:38:19\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ATACseq for Atlantic Salmon tissues\"],\n [{\n            'v': 778,\n            'f': \"778\",\n        },\n\"dcc_faang\",\n\"PRJEB47409\",\n\"Webin-40564\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 84,\n            'f': \"84\",\n        },\n\"2021-09-07 17:38:22\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNAseq for Atlantic Salmon tissues\"],\n [{\n            'v': 779,\n            'f': \"779\",\n        },\n\"dcc_faang\",\n\"PRJEB47410\",\n\"Webin-40564\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ChIPseq for Atlantic Salmon tissues\"],\n [{\n            'v': 780,\n            'f': \"780\",\n        },\n\"dcc_faang\",\n\"PRJEB47517\",\n\"Webin-60092\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 63,\n            'f': \"63\",\n        },\n\"2022-09-20 14:09:24\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Assessment of DNA methylation in porcine immune cells\"],\n [{\n            'v': 781,\n            'f': \"781\",\n        },\n\"dcc_faang\",\n\"PRJEB47796\",\n\"Webin-60215\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNAseq for Turbot\"],\n [{\n            'v': 782,\n            'f': \"782\",\n        },\n\"dcc_faang\",\n\"PRJEB47799\",\n\"Webin-60215\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ATACseq for Turbot\"],\n [{\n            'v': 783,\n            'f': \"783\",\n        },\n\"dcc_faang\",\n\"PRJEB47933\",\n\"Webin-60170\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 154,\n            'f': \"154\",\n        },\n\"2021-10-06 09:30:51\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNAseq for Turbot\"],\n [{\n            'v': 784,\n            'f': \"784\",\n        },\n\"dcc_faang\",\n\"PRJEB47934\",\n\"Webin-60170\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 116,\n            'f': \"116\",\n        },\n\"2021-10-06 09:30:52\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ATACseq for Turbot\"],\n [{\n            'v': 785,\n            'f': \"785\",\n        },\n\"dcc_walton\",\n\"PRJEB48024\",\n\"Webin-60000\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 20,\n            'f': \"20\",\n        },\n\"2021-10-11 10:13:03\",\n{\n            'v': 60,\n            'f': \"60\",\n        },\n\"2022-02-11 15:55:10\",\n\"Seqneth Pilot for uploading SARS-CoV-2 bam files\"],\n [{\n            'v': 786,\n            'f': \"786\",\n        },\n\"dcc_walton\",\n\"PRJEB48025\",\n\"Webin-60016\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 20,\n            'f': \"20\",\n        },\n\"2021-10-11 14:40:14\",\n{\n            'v': 60,\n            'f': \"60\",\n        },\n\"2022-02-14 11:30:20\",\n\"SARS-CoV-2 datashare\"],\n [{\n            'v': 787,\n            'f': \"787\",\n        },\n\"dcc_korman\",\n\"PRJEB48026\",\n\"Webin-56701\",\n\"2025-01-17 00:00:00\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 126,\n            'f': \"126\",\n        },\n\"2022-05-09 13:21:12\",\n{\n            'v': 126,\n            'f': \"126\",\n        },\n\"2022-05-10 14:35:50\",\n\"Total RNA sequencing of BOVREG cattle tissues and cell lines\"],\n [{\n            'v': 788,\n            'f': \"788\",\n        },\n\"dcc_faang\",\n\"PRJEB48060\",\n\"Webin-60152\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 33,\n            'f': \"33\",\n        },\n\"2022-06-13 13:19:55\",\n{\n            'v': 203,\n            'f': \"203\",\n        },\n\"2023-01-11 02:28:16\",\n\"GENE-SWITCH Chicken transcriptome and gene expression atlas\"],\n [{\n            'v': 789,\n            'f': \"789\",\n        },\n\"dcc_jamal\",\n\"PRJEB48206\",\n\"Webin-40757\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 695,\n            'f': \"695\",\n        },\n\"2022-05-27 12:35:42\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Wastewater metagenomics\\u00a0sequencing for\\u00a0the monitoring of SARS-CoV-2\\u00a0dynamics in\\u00a0Rotterdam,\\u00a0the Netherlands\"],\n [{\n            'v': 790,\n            'f': \"790\",\n        },\n\"dcc_walton\",\n\"PRJEB48256\",\n\"Webin-59717\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 20,\n            'f': \"20\",\n        },\n\"2021-10-22 08:38:32\",\n{\n            'v': 60,\n            'f': \"60\",\n        },\n\"2022-02-11 17:05:51\",\n\"SeqNeth Pilot ENA datahub\"],\n [{\n            'v': 791,\n            'f': \"791\",\n        },\n\"dcc_korman\",\n\"PRJEB48309\",\n\"Webin-56701\",\n\"2025-01-17 00:00:00\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 121,\n            'f': \"121\",\n        },\n\"2022-03-11 16:05:59\",\n{\n            'v': 121,\n            'f': \"121\",\n        },\n\"2022-05-03 15:12:22\",\n\"mRNA sequencing of BOVREG cattle tissues and cell lines\"],\n [{\n            'v': 792,\n            'f': \"792\",\n        },\n\"dcc_walton\",\n\"PRJEB48419\",\n\"Webin-43373\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 20,\n            'f': \"20\",\n        },\n\"2021-11-02 15:35:15\",\n{\n            'v': 60,\n            'f': \"60\",\n        },\n\"2022-02-22 17:41:34\",\n\"PILOT ENA SeqNeth\"],\n [{\n            'v': 793,\n            'f': \"793\",\n        },\n\"dcc_korman\",\n\"PRJEB48884\",\n\"Webin-56701\",\n\"2025-01-17 00:00:00\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 121,\n            'f': \"121\",\n        },\n\"2022-03-08 14:44:28\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Small RNA sequencing of BOVREG cattle tissues and cell lines\"],\n [{\n            'v': 794,\n            'f': \"794\",\n        },\n\"dcc_faang\",\n\"PRJEB49224\",\n\"Webin-60152\",\nNaN,\n{\n            'v': 5,\n            'f': \"5\",\n        },\n{\n            'v': 126,\n            'f': \"126\",\n        },\n\"2022-02-16 14:44:52\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"GENE-SWITCH Chicken transcriptome and gene expression atlas: HIFI, FLNC, singleton FLNC\"],\n [{\n            'v': 795,\n            'f': \"795\",\n        },\n\"dcc_bonner\",\n\"PRJEB49302\",\n\"Webin-54123\",\n\"2023-12-09 17:44:33\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 36,\n            'f': \"36\",\n        },\n\"2021-12-09 17:44:37\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS15_UKN4a_DART_LUH_neurite_24h_01 raw data (EUT096)\"],\n [{\n            'v': 796,\n            'f': \"796\",\n        },\n\"dcc_bonner\",\n\"PRJEB49303\",\n\"Webin-54123\",\n\"2023-12-09 17:52:59\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2021-12-09 17:53:01\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS15_UKN5_DART_iDRG_neurite_24h raw data (EUT096)\"],\n [{\n            'v': 797,\n            'f': \"797\",\n        },\n\"dcc_bonner\",\n\"PRJEB49346\",\n\"Webin-54123\",\n\"2023-12-13 11:36:13\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 70,\n            'f': \"70\",\n        },\n\"2021-12-13 11:36:20\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WP7_UKN5_DART_iDRG_neurite_24h raw data (EUT111)\"],\n [{\n            'v': 798,\n            'f': \"798\",\n        },\n\"dcc_bonner\",\n\"PRJEB49360\",\n\"Webin-54123\",\n\"2023-12-13 15:04:42\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 114,\n            'f': \"114\",\n        },\n\"2021-12-13 15:04:50\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS14_SU3b_Neuro_SH_Diff_3d_WT-TempoSeq_Exp2h_6h_raw raw data (EUT104)\"],\n [{\n            'v': 799,\n            'f': \"799\",\n        },\n\"dcc_faang\",\n\"PRJEB49412\",\n\"Webin-60092\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 554,\n            'f': \"554\",\n        },\n\"2021-12-15 14:17:55\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Characteriziation of allele-biased expression and methylation events in porcine fetal tissues\"],\n [{\n            'v': 800,\n            'f': \"800\",\n        },\n\"dcc_bonner\",\n\"PRJEB49486\",\n\"Webin-54123\",\n\"2023-12-16 15:21:59\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 270,\n            'f': \"270\",\n        },\n\"2021-12-16 15:22:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS14_SU3c_Neuro_SH_Diff_3d_WT-TempoSeq_Exp6h_24h_raw raw data (EUT116)\"],\n [{\n            'v': 801,\n            'f': \"801\",\n        },\n\"dcc_bonner\",\n\"PRJEB49500\",\n\"Webin-54123\",\n\"2023-12-16 18:32:20\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 90,\n            'f': \"90\",\n        },\n\"2021-12-16 18:32:29\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS14_SU3a_Neuro_SH_Diff_3d_WT-TempoSeq_Exp6h_24h_raw raw data (EUT084)\"],\n [{\n            'v': 802,\n            'f': \"802\",\n        },\n\"dcc_bonner\",\n\"PRJEB49506\",\n\"Webin-54123\",\n\"2023-12-16 23:53:51\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 286,\n            'f': \"286\",\n        },\n\"2021-12-16 23:54:13\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS3_UL51_TGX_PHH_10donorPool_S1500+v2_24hTP_01 raw data (EUT006)\"],\n [{\n            'v': 803,\n            'f': \"803\",\n        },\n\"dcc_bonner\",\n\"PRJEB49535\",\n\"Webin-54123\",\n\"2023-12-17 18:21:29\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 521,\n            'f': \"521\",\n        },\n\"2021-12-17 18:22:13\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS4_UKN4a_DART_LUH_neurite_24h_01 raw data (EUT014)\"],\n [{\n            'v': 804,\n            'f': \"804\",\n        },\n\"dcc_bonner\",\n\"PRJEB49537\",\n\"Webin-54123\",\n\"2023-12-17 23:57:16\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 70,\n            'f': \"70\",\n        },\n\"2021-12-17 23:57:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS4_VU3b_MitoTox_RT_TempOSeq_24h raw data (EUT078)\"],\n [{\n            'v': 805,\n            'f': \"805\",\n        },\n\"dcc_bonner\",\n\"PRJEB49543\",\n\"Webin-54123\",\n\"2023-12-18 11:48:11\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 45,\n            'f': \"45\",\n        },\n\"2021-12-18 11:48:15\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WP7_UKN5_DART_iDRG_neurite_24h raw data (EUT056)\"],\n [{\n            'v': 806,\n            'f': \"806\",\n        },\n\"dcc_bonner\",\n\"PRJEB49544\",\n\"Webin-54123\",\n\"2023-12-18 18:38:45\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 147,\n            'f': \"147\",\n        },\n\"2021-12-18 18:38:56\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WP6_NA raw data (EUT052)\"],\n [{\n            'v': 807,\n            'f': \"807\",\n        },\n\"dcc_faang\",\n\"PRJEB49984\",\n\"Webin-60490\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 36,\n            'f': \"36\",\n        },\n\"2022-01-10 13:38:26\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNAseq for Rainbow trout head kidney tissue\"],\n [{\n            'v': 808,\n            'f': \"808\",\n        },\n\"dcc_faang\",\n\"PRJEB50076\",\n\"Webin-61127\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 42,\n            'f': \"42\",\n        },\n\"2022-10-20 09:49:17\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNAseq - Advancing European Aquaculture by Genome Functional Annotation\"],\n [{\n            'v': 809,\n            'f': \"809\",\n        },\n\"dcc_faang\",\n\"PRJEB50077\",\n\"Webin-61127\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 36,\n            'f': \"36\",\n        },\n\"2022-01-14 09:59:56\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ATACseq - Advancing European Aquaculture by Genome Functional Annotation\"],\n [{\n            'v': 810,\n            'f': \"810\",\n        },\n\"dcc_lightsey\",\n\"PRJEB50279\",\n\"Webin-61247\",\n\"2024-01-19 00:00:00\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 40,\n            'f': \"40\",\n        },\n\"2022-01-20 11:24:25\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"European lobster RNAseq data\"],\n [{\n            'v': 811,\n            'f': \"811\",\n        },\n\"dcc_bonner\",\n\"PRJEB50574\",\n\"Webin-54123\",\n\"2024-02-01 16:46:34\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 96,\n            'f': \"96\",\n        },\n\"2022-02-01 16:46:49\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS8_ITEM2a_Lung_PBEC_TGX_ALI_exp_01 raw data (EUT035)\"],\n [{\n            'v': 812,\n            'f': \"812\",\n        },\n\"dcc_bonner\",\n\"PRJEB50753\",\n\"Webin-54123\",\n\"2024-02-08 13:45:03\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 1855,\n            'f': \"1855\",\n        },\n\"2022-02-08 13:49:05\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS10_EUT_101_VU_WP10111216_RT_TempOSeq_24h raw data (EUT101)\"],\n [{\n            'v': 813,\n            'f': \"813\",\n        },\n\"dcc_bonner\",\n\"PRJEB50758\",\n\"Webin-54123\",\n\"2024-02-08 16:14:32\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 170,\n            'f': \"170\",\n        },\n\"2022-02-08 16:14:52\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS8_ITEM2a_Lung_PBEC_TGX_ALI_exp_01 raw data (EUT030)\"],\n [{\n            'v': 814,\n            'f': \"814\",\n        },\n\"dcc_faang\",\n\"PRJEB50760\",\n\"Webin-61302\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 632,\n            'f': \"632\",\n        },\n\"2022-02-08 23:24:28\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"GENE-SWitCH Pig chromatin accessibility profiling by ATAC-seq\"],\n [{\n            'v': 815,\n            'f': \"815\",\n        },\n\"dcc_faang\",\n\"PRJEB50963\",\n\"Webin-60152\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 27,\n            'f': \"27\",\n        },\n\"2022-06-13 13:15:36\",\n{\n            'v': 28,\n            'f': \"28\",\n        },\n\"2023-01-11 02:18:32\",\n\"GENE-SWITCH Pig transcriptome and gene expression atlas\"],\n [{\n            'v': 816,\n            'f': \"816\",\n        },\n\"dcc_bonner\",\n\"PRJEB51022\",\n\"Webin-54123\",\n\"2024-02-18 11:17:22\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 189,\n            'f': \"189\",\n        },\n\"2022-02-18 11:17:52\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS8_ITEM2a_Lung_PBEC_TGX_ALI_exp_01 raw data (EUT028)\"],\n [{\n            'v': 817,\n            'f': \"817\",\n        },\n\"dcc_bonner\",\n\"PRJEB51026\",\n\"Webin-54123\",\n\"2024-02-18 11:52:20\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 84,\n            'f': \"84\",\n        },\n\"2022-02-18 11:52:31\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS8_ITEM2a_Lung_PBEC_TGX_ALI_exp_01 raw data (EUT023)\"],\n [{\n            'v': 818,\n            'f': \"818\",\n        },\n\"dcc_bonner\",\n\"PRJEB51066\",\n\"Webin-54123\",\n\"2024-02-21 15:09:45\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 94,\n            'f': \"94\",\n        },\n\"2022-02-21 15:09:57\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS8_ITEM2a_Lung_PBEC_TGX_ALI_exp_01 raw data (EUT022)\"],\n [{\n            'v': 819,\n            'f': \"819\",\n        },\n\"dcc_bonner\",\n\"PRJEB51070\",\n\"Webin-54123\",\n\"2024-02-21 16:52:32\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 72,\n            'f': \"72\",\n        },\n\"2022-02-21 16:52:41\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS8_ITEM2a_Lung_PBEC_TGX_ALI_exp_01 raw data (EUT012)\"],\n [{\n            'v': 820,\n            'f': \"820\",\n        },\n\"dcc_korman\",\n\"PRJEB51163\",\n\"Webin-56701\",\n\"2025-01-17 00:00:00\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 105,\n            'f': \"105\",\n        },\n\"2022-02-25 12:53:23\",\n\"ATAC-seq of BOVREG cattle tissues and cell lines\"],\n [{\n            'v': 821,\n            'f': \"821\",\n        },\n\"dcc_korman\",\n\"PRJEB51167\",\n\"Webin-56701\",\n\"2025-01-17 00:00:00\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 585,\n            'f': \"585\",\n        },\n\"2022-02-28 15:15:28\",\n\"ChIP-seq of BOVREG cattle tissues and cell lines\"],\n [{\n            'v': 822,\n            'f': \"822\",\n        },\n\"dcc_korman\",\n\"PRJEB51422\",\n\"Webin-56701\",\nNaN,\n{\n            'v': 3,\n            'f': \"3\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2022-03-07 11:20:13\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"mRNA sequencing of BOVREG cattle tissues\"],\n [{\n            'v': 823,\n            'f': \"823\",\n        },\n\"dcc_korman\",\n\"PRJEB51474\",\n\"Webin-56701\",\n\"2025-01-17 00:00:00\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2022-03-09 09:45:29\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"mRNA sequencing of BOVREG cattle cell lines\"],\n [{\n            'v': 824,\n            'f': \"824\",\n        },\n\"dcc_barron\",\n\"PRJEB5162\",\n\"Webin-192\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 253,\n            'f': \"253\",\n        },\n\"2014-01-06 14:25:32\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"An observational study of M. tuberculosis transmission in Oxfordshire using whole pathogen genome sequences\"],\n [{\n            'v': 825,\n            'f': \"825\",\n        },\n\"dcc_bonner\",\n\"PRJEB51640\",\n\"Webin-54123\",\n\"2024-03-15 10:41:54\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 173,\n            'f': \"173\",\n        },\n\"2022-03-15 10:42:18\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS2_UHEI_DART_96-120h raw data (EUT080)\"],\n [{\n            'v': 826,\n            'f': \"826\",\n        },\n\"dcc_bonner\",\n\"PRJEB51641\",\n\"Webin-54123\",\n\"2024-03-15 10:53:08\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 25,\n            'f': \"25\",\n        },\n\"2022-03-15 10:53:15\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS2_UHEI_DART_96-120h raw data (EUT053)\"],\n [{\n            'v': 827,\n            'f': \"827\",\n        },\n\"dcc_bonner\",\n\"PRJEB51642\",\n\"Webin-54123\",\n\"2024-03-15 11:09:50\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 16,\n            'f': \"16\",\n        },\n\"2022-03-15 11:09:53\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WP7_LUMC1_lung_PBEC_ALI_24h raw data (EUT043)\"],\n [{\n            'v': 828,\n            'f': \"828\",\n        },\n\"dcc_bonner\",\n\"PRJEB51649\",\n\"Webin-54123\",\n\"2024-03-15 13:53:36\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 417,\n            'f': \"417\",\n        },\n\"2022-03-15 13:54:31\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CS15_VU3d_CS15b_RT_TempOSeq_24h raw data (EUT100b)\"],\n [{\n            'v': 829,\n            'f': \"829\",\n        },\n\"dcc_faang\",\n\"PRJEB51699\",\n\"Webin-60092\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 630,\n            'f': \"630\",\n        },\n\"2022-09-20 15:00:26\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Characteriziation of regulatory elements in sorted porcine immune cells\"],\n [{\n            'v': 830,\n            'f': \"830\",\n        },\n\"dcc_korman\",\n\"PRJEB51824\",\n\"Webin-56701\",\n\"2025-01-17 00:00:00\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2023-03-21 08:05:53\",\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2023-03-21 07:22:36\",\n\"Whole-genome sequencing of BOVREG bovine cell lines\"],\n [{\n            'v': 831,\n            'f': \"831\",\n        },\n\"dcc_faang\",\n\"PRJEB51854\",\n\"Webin-60408\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 21,\n            'f': \"21\",\n        },\n\"2022-03-22 14:48:48\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ATACseq for Atlantic Salmon embryos\"],\n [{\n            'v': 832,\n            'f': \"832\",\n        },\n\"dcc_faang\",\n\"PRJEB51855\",\n\"Webin-60408\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 42,\n            'f': \"42\",\n        },\n\"2022-03-22 14:48:47\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNAseq for Atlantic Salmon embryos\"],\n [{\n            'v': 833,\n            'f': \"833\",\n        },\n\"dcc_faang\",\n\"PRJEB51856\",\n\"Webin-60408\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 17,\n            'f': \"17\",\n        },\n\"2022-03-22 15:07:08\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ATACseq for Rainbow trout embryos\"],\n [{\n            'v': 834,\n            'f': \"834\",\n        },\n\"dcc_faang\",\n\"PRJEB51857\",\n\"Webin-60408\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 42,\n            'f': \"42\",\n        },\n\"2022-03-22 15:07:08\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNAseq for Rainbow trout embryos\"],\n [{\n            'v': 835,\n            'f': \"835\",\n        },\n\"dcc_bonner\",\n\"PRJEB52157\",\n\"Webin-54123\",\n\"2024-04-06 10:12:57\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 165,\n            'f': \"165\",\n        },\n\"2022-04-06 10:13:20\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WP8_TissUse_Chip4 raw data (EUT107)\"],\n [{\n            'v': 836,\n            'f': \"836\",\n        },\n\"dcc_faang\",\n\"PRJEB52159\",\n\"Webin-60858\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2022-04-06 10:15:07\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ATACseq for European Seabass embryos\"],\n [{\n            'v': 837,\n            'f': \"837\",\n        },\n\"dcc_faang\",\n\"PRJEB52160\",\n\"Webin-60858\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 45,\n            'f': \"45\",\n        },\n\"2022-04-06 10:15:09\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNAseq for European Seabass embryos\"],\n [{\n            'v': 838,\n            'f': \"838\",\n        },\n\"dcc_faang\",\n\"PRJEB52247\",\n\"Webin-45927\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 36,\n            'f': \"36\",\n        },\n\"2022-04-09 14:14:11\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Whole genome sequencing of Texel and Scottish Blackface sheep.\"],\n [{\n            'v': 839,\n            'f': \"839\",\n        },\n\"dcc_faang\",\n\"PRJEB52253\",\n\"Webin-45927\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 72,\n            'f': \"72\",\n        },\n\"2022-04-10 14:33:41\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Sheep transcriptome analysis for growth traits\"],\n [{\n            'v': 840,\n            'f': \"840\",\n        },\n\"dcc_faang\",\n\"PRJEB52282\",\n\"Webin-60858\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2022-04-12 11:08:22\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ATACseq for European Seabass leukocytes\"],\n [{\n            'v': 841,\n            'f': \"841\",\n        },\n\"dcc_faang\",\n\"PRJEB52283\",\n\"Webin-60858\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2022-04-12 11:08:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNAseq for European Seabass leukocytes\"],\n [{\n            'v': 842,\n            'f': \"842\",\n        },\n\"dcc_faang\",\n\"PRJEB52284\",\n\"Webin-60858\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2022-04-12 11:08:22\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ATACseq for European Seabass head kidney\"],\n [{\n            'v': 843,\n            'f': \"843\",\n        },\n\"dcc_faang\",\n\"PRJEB52285\",\n\"Webin-60858\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2022-04-12 11:08:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNAseq for European Seabass head kidney\"],\n [{\n            'v': 844,\n            'f': \"844\",\n        },\n\"dcc_faang\",\n\"PRJEB52456\",\n\"Webin-48466\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4052,\n            'f': \"4052\",\n        },\n\"2022-04-26 07:08:24\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genetic variation in histone modifications and gene expression identifies regulatory variants in the mammary gland of cattle\"],\n [{\n            'v': 845,\n            'f': \"845\",\n        },\n\"dcc_mozart\",\n\"PRJEB5265\",\n\"Webin-367\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 7,\n            'f': \"7\",\n        },\n\"2014-01-27 15:26:38\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Artic fresh water viromes\"],\n [{\n            'v': 846,\n            'f': \"846\",\n        },\n\"dcc_faang\",\n\"PRJEB52775\",\n\"Webin-60858\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 60,\n            'f': \"60\",\n        },\n\"2022-05-11 11:41:48\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ATACseq for European Seabass tissues\"],\n [{\n            'v': 847,\n            'f': \"847\",\n        },\n\"dcc_faang\",\n\"PRJEB52776\",\n\"Webin-60858\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 84,\n            'f': \"84\",\n        },\n\"2022-05-11 11:41:49\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNAseq for European Seabass tissues\"],\n [{\n            'v': 848,\n            'f': \"848\",\n        },\n\"dcc_faang\",\n\"PRJEB52831\",\n\"Webin-62156\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 312,\n            'f': \"312\",\n        },\n\"2022-05-13 09:57:25\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Tissue specific RNA-seq of fetuses and piglets\"],\n [{\n            'v': 849,\n            'f': \"849\",\n        },\n\"dcc_faang\",\n\"PRJEB53020\",\n\"Webin-57361\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 20,\n            'f': \"20\",\n        },\n\"2022-05-23 14:45:50\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Improved Equine Transcriptome\"],\n [{\n            'v': 850,\n            'f': \"850\",\n        },\n\"dcc_faang\",\n\"PRJEB53037\",\n\"Webin-57361\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 32,\n            'f': \"32\",\n        },\n\"2022-05-23 15:33:46\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ATAC-seq of Eight Prioritized Equine FAANG Tissues\"],\n [{\n            'v': 851,\n            'f': \"851\",\n        },\n\"dcc_faang\",\n\"PRJEB53044\",\n\"Webin-48466\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 325,\n            'f': \"325\",\n        },\n\"2022-05-24 01:26:47\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Predicting allele specific expression from allele specific binding\"],\n [{\n            'v': 852,\n            'f': \"852\",\n        },\n\"dcc_hines\",\n\"PRJEB53317\",\n\"Webin-40757\",\n\"2025-03-01 00:00:00\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 10,\n            'f': \"10\",\n        },\n\"2022-09-13 12:16:34\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Monkeypox whole-genome sequencing from Netherlands associated with the current non-endemic outbreak in multiple countries.\"],\n [{\n            'v': 853,\n            'f': \"853\",\n        },\n\"dcc_faang\",\n\"PRJEB53382\",\n\"Webin-62439\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 46,\n            'f': \"46\",\n        },\n\"2022-06-07 17:07:09\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Thoroughbred Stallion gene expression\"],\n [{\n            'v': 854,\n            'f': \"854\",\n        },\n\"dcc_faang\",\n\"PRJEB53399\",\n\"Webin-60408\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 78,\n            'f': \"78\",\n        },\n\"2022-06-08 16:13:56\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ChIPseq for Atlantic Salmon embryos\"],\n [{\n            'v': 855,\n            'f': \"855\",\n        },\n\"dcc_faang\",\n\"PRJEB53440\",\n\"Webin-61302\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 640,\n            'f': \"640\",\n        },\n\"2022-06-09 17:22:55\",\n{\n            'v': 333,\n            'f': \"333\",\n        },\n\"2022-07-18 16:02:48\",\n\"GENE-SWitCH Pig chromatin accessibility profiling by ATAC-seq\"],\n [{\n            'v': 856,\n            'f': \"856\",\n        },\n\"dcc_mozart\",\n\"PRJEB5347\",\n\"Webin-367\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2014-02-11 15:26:36\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Artic fresh water viromes\"],\n [{\n            'v': 857,\n            'f': \"857\",\n        },\n\"dcc_faang\",\n\"PRJEB53920\",\n\"Webin-62526\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"FAANG_Chicken\"],\n [{\n            'v': 858,\n            'f': \"858\",\n        },\n\"dcc_faang\",\n\"PRJEB53986\",\n\"Webin-57578\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 21,\n            'f': \"21\",\n        },\n\"2022-06-29 22:02:34\",\n{\n            'v': 21,\n            'f': \"21\",\n        },\n\"2023-02-15 15:56:35\",\n\"GENE-SWitCH chicken promoter Capture Hi-C\"],\n [{\n            'v': 859,\n            'f': \"859\",\n        },\n\"dcc_newborn\",\n\"PRJEB54593\",\n\"Webin-56719\",\n\"2024-07-07 00:00:00\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 11001,\n            'f': \"11001\",\n        },\n\"2022-08-18 14:31:25\",\n\"Study for CoVEO VCF and Coverage Files (To be deleted)\"],\n [{\n            'v': 860,\n            'f': \"860\",\n        },\n\"dcc_faang\",\n\"PRJEB54692\",\n\"Webin-62526\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"FAANG_Chicken\"],\n [{\n            'v': 861,\n            'f': \"861\",\n        },\n\"dcc_faang\",\n\"PRJEB54773\",\n\"Webin-57578\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2022-07-18 09:34:50\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2023-02-15 15:56:35\",\n\"GENE-SWitCH chicken promoter Capture Hi-C\"],\n [{\n            'v': 862,\n            'f': \"862\",\n        },\n\"dcc_faang\",\n\"PRJEB55010\",\n\"Webin-60408\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 78,\n            'f': \"78\",\n        },\n\"2022-07-26 16:45:46\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ChIPseq for Rainbow trout embryos\"],\n [{\n            'v': 863,\n            'f': \"863\",\n        },\n\"dcc_faang\",\n\"PRJEB55063\",\n\"Webin-40564\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 196,\n            'f': \"196\",\n        },\n\"2022-07-28 16:00:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ChIP-seq for Atlantic Salmon tissues\"],\n [{\n            'v': 864,\n            'f': \"864\",\n        },\n\"dcc_faang\",\n\"PRJEB55656\",\n\"Webin-62526\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 104,\n            'f': \"104\",\n        },\n\"2022-08-30 19:38:32\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Functional Annotation of Chicken Genome\"],\n [{\n            'v': 865,\n            'f': \"865\",\n        },\n\"dcc_faang\",\n\"PRJEB56400\",\n\"Webin-58110\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2022-09-30 12:59:47\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNA-seq for Cyprinus carpio head kidney tissues\"],\n [{\n            'v': 866,\n            'f': \"866\",\n        },\n\"dcc_faang\",\n\"PRJEB56401\",\n\"Webin-58110\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2022-09-30 12:59:48\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNA-seq for Cyprinus carpio head kidney tissues\"],\n [{\n            'v': 867,\n            'f': \"867\",\n        },\n\"dcc_faang\",\n\"PRJEB56402\",\n\"Webin-58110\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2022-09-30 12:59:48\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ATAC-seq for Cyprinus carpio head kidney tissues\"],\n [{\n            'v': 868,\n            'f': \"868\",\n        },\n\"dcc_faang\",\n\"PRJEB56698\",\n\"Webin-61127\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 78,\n            'f': \"78\",\n        },\n\"2022-10-14 14:49:52\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ChIPseq - Advancing European Aquaculture by Genome Functional Annotation\"],\n [{\n            'v': 869,\n            'f': \"869\",\n        },\n\"dcc_faang\",\n\"PRJEB57084\",\n\"Webin-63251\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 60,\n            'f': \"60\",\n        },\n\"2022-10-26 15:14:05\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ATACseq for Common carp tissues\"],\n [{\n            'v': 870,\n            'f': \"870\",\n        },\n\"dcc_faang\",\n\"PRJEB57085\",\n\"Webin-63251\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 84,\n            'f': \"84\",\n        },\n\"2022-10-26 15:14:06\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNAseq for Common carp tissues\"],\n [{\n            'v': 871,\n            'f': \"871\",\n        },\n\"dcc_faang\",\n\"PRJEB57086\",\n\"Webin-63251\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 242,\n            'f': \"242\",\n        },\n\"2023-02-22 14:22:05\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ChIPseq for Common carp tissues\"],\n [{\n            'v': 872,\n            'f': \"872\",\n        },\n\"dcc_faang\",\n\"PRJEB57190\",\n\"Webin-63276\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 48,\n            'f': \"48\",\n        },\n\"2022-10-31 09:29:04\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ATACseq for Rainbow Trout tissues\"],\n [{\n            'v': 873,\n            'f': \"873\",\n        },\n\"dcc_faang\",\n\"PRJEB57191\",\n\"Webin-63276\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 90,\n            'f': \"90\",\n        },\n\"2022-10-31 09:29:03\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNAseq for Rainbow Trout tissues\"],\n [{\n            'v': 874,\n            'f': \"874\",\n        },\n\"dcc_faang\",\n\"PRJEB57295\",\n\"Webin-63272\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2022-11-04 13:53:58\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Sequence\"],\n [{\n            'v': 875,\n            'f': \"875\",\n        },\n\"dcc_faang\",\n\"PRJEB57395\",\n\"Webin-60490\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 36,\n            'f': \"36\",\n        },\n\"2022-11-09 13:50:05\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Advancing European Aquaculture by Genome Functional Annotation\"],\n [{\n            'v': 876,\n            'f': \"876\",\n        },\n\"dcc_faang\",\n\"PRJEB57595\",\n\"Webin-60092\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2022-11-15 15:02:37\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Transcriptome profiling of porcine neutrophils\"],\n [{\n            'v': 877,\n            'f': \"877\",\n        },\n\"dcc_faang\",\n\"PRJEB57637\",\n\"Webin-62439\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 112,\n            'f': \"112\",\n        },\n\"2022-11-16 22:37:34\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Functional annotation of putative cis-regulatory elements in the genomes of two Thoroughbred stallions\"],\n [{\n            'v': 878,\n            'f': \"878\",\n        },\n\"dcc_faang\",\n\"PRJEB57784\",\n\"Webin-60170\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 364,\n            'f': \"364\",\n        },\n\"2022-12-19 12:35:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ChIPseq for Turbot\"],\n [{\n            'v': 879,\n            'f': \"879\",\n        },\n\"dcc_korman\",\n\"PRJEB57838\",\n\"Webin-56701\",\n\"2025-01-17 00:00:00\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 162,\n            'f': \"162\",\n        },\n\"2022-11-28 09:27:05\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RRBS sequencing of BOVREG cattle tissues\"],\n [{\n            'v': 880,\n            'f': \"880\",\n        },\n\"dcc_faang\",\n\"PRJEB57948\",\n\"Webin-60490\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 32,\n            'f': \"32\",\n        },\n\"2022-11-30 15:15:42\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Advancing European Aquaculture by Genome Functional Annotation\"],\n [{\n            'v': 881,\n            'f': \"881\",\n        },\n\"dcc_faang\",\n\"PRJEB57949\",\n\"Webin-60490\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 32,\n            'f': \"32\",\n        },\n\"2022-11-30 15:15:42\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Advancing European Aquaculture by Genome Functional Annotation\"],\n [{\n            'v': 882,\n            'f': \"882\",\n        },\n\"dcc_faang\",\n\"PRJEB57956\",\n\"Webin-63276\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 171,\n            'f': \"171\",\n        },\n\"2023-01-23 11:45:50\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ChIPseq for Rainbow Trout tissues\"],\n [{\n            'v': 883,\n            'f': \"883\",\n        },\n\"dcc_faang\",\n\"PRJEB58030\",\n\"Webin-63235\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2980,\n            'f': \"2980\",\n        },\n\"2022-12-04 02:35:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"PIG whole genome sequencing\"],\n [{\n            'v': 884,\n            'f': \"884\",\n        },\n\"dcc_faang\",\n\"PRJEB58031\",\n\"Webin-63235\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 6229,\n            'f': \"6229\",\n        },\n\"2022-12-04 02:37:27\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"PIG transcriptome and gene expression\"],\n [{\n            'v': 885,\n            'f': \"885\",\n        },\n\"dcc_faang\",\n\"PRJEB58434\",\n\"Webin-60170\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ChIPseq for Turbot, data missed on previous release\"],\n [{\n            'v': 886,\n            'f': \"886\",\n        },\n\"dcc_faang\",\n\"PRJEB58732\",\n\"Webin-63251\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ChIPseq for Common carp tissues\"],\n [{\n            'v': 887,\n            'f': \"887\",\n        },\n\"dcc_faang\",\n\"PRJEB59102\",\n\"Webin-40564\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 24,\n            'f': \"24\",\n        },\n\"2023-01-19 14:49:05\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"microRNA profiling for Atlantic Salmon tissues\"],\n [{\n            'v': 888,\n            'f': \"888\",\n        },\n\"dcc_faang\",\n\"PRJEB59432\",\n\"Webin-60858\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 236,\n            'f': \"236\",\n        },\n\"2023-02-01 09:12:40\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ChIPseq for European Seabass tissues\"],\n [{\n            'v': 889,\n            'f': \"889\",\n        },\n\"dcc_faang\",\n\"PRJEB59434\",\n\"Webin-60858\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 58,\n            'f': \"58\",\n        },\n\"2023-02-01 10:05:54\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ChIPseq for European Seabass embryos\"],\n [{\n            'v': 890,\n            'f': \"890\",\n        },\n\"dcc_faang\",\n\"PRJEB59435\",\n\"Webin-60170\",\nNaN,\n{\n            'v': 5,\n            'f': \"5\",\n        },\n{\n            'v': 128,\n            'f': \"128\",\n        },\n\"2023-02-07 09:23:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"microRNA profiling for Turbot\"],\n [{\n            'v': 891,\n            'f': \"891\",\n        },\n\"dcc_faang\",\n\"PRJEB59474\",\n\"Webin-60809\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 99,\n            'f': \"99\",\n        },\n\"2023-02-03 22:33:07\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"FAANG Pig IPECJ2 cell line\"],\n [{\n            'v': 892,\n            'f': \"892\",\n        },\n\"dcc_faang\",\n\"PRJEB59475\",\n\"Webin-60809\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 101,\n            'f': \"101\",\n        },\n\"2023-02-04 00:09:36\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"FAANG Chicken SL-29 cell line\"],\n [{\n            'v': 893,\n            'f': \"893\",\n        },\n\"dcc_faang\",\n\"PRJEB59489\",\n\"Webin-63251\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 24,\n            'f': \"24\",\n        },\n\"2023-02-03 09:33:49\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"microRNA profiling for Common carp tissues\"],\n [{\n            'v': 894,\n            'f': \"894\",\n        },\n\"dcc_faang\",\n\"PRJEB59557\",\n\"Webin-60858\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 56,\n            'f': \"56\",\n        },\n\"2023-02-06 14:45:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ChIPseq for European Seabass leukocytes\"],\n [{\n            'v': 895,\n            'f': \"895\",\n        },\n\"dcc_faang\",\n\"PRJEB59558\",\n\"Webin-60858\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 55,\n            'f': \"55\",\n        },\n\"2023-02-06 14:45:22\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ChIPseq for European Seabass head kidney\"],\n [{\n            'v': 896,\n            'f': \"896\",\n        },\n\"dcc_faang\",\n\"PRJEB59755\",\n\"Webin-45927\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2023-02-09 08:47:15\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Sheep CAGE-seq analysis for transcriptional start sites and validation of the ISO-seq data.\"],\n [{\n            'v': 897,\n            'f': \"897\",\n        },\n\"dcc_faang\",\n\"PRJEB59757\",\n\"Webin-45927\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2023-02-09 10:23:50\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Sheep RNA isoform analysis for growth traits using ISO-Seq (HiFi)\"],\n [{\n            'v': 898,\n            'f': \"898\",\n        },\n\"dcc_faang\",\n\"PRJEB59878\",\n\"Webin-64157\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Transcriptional landscape of porcine circulating immune cells\"],\n [{\n            'v': 899,\n            'f': \"899\",\n        },\n\"dcc_faang\",\n\"PRJEB60002\",\n\"Webin-63276\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 24,\n            'f': \"24\",\n        },\n\"2023-02-16 15:29:41\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"microRNA profiling for Rainbow Trout tissues\"],\n [{\n            'v': 900,\n            'f': \"900\",\n        },\n\"dcc_faang\",\n\"PRJEB60290\",\n\"Webin-60858\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 24,\n            'f': \"24\",\n        },\n\"2023-02-28 11:43:11\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"microRNA profiling for Dicentrarchus labrax tissues\"],\n [{\n            'v': 901,\n            'f': \"901\",\n        },\n\"dcc_faang\",\n\"PRJEB60291\",\n\"Webin-60858\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 25,\n            'f': \"25\",\n        },\n\"2023-02-28 11:43:12\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"microRNA profiling for Dicentrarchus labrax embryos\"],\n [{\n            'v': 902,\n            'f': \"902\",\n        },\n\"dcc_faang\",\n\"PRJEB60369\",\n\"Webin-60170\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 64,\n            'f': \"64\",\n        },\n\"2023-03-02 10:40:18\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"microRNA profiling for Turbot\"],\n [{\n            'v': 903,\n            'f': \"903\",\n        },\n\"dcc_faang\",\n\"PRJEB60517\",\n\"Webin-57578\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 32,\n            'f': \"32\",\n        },\n\"2023-03-10 09:29:53\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Deciphering the embryonic molecular environment before implantation\"],\n [{\n            'v': 904,\n            'f': \"904\",\n        },\n\"dcc_faang\",\n\"PRJEB61138\",\n\"Webin-60858\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2023-04-04 10:40:01\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"microRNA profiling for European Seabass head kidney\"],\n [{\n            'v': 905,\n            'f': \"905\",\n        },\n\"dcc_korman\",\n\"PRJEB62126\",\n\"Webin-56701\",\n\"2025-05-14 00:00:00\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2023-05-15 10:15:28\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Whole-genome enzymatic methylation study of BOVREG bovine cell lines\"],\n [{\n            'v': 906,\n            'f': \"906\",\n        },\n\"dcc_faang\",\n\"PRJEB63475\",\n\"Webin-50285\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 878,\n            'f': \"878\",\n        },\n\"2023-07-03 08:44:14\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"GGA_INRAE-InstitutAgro_EFFICACE-GEroNIMO_2023_RNAseq\"],\n [{\n            'v': 907,\n            'f': \"907\",\n        },\n\"dcc_faang\",\n\"PRJEB63591\",\n\"Webin-50285\",\nNaN,\n{\n            'v': 5,\n            'f': \"5\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"GGA_INRAE-InstitutAgro_EFFICACE-GEroNIMO_2023_RNAseq_2\"],\n [{\n            'v': 908,\n            'f': \"908\",\n        },\n\"dcc_faang\",\n\"PRJEB63633\",\n\"Webin-50285\",\nNaN,\n{\n            'v': 5,\n            'f': \"5\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"GGA_INRAE-InstitutAgro_EFFICACE-GEroNIMO_2023_RNAseq_3\"],\n [{\n            'v': 909,\n            'f': \"909\",\n        },\n\"dcc_faang\",\n\"PRJEB63990\",\n\"Webin-50285\",\nNaN,\n{\n            'v': 5,\n            'f': \"5\",\n        },\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"GGA_INRAE-InstitutAgro_EFFICACE-GEroNIMO_2023_RNAseq_4\"],\n [{\n            'v': 910,\n            'f': \"910\",\n        },\n\"dcc_faang\",\n\"PRJEB64165\",\n\"Webin-61302\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 32,\n            'f': \"32\",\n        },\n\"2023-07-07 12:19:29\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Pig Intestine Segments Tissus and their derived Organoids gene expression profiling by RNA-seq\"],\n [{\n            'v': 911,\n            'f': \"911\",\n        },\n\"dcc_faang\",\n\"PRJEB64876\",\n\"Webin-58177\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2023-08-08 12:35:01\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ATACseq for Gilthead seabream tissue\"],\n [{\n            'v': 912,\n            'f': \"912\",\n        },\n\"dcc_faang\",\n\"PRJEB64877\",\n\"Webin-58177\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2023-08-08 12:35:00\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNAseq for Gilthead seabream tissue\"],\n [{\n            'v': 913,\n            'f': \"913\",\n        },\n\"dcc_faang\",\n\"PRJEB64878\",\n\"Webin-58177\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 53,\n            'f': \"53\",\n        },\n\"2023-08-08 12:35:04\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ChIPseq for Gilthead seabream tissue\"],\n [{\n            'v': 914,\n            'f': \"914\",\n        },\n\"dcc_faang\",\n\"PRJEB64879\",\n\"Webin-58177\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2023-08-08 12:35:01\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ATACseq for Gilthead seabream head kidney leukocytes\"],\n [{\n            'v': 915,\n            'f': \"915\",\n        },\n\"dcc_faang\",\n\"PRJEB64880\",\n\"Webin-58177\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2023-08-08 12:35:00\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNAseq for Gilthead seabream head kidney leukocytes\"],\n [{\n            'v': 916,\n            'f': \"916\",\n        },\n\"dcc_faang\",\n\"PRJEB64881\",\n\"Webin-58177\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 54,\n            'f': \"54\",\n        },\n\"2023-08-08 12:35:03\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ChIPseq for Gilthead seabream head kidney leukocytes\"],\n [{\n            'v': 917,\n            'f': \"917\",\n        },\n\"dcc_chopin\",\n\"PRJEB6491\",\n\"Webin-36761\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2014-06-08 02:47:10\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Isolation of a Salmonella Kentucky ST-198 strain and its H2S-negative variant from the same patient.\"],\n [{\n            'v': 918,\n            'f': \"918\",\n        },\n\"dcc_basie\",\n\"PRJEB6499\",\n\"Webin-215\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 185,\n            'f': \"185\",\n        },\n\"2014-06-09 22:26:34\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Parfrey_euk_testing_samples_515_1119\"],\n [{\n            'v': 919,\n            'f': \"919\",\n        },\n\"dcc_korman\",\n\"PRJEB65242\",\n\"Webin-56701\",\n\"2025-08-15 00:00:00\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2023-08-16 16:47:48\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CAGE sequencing of BOVREG Cell Lines\"],\n [{\n            'v': 920,\n            'f': \"920\",\n        },\n\"dcc_korman\",\n\"PRJEB65575\",\n\"Webin-56701\",\n\"2025-08-29 00:00:00\",\n{\n            'v': 2,\n            'f': \"2\",\n        },\n{\n            'v': 13,\n            'f': \"13\",\n        },\n\"2023-08-30 06:05:32\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Single-cell and single-nuclei RNA sequencing of BovReg key tissues and matrices\"],\n [{\n            'v': 921,\n            'f': \"921\",\n        },\n\"dcc_faang\",\n\"PRJEB65849\",\n\"Webin-60408\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 26,\n            'f': \"26\",\n        },\n\"2023-09-04 10:12:34\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"microRNA profiling for Atlantic Salmon embryos\"],\n [{\n            'v': 922,\n            'f': \"922\",\n        },\n\"dcc_faang\",\n\"PRJEB65850\",\n\"Webin-60408\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 26,\n            'f': \"26\",\n        },\n\"2023-09-04 10:12:35\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"microRNA profiling for Rainbow Trout embryos\"],\n [{\n            'v': 923,\n            'f': \"923\",\n        },\n\"dcc_faang\",\n\"PRJEB65932\",\n\"Webin-47535\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 35,\n            'f': \"35\",\n        },\n\"2023-09-08 09:28:20\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WGS of northern-Eurasian reindeer populations\"],\n [{\n            'v': 924,\n            'f': \"924\",\n        },\n\"dcc_mozart\",\n\"PRJEB6606\",\n\"Webin-9\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 35,\n            'f': \"35\",\n        },\n\"2014-09-22 10:23:54\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Amplicon sequencing of Tara Oceans DNA samples corresponding to size fractions for large DNA viruses.\"],\n [{\n            'v': 925,\n            'f': \"925\",\n        },\n\"dcc_marc\",\n\"PRJEB7314\",\n\"Webin-2\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2014-12-12 19:13:41\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Sequencing bacterial genomes\"],\n [{\n            'v': 926,\n            'f': \"926\",\n        },\n\"dcc_marc\",\n\"PRJEB8230\",\n\"Webin-39700\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 22,\n            'f': \"22\",\n        },\n\"2015-01-16 19:31:32\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Improved data analysis for the MinION nanopore sequencer\"],\n [{\n            'v': 927,\n            'f': \"927\",\n        },\n\"dcc_marc\",\n\"PRJEB8318\",\n\"Webin-39700\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2015-01-27 06:50:53\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"A reference dataset for phage M13 dsDNA generated with the Oxford Nanopore MinION\"],\n [{\n            'v': 928,\n            'f': \"928\",\n        },\n\"dcc_basie\",\n\"PRJEB84566\",\n\"Webin-47627\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2017-09-05 13:58:20\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Diversity of  Picoeukaryotic community in the Gulf of Gab\\u00e8s (south-eastern Mediterranean)\"],\n [{\n            'v': 929,\n            'f': \"929\",\n        },\n\"dcc_marc\",\n\"PRJEB8656\",\n\"Webin-39852\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2015-03-05 11:15:41\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"MinION poxvirus seq\"],\n [{\n            'v': 930,\n            'f': \"930\",\n        },\n\"dcc_marc\",\n\"PRJEB8672\",\n\"Webin-36089\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8,\n            'f': \"8\",\n        },\n\"2016-10-31 17:02:13\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Experimental Metagenome on MinION\"],\n [{\n            'v': 931,\n            'f': \"931\",\n        },\n\"dcc_marc\",\n\"PRJEB8674\",\n\"Webin-154\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2015-02-27 13:34:48\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Ambystoma Mexicanum\"],\n [{\n            'v': 932,\n            'f': \"932\",\n        },\n\"dcc_marc\",\n\"PRJEB8680\",\n\"Webin-39878\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2015-03-01 15:59:36\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"TS_UNC-2\"],\n [{\n            'v': 933,\n            'f': \"933\",\n        },\n\"dcc_marc\",\n\"PRJEB8681\",\n\"Webin-39851\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2015-03-01 16:09:37\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"A. thaliana (Col-0) Oxford Nanopore Sequencing (MARC)\"],\n [{\n            'v': 934,\n            'f': \"934\",\n        },\n\"dcc_marc\",\n\"PRJEB8715\",\n\"Webin-39878\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2015-03-05 14:07:16\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"TS_CCA-1\"],\n [{\n            'v': 935,\n            'f': \"935\",\n        },\n\"dcc_marc\",\n\"PRJEB8716\",\n\"Webin-36089\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2015-03-05 14:27:55\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Experimental Metagenome on MinION\"],\n [{\n            'v': 936,\n            'f': \"936\",\n        },\n\"dcc_marc\",\n\"PRJEB8730\",\n\"Webin-38425\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2015-06-22 17:39:05\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"MinION full 16S amplicon sequencing from mock microbial community and bacterial communities recovered by modified culture media.\"],\n [{\n            'v': 937,\n            'f': \"937\",\n        },\n\"dcc_faang\",\n\"PRJEB8784\",\n\"Webin-40040\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 59,\n            'f': \"59\",\n        },\n\"2015-07-28 10:36:04\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Porcine Transcriptome and Methylome Map\"],\n [{\n            'v': 938,\n            'f': \"938\",\n        },\n\"dcc_marc\",\n\"PRJEB8790\",\n\"Webin-2\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2015-03-13 09:52:34\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Oxford_Nanopore_sequencing_tests\"],\n [{\n            'v': 939,\n            'f': \"939\",\n        },\n\"dcc_marc\",\n\"PRJEB8828\",\n\"Webin-40157\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 10,\n            'f': \"10\",\n        },\n\"2015-04-16 09:23:15\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Yeast\"],\n [{\n            'v': 940,\n            'f': \"940\",\n        },\n\"dcc_basie\",\n\"PRJEB8832\",\n\"Webin-40224\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2015-03-20 10:35:24\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Microbial diversity on Icelandic glaciers and ice caps\"],\n [{\n            'v': 941,\n            'f': \"941\",\n        },\n\"dcc_marc\",\n\"PRJEB8982\",\n\"Webin-40273\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2015-03-29 21:02:57\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Mitochondrial Genome Acquisition Restores Respiratory Function and Tumorigenic Potential of Cancer Cells without Mitochondrial DNA\"],\n [{\n            'v': 942,\n            'f': \"942\",\n        },\n\"dcc_marc\",\n\"PRJEB9029\",\n\"Webin-38826\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2015-04-08 11:40:59\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Sequencing of LCMV Clone 13 genomic fragment from the late stage of infection\"],\n [{\n            'v': 943,\n            'f': \"943\",\n        },\n\"dcc_marc\",\n\"PRJEB9065\",\n\"Webin-40158\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2015-04-09 12:42:27\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"kj_45\"],\n [{\n            'v': 944,\n            'f': \"944\",\n        },\n\"dcc_marc\",\n\"PRJEB9084\",\n\"Webin-40359\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2015-07-22 12:53:49\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WTCHG_MARC_Ph1.1\"],\n [{\n            'v': 945,\n            'f': \"945\",\n        },\n\"dcc_marc\",\n\"PRJEB9088\",\n\"Webin-39700\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2015-07-22 12:53:36\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"MinION sequencing run for MARC Phase Ia E. coli\"],\n [{\n            'v': 946,\n            'f': \"946\",\n        },\n\"dcc_marc\",\n\"PRJEB9111\",\n\"Webin-40359\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2015-07-22 12:53:52\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WTCHG_MARC_Ph1.2\"],\n [{\n            'v': 947,\n            'f': \"947\",\n        },\n\"dcc_marc\",\n\"PRJEB9131\",\n\"Webin-40352\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2015-07-22 12:54:01\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"MARC phase1a\"],\n [{\n            'v': 948,\n            'f': \"948\",\n        },\n\"dcc_marc\",\n\"PRJEB9135\",\n\"Webin-40157\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2015-07-22 12:53:30\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"MARC phase 1a\"],\n [{\n            'v': 949,\n            'f': \"949\",\n        },\n\"dcc_marc\",\n\"PRJEB9138\",\n\"Webin-40363\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2015-07-22 12:53:44\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"E.Coli MARC1\"],\n [{\n            'v': 950,\n            'f': \"950\",\n        },\n\"dcc_marc\",\n\"PRJEB9289\",\n\"Webin-39700\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2015-07-22 12:54:15\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"MinION Analysis and Reference Consortium Phase Ib sequencing runs\"],\n [{\n            'v': 951,\n            'f': \"951\",\n        },\n\"dcc_marc\",\n\"PRJEB9352\",\n\"Webin-40352\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2015-07-22 12:54:35\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"MARC phase1b\"],\n [{\n            'v': 952,\n            'f': \"952\",\n        },\n\"dcc_marc\",\n\"PRJEB9442\",\n\"Webin-40359\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2015-07-22 12:54:28\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WTCHG_MARC_Ph1b.2\"],\n [{\n            'v': 953,\n            'f': \"953\",\n        },\n\"dcc_marc\",\n\"PRJEB9447\",\n\"Webin-40031\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2015-07-22 12:54:25\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WTCHG_MARC_Ph1b.1\"],\n [{\n            'v': 954,\n            'f': \"954\",\n        },\n\"dcc_marc\",\n\"PRJEB9448\",\n\"Webin-40157\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4,\n            'f': \"4\",\n        },\n\"2015-07-22 12:54:08\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"MARC_1b_042915\"],\n [{\n            'v': 955,\n            'f': \"955\",\n        },\n\"dcc_marc\",\n\"PRJEB9455\",\n\"Webin-40363\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2015-07-22 12:54:19\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Phase 1b Evaluation\"],\n [{\n            'v': 956,\n            'f': \"956\",\n        },\n\"dcc_marc\",\n\"PRJEB9456\",\n\"Webin-40363\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2015-07-22 12:54:22\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Phase 1b Evaluation\"],\n [{\n            'v': 957,\n            'f': \"957\",\n        },\n\"dcc_faang\",\n\"PRJEB9561\",\n\"Webin-40650\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2015-09-16 03:17:29\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Swine DNA methylation study using RRBS : FAANG\"],\n [{\n            'v': 958,\n            'f': \"958\",\n        },\n\"dcc_verdi\",\n\"PRJEB9685\",\n\"Webin-2\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 768,\n            'f': \"768\",\n        },\n\"2015-11-19 13:44:53\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Gastrulation_RNA_Seq\"],\n [{\n            'v': 959,\n            'f': \"959\",\n        },\n\"dcc_sibelius\",\n\"PRJEB9687\",\n\"Webin-40757\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 16,\n            'f': \"16\",\n        },\n\"2018-12-10 13:24:27\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"COMPARE Influenza H5N8 pilot - EMC\"],\n [{\n            'v': 960,\n            'f': \"960\",\n        },\n\"dcc_mozart\",\n\"PRJEB9742\",\n\"Webin-9\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 89,\n            'f': \"89\",\n        },\n\"2018-08-29 05:08:32\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Shotgun Sequencing of Tara Oceans Polar Circle DNA samples corresponding to size fractions for  small DNA viruses.\"],\n [{\n            'v': 961,\n            'f': \"961\",\n        },\n\"dcc_sibelius\",\n\"PRJEB9846\",\n\"Webin-40760\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 17,\n            'f': \"17\",\n        },\n\"2019-01-24 17:25:24\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Comparative analysis of avian influenza H5N8 isolated in Europe in 2014\"],\n [{\n            'v': 962,\n            'f': \"962\",\n        },\n\"dcc_basie\",\n\"PRJNA179200\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2013-11-11 16:04:09\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Partial ribosomal RNA gene of DHAB Ciliates\"],\n [{\n            'v': 963,\n            'f': \"963\",\n        },\n\"dcc_schubert\",\n\"PRJNA183847\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 916,\n            'f': \"916\",\n        },\n\"2023-07-11 01:42:18\",\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2017-07-26 15:38:02\",\n\"GenomeTrakr Project: Arizona State Public Health Laboratory\"],\n [{\n            'v': 964,\n            'f': \"964\",\n        },\n\"dcc_schubert\",\n\"PRJNA183848\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2254,\n            'f': \"2254\",\n        },\n\"2016-09-07 01:21:41\",\n{\n            'v': 7,\n            'f': \"7\",\n        },\n\"2017-07-26 15:37:24\",\n\"GenomeTrakr Project: Florida Department of Health, University of Florida\"],\n [{\n            'v': 965,\n            'f': \"965\",\n        },\n\"dcc_benoit\",\n\"PRJNA183850\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2179,\n            'f': \"2179\",\n        },\n\"2023-06-14 09:34:16\",\n{\n            'v': 5249,\n            'f': \"5249\",\n        },\n\"2019-03-14 23:02:23\",\n\"GenomeTrakr Project: NY State Dept. of Health, Wadsworth Center\"],\n [{\n            'v': 966,\n            'f': \"966\",\n        },\n\"dcc_schubert\",\n\"PRJNA183850\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2179,\n            'f': \"2179\",\n        },\n\"2023-06-14 09:34:16\",\n{\n            'v': 5249,\n            'f': \"5249\",\n        },\n\"2019-03-14 23:02:23\",\n\"GenomeTrakr Project: NY State Dept. of Health, Wadsworth Center\"],\n [{\n            'v': 967,\n            'f': \"967\",\n        },\n\"dcc_schubert\",\n\"PRJNA183851\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1175,\n            'f': \"1175\",\n        },\n\"2023-08-27 01:09:00\",\n{\n            'v': 27,\n            'f': \"27\",\n        },\n\"2017-07-26 15:38:02\",\n\"GenomeTrakr Project: Washington State Department of Health Public Health Laboratory\"],\n [{\n            'v': 968,\n            'f': \"968\",\n        },\n\"dcc_grusin\",\n\"PRJNA189553\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2013-11-11 16:04:33\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Analysis of the host transcriptome from demyelinating spinal cord of murine coronavirus infected mice\"],\n [{\n            'v': 969,\n            'f': \"969\",\n        },\n\"dcc_schubert\",\n\"PRJNA203445\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 4941,\n            'f': \"4941\",\n        },\n\"2023-01-02 03:28:39\",\n{\n            'v': 7,\n            'f': \"7\",\n        },\n\"2017-07-26 15:37:01\",\n\"Major food bacterial pathogens in the united states and around the world, including Salmonella enterica, E. coli, Shigella, Listeria monocytogenes, Campylobacter Jejuni, and Vibrio parahaemolyticus genome sequencing\"],\n [{\n            'v': 970,\n            'f': \"970\",\n        },\n\"dcc_schubert\",\n\"PRJNA219491\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 893,\n            'f': \"893\",\n        },\n\"2023-08-24 02:26:47\",\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2017-07-26 15:37:46\",\n\"GenomeTrakr Project: Virginia - Division of Consolidated Laboratory Services\"],\n [{\n            'v': 971,\n            'f': \"971\",\n        },\n\"dcc_basie\",\n\"PRJNA224766\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 6,\n            'f': \"6\",\n        },\n\"2015-03-24 20:40:19\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Diversity partitioning of protistan and fungal plankton communities in glacierfed turbid and clear high mountain lakes\"],\n [{\n            'v': 972,\n            'f': \"972\",\n        },\n\"dcc_clark\",\n\"PRJNA231221\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3672,\n            'f': \"3672\",\n        },\n\"2022-01-06 02:13:46\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"FDA-ARGOS is a database with public quality-controlled reference genomes for diagnostic use and regulatory science\"],\n [{\n            'v': 973,\n            'f': \"973\",\n        },\n\"dcc_grusin\",\n\"PRJNA233943\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 134,\n            'f': \"134\",\n        },\n\"2014-03-24 11:46:07\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Transcriptomic analysis of the Novel Middle East Respiratory Syndrome Coronavirus (MERS-CoV)\"],\n [{\n            'v': 974,\n            'f': \"974\",\n        },\n\"dcc_grusin\",\n\"PRJNA233944\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 136,\n            'f': \"136\",\n        },\n\"2014-03-24 11:46:12\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Transcriptomic analysis of the Novel Middle East Respiratory Syndrome Coronavirus (MERS-CoV)\"],\n [{\n            'v': 975,\n            'f': \"975\",\n        },\n\"dcc_grusin\",\n\"PRJNA238265\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 43,\n            'f': \"43\",\n        },\n\"2014-07-14 02:28:31\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Transcriptional responses of Middle East respiratory syndrome coronavirus (MERS-CoV) in marmosets\"],\n [{\n            'v': 976,\n            'f': \"976\",\n        },\n\"dcc_schubert\",\n\"PRJNA242614\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 108,\n            'f': \"108\",\n        },\n\"2017-05-18 02:10:38\",\n{\n            'v': 104,\n            'f': \"104\",\n        },\n\"2017-07-26 15:37:20\",\n\"Salmonella enterica MDR Genome sequencing\"],\n [{\n            'v': 977,\n            'f': \"977\",\n        },\n\"dcc_grusin\",\n\"PRJNA253693\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 6,\n            'f': \"6\",\n        },\n\"2015-06-27 01:14:01\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Alphacoronavirus Transcriptome or Gene expression\"],\n [{\n            'v': 978,\n            'f': \"978\",\n        },\n\"dcc_clark\",\n\"PRJNA257008\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 148,\n            'f': \"148\",\n        },\n\"2018-06-02 02:56:16\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Consortium for Microbial Forensics and Genomics (\\u00b5FORGE): Capturing Global Biodiversity of Pathogens\"],\n [{\n            'v': 979,\n            'f': \"979\",\n        },\n\"dcc_clark\",\n\"PRJNA257197\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 891,\n            'f': \"891\",\n        },\n\"2015-06-05 17:42:24\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Zaire ebolavirus Genome sequencing\"],\n [{\n            'v': 980,\n            'f': \"980\",\n        },\n\"dcc_clark\",\n\"PRJNA258131\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2017-11-08 14:39:12\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Ebola virus - Mayinga, Zaire, 1976 Transcriptome or Gene expression\"],\n [{\n            'v': 981,\n            'f': \"981\",\n        },\n\"dcc_basie\",\n\"PRJNA259710\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2015-02-12 07:15:25\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Freshwater Eukaryotes Targeted Locus (Loci)\"],\n [{\n            'v': 982,\n            'f': \"982\",\n        },\n\"dcc_clark\",\n\"PRJNA264121\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 9,\n            'f': \"9\",\n        },\n\"2017-11-08 14:39:12\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Orthomarburgvirus marburgense strain:Angola Transcriptome or Gene expression\"],\n [{\n            'v': 983,\n            'f': \"983\",\n        },\n\"dcc_clark\",\n\"PRJNA264425\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 6,\n            'f': \"6\",\n        },\n\"2015-10-01 01:16:53\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Zaire ebolavirus strain:1995 Genome sequencing\"],\n [{\n            'v': 984,\n            'f': \"984\",\n        },\n\"dcc_schubert\",\n\"PRJNA266657\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 152,\n            'f': \"152\",\n        },\n\"2015-11-03 17:05:57\",\n{\n            'v': 76,\n            'f': \"76\",\n        },\n\"2017-07-26 15:37:24\",\n\"Escherichia coli multidrug-resistant sequencing\"],\n [{\n            'v': 985,\n            'f': \"985\",\n        },\n\"dcc_schubert\",\n\"PRJNA271470\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1134,\n            'f': \"1134\",\n        },\n\"2023-07-21 01:08:41\",\n{\n            'v': 9,\n            'f': \"9\",\n        },\n\"2017-07-26 15:37:25\",\n\"GenomeTrakr Project: Maryland Department of Health and Mental Hygiene\"],\n [{\n            'v': 986,\n            'f': \"986\",\n        },\n\"dcc_schubert\",\n\"PRJNA284276\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1487,\n            'f': \"1487\",\n        },\n\"2023-09-17 04:21:33\",\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2017-07-26 15:38:02\",\n\"GenomeTrakr Project: Texas Department of State Health Services\"],\n [{\n            'v': 987,\n            'f': \"987\",\n        },\n\"dcc_schubert\",\n\"PRJNA292661\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8005,\n            'f': \"8005\",\n        },\n\"2023-10-22 11:28:40\",\n{\n            'v': 1217,\n            'f': \"1217\",\n        },\n\"2017-07-26 15:38:44\",\n\"CVM NARMS Salmonella\"],\n [{\n            'v': 988,\n            'f': \"988\",\n        },\n\"dcc_schubert\",\n\"PRJNA292666\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 11743,\n            'f': \"11743\",\n        },\n\"2023-10-24 06:26:37\",\n{\n            'v': 337,\n            'f': \"337\",\n        },\n\"2017-07-26 15:38:28\",\n\"FSIS NARMS Salmonella\"],\n [{\n            'v': 989,\n            'f': \"989\",\n        },\n\"dcc_schubert\",\n\"PRJNA292901\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 87,\n            'f': \"87\",\n        },\n\"2018-10-07 02:02:02\",\n{\n            'v': 9,\n            'f': \"9\",\n        },\n\"2018-01-26 14:28:09\",\n\"Enterobacteriaceae Carbapenem Breakpoint genome sequencing and assembly\"],\n [{\n            'v': 990,\n            'f': \"990\",\n        },\n\"dcc_schubert\",\n\"PRJNA292902\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 200,\n            'f': \"200\",\n        },\n\"2018-09-21 03:03:16\",\n{\n            'v': 9,\n            'f': \"9\",\n        },\n\"2018-01-26 14:31:12\",\n\"Enterobacteriaceae Carbapenemase Diversity genome sequencing and assembly\"],\n [{\n            'v': 991,\n            'f': \"991\",\n        },\n\"dcc_schubert\",\n\"PRJNA292904\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 219,\n            'f': \"219\",\n        },\n\"2018-09-23 05:28:46\",\n{\n            'v': 13,\n            'f': \"13\",\n        },\n\"2018-01-26 14:33:49\",\n\"Gram Negative Carbapenemase Detection Genome sequencing and assembly\"],\n [{\n            'v': 992,\n            'f': \"992\",\n        },\n\"dcc_basie\",\n\"PRJNA299108\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 19,\n            'f': \"19\",\n        },\n\"2016-01-02 01:17:07\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"high-mountain lakes Metagenome\"],\n [{\n            'v': 993,\n            'f': \"993\",\n        },\n\"dcc_grusin\",\n\"PRJNA302555\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 10,\n            'f': \"10\",\n        },\n\"2015-12-25 15:28:14\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Middle East respiratory syndrome-related coronavirus isolate:HCoV-EMC/2012 Raw sequence reads\"],\n [{\n            'v': 994,\n            'f': \"994\",\n        },\n\"dcc_basie\",\n\"PRJNA305428\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 110,\n            'f': \"110\",\n        },\n\"2016-06-16 01:14:01\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Freshwater Eukaryotes from Ponds and Brooks Targeted Locus (Loci)\"],\n [{\n            'v': 995,\n            'f': \"995\",\n        },\n\"dcc_basie\",\n\"PRJNA311248\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 392,\n            'f': \"392\",\n        },\n\"2020-06-02 02:02:41\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"DNA and RNA based tag sequences raw sequence reads\"],\n [{\n            'v': 996,\n            'f': \"996\",\n        },\n\"dcc_grusin\",\n\"PRJNA313952\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8,\n            'f': \"8\",\n        },\n\"2016-06-01 01:20:32\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Human coronavirus OC43 Raw sequence reads\"],\n [{\n            'v': 997,\n            'f': \"997\",\n        },\n\"dcc_basie\",\n\"PRJNA315104\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2017-03-15 01:18:50\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Wastewater Raw sequence reads\"],\n [{\n            'v': 998,\n            'f': \"998\",\n        },\n\"dcc_grusin\",\n\"PRJNA316178\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 16,\n            'f': \"16\",\n        },\n\"2016-07-06 01:20:54\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Middle East respiratory syndrome coronavirus Genome sequencing : MERS-CoV_KOR-2015\"],\n [{\n            'v': 999,\n            'f': \"999\",\n        },\n\"dcc_grusin\",\n\"PRJNA318729\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2017-04-29 01:41:58\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Transcriptome analysis of swine testis cells following swine transmissible gastroenteritis coronavirus infection\"],\n [{\n            'v': 1000,\n            'f': \"1000\",\n        },\n\"dcc_basie\",\n\"PRJNA326482\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 181,\n            'f': \"181\",\n        },\n\"2019-11-01 07:17:19\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Microbial Metagenome, Metatranscriptome & Phylogenetic Libraries\"],\n [{\n            'v': 1001,\n            'f': \"1001\",\n        },\n\"dcc_basie\",\n\"PRJNA329332\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 106,\n            'f': \"106\",\n        },\n\"2018-01-02 01:32:44\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"microbial mat metagenome Targeted loci environmental\"],\n [{\n            'v': 1002,\n            'f': \"1002\",\n        },\n\"dcc_ammons\",\n\"PRJNA329513\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 27,\n            'f': \"27\",\n        },\n\"2016-05-19 01:10:03\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Metagenomic assembly of Zika virus\"],\n [{\n            'v': 1003,\n            'f': \"1003\",\n        },\n\"dcc_basie\",\n\"PRJNA342537\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2016-09-18 01:06:56\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"uncultured marine eukaryote Raw sequence reads\"],\n [{\n            'v': 1004,\n            'f': \"1004\",\n        },\n\"dcc_basie\",\n\"PRJNA349791\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 24,\n            'f': \"24\",\n        },\n\"2017-05-02 10:10:44\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"global salterns eukaryotic V4 Metagenome\"],\n [{\n            'v': 1005,\n            'f': \"1005\",\n        },\n\"dcc_grusin\",\n\"PRJNA350685\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 23,\n            'f': \"23\",\n        },\n\"2017-03-16 01:20:05\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"ChIP-seq analysis of coronavirus-infected human cells\"],\n [{\n            'v': 1006,\n            'f': \"1006\",\n        },\n\"dcc_basie\",\n\"PRJNA360862\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2017-10-18 01:37:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Efficient retrieval of cercozoan sequences in next-generation environmental diversity surveys\"],\n [{\n            'v': 1007,\n            'f': \"1007\",\n        },\n\"dcc_basie\",\n\"PRJNA362945\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 128,\n            'f': \"128\",\n        },\n\"2018-02-01 03:37:57\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Canada Basin Microbial Eukaryotic Communities\"],\n [{\n            'v': 1008,\n            'f': \"1008\",\n        },\n\"dcc_grusin\",\n\"PRJNA376858\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2017-03-07 01:28:11\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Middle East respiratory syndrome-related coronavirus Raw sequence reads\"],\n [{\n            'v': 1009,\n            'f': \"1009\",\n        },\n\"dcc_basie\",\n\"PRJNA377956\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 55,\n            'f': \"55\",\n        },\n\"2017-12-01 14:36:52\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"CARBOM cruise 2013 flow cytometry sorted phytoplankton\"],\n [{\n            'v': 1010,\n            'f': \"1010\",\n        },\n\"dcc_basie\",\n\"PRJNA382696\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2017-04-16 01:56:03\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Global patterns of cercozoan diversity in drylands\"],\n [{\n            'v': 1011,\n            'f': \"1011\",\n        },\n\"dcc_basie\",\n\"PRJNA383398\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 32,\n            'f': \"32\",\n        },\n\"2017-04-20 01:37:30\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Eukaryotic natural population from Northern Baffin Bay - Raw sequence reads\"],\n [{\n            'v': 1012,\n            'f': \"1012\",\n        },\n\"dcc_basie\",\n\"PRJNA385736\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 27379,\n            'f': \"27379\",\n        },\n\"2022-03-20 05:18:49\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Marine metagenomes Metagenome\"],\n [{\n            'v': 1013,\n            'f': \"1013\",\n        },\n\"dcc_basie\",\n\"PRJNA391338\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 10,\n            'f': \"10\",\n        },\n\"2017-12-01 14:40:53\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SPOT April V4 tag sequences Raw sequence reads\"],\n [{\n            'v': 1014,\n            'f': \"1014\",\n        },\n\"dcc_basie\",\n\"PRJNA391477\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 130,\n            'f': \"130\",\n        },\n\"2017-12-16 01:56:25\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Seasonal HOT cruise vertical profile 18S tag (2014-2015) Raw sequence reads\"],\n [{\n            'v': 1015,\n            'f': \"1015\",\n        },\n\"dcc_basie\",\n\"PRJNA391741\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 23,\n            'f': \"23\",\n        },\n\"2017-12-21 01:30:55\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Deep sea hydrothermal vent from Guaymas Basin Raw sequence reads\"],\n [{\n            'v': 1016,\n            'f': \"1016\",\n        },\n\"dcc_basie\",\n\"PRJNA393049\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 42,\n            'f': \"42\",\n        },\n\"2017-12-16 01:56:30\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"18S tag sequencing - sediment trap material Stn ALOHA Raw sequence reads\"],\n [{\n            'v': 1017,\n            'f': \"1017\",\n        },\n\"dcc_basie\",\n\"PRJNA393172\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 38,\n            'f': \"38\",\n        },\n\"2017-12-16 01:56:31\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"18S tag sequencing - diel sampling stn. ALOHA Raw sequence reads\"],\n [{\n            'v': 1018,\n            'f': \"1018\",\n        },\n\"dcc_basie\",\n\"PRJNA396681\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 36,\n            'f': \"36\",\n        },\n\"2018-09-19 02:51:07\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Soil microbiome\"],\n [{\n            'v': 1019,\n            'f': \"1019\",\n        },\n\"dcc_grusin\",\n\"PRJNA396802\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-02-03 02:08:43\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Viral metagenomic analysis of serotine bats (Neoromicia) identifies a Middle East respiratory syndrome-related coronavirus among various novel mammalian viruses\"],\n [{\n            'v': 1020,\n            'f': \"1020\",\n        },\n\"dcc_grusin\",\n\"PRJNA396817\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-07-04 02:14:08\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Viral metagenomic analysis of serotine bats (Neoromicia) identifies a Middle East respiratory syndrome-related coronavirus among various novel mammalian viruses\"],\n [{\n            'v': 1021,\n            'f': \"1021\",\n        },\n\"dcc_grusin\",\n\"PRJNA400643\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 13,\n            'f': \"13\",\n        },\n\"2018-04-05 01:54:50\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Sequencing data of Swine Acute Diarrhea Syndrome coronavirus screening and challenge experiments\"],\n [{\n            'v': 1022,\n            'f': \"1022\",\n        },\n\"dcc_faang\",\n\"PRJNA414087\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 307,\n            'f': \"307\",\n        },\n\"2022-12-22 02:17:24\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Ovis aries Rambouillet Genome sequencing and assembly\"],\n [{\n            'v': 1023,\n            'f': \"1023\",\n        },\n\"dcc_basie\",\n\"PRJNA414535\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2017-12-01 14:50:09\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Metacommunity analysis of seasonal and spatial variation of Cercozoa in a grassland soil\"],\n [{\n            'v': 1024,\n            'f': \"1024\",\n        },\n\"dcc_faang\",\n\"PRJNA417061\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 100,\n            'f': \"100\",\n        },\n\"2020-11-04 02:23:22\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNA sequencing of Bos indicus testicular samples (adult and fetus) and liver samples (adult) [small RNA]\"],\n [{\n            'v': 1025,\n            'f': \"1025\",\n        },\n\"dcc_faang\",\n\"PRJNA417062\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 75,\n            'f': \"75\",\n        },\n\"2020-11-04 02:23:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"RNA sequencing of Bos indicus testicular samples (adult and fetus) and liver samples (adult) [total RNA]\"],\n [{\n            'v': 1026,\n            'f': \"1026\",\n        },\n\"dcc_basie\",\n\"PRJNA422752\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2017-12-21 01:35:28\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Myxomycetes (Amoebozoa) diversity in global drylands\"],\n [{\n            'v': 1027,\n            'f': \"1027\",\n        },\n\"dcc_basie\",\n\"PRJNA427875\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 44,\n            'f': \"44\",\n        },\n\"2017-12-30 01:33:05\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Euk_V4\"],\n [{\n            'v': 1028,\n            'f': \"1028\",\n        },\n\"dcc_basie\",\n\"PRJNA432581\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 69,\n            'f': \"69\",\n        },\n\"2018-02-03 02:11:50\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"California Current Cruise CCE-P1408 Process Cruise #6\"],\n [{\n            'v': 1029,\n            'f': \"1029\",\n        },\n\"dcc_grusin\",\n\"PRJNA436404\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 15,\n            'f': \"15\",\n        },\n\"2019-01-02 03:15:52\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"WNS fungus increases replication of bat coronavirus\"],\n [{\n            'v': 1030,\n            'f': \"1030\",\n        },\n\"dcc_faang\",\n\"PRJNA436715\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 72,\n            'f': \"72\",\n        },\n\"2019-03-28 02:45:34\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Transcriptome related to pubertal development in Brahman heifers\"],\n [{\n            'v': 1031,\n            'f': \"1031\",\n        },\n\"dcc_grusin\",\n\"PRJNA507154\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 11,\n            'f': \"11\",\n        },\n\"2019-03-21 01:42:21\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Human coronavirus OC43 genome sequencing and assembly in clinical context with amplicon-seq data\"],\n [{\n            'v': 1032,\n            'f': \"1032\",\n        },\n\"dcc_grusin\",\n\"PRJNA509533\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2018-12-26 01:22:59\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Whole genome sequence of Human Coronavirus HKU1 from Thailand\"],\n [{\n            'v': 1033,\n            'f': \"1033\",\n        },\n\"dcc_faang\",\n\"PRJNA531151\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 18,\n            'f': \"18\",\n        },\n\"2019-10-02 04:36:20\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Genome-wide DNA methylation analysis using next generation sequencing to reveal candidate genes responsible for boar taint in pigs\"],\n [{\n            'v': 1034,\n            'f': \"1034\",\n        },\n\"dcc_grusin\",\n\"PRJNA580021\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 27,\n            'f': \"27\",\n        },\n\"2019-12-02 11:22:54\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Transcriptomic Analysis Of circRNAs/miRNAs/mRNAs upon Middle East Respiratory Syndrome Coronavirus (MERS-CoV) infection\"],\n [{\n            'v': 1035,\n            'f': \"1035\",\n        },\n\"dcc_grusin\",\n\"PRJNA601630\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2020-02-05 11:12:20\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Genome sequencing\"],\n [{\n            'v': 1036,\n            'f': \"1036\",\n        },\n\"dcc_grusin\",\n\"PRJNA601736\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2020-01-20 22:35:10\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Metatranscriptomics of two pneumonia cases\"],\n [{\n            'v': 1037,\n            'f': \"1037\",\n        },\n\"dcc_grusin\",\n\"PRJNA603194\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-02-05 11:12:20\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Complete genome of a novel coronavirus associated with severe human respiratory disease in Wuhan, China\"],\n [{\n            'v': 1038,\n            'f': \"1038\",\n        },\n\"dcc_grusin\",\n\"PRJNA605247\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 27,\n            'f': \"27\",\n        },\n\"2020-02-21 15:25:02\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"dsRNA IP of Coronavirus Infected IFNAR-/- BMDMs\"],\n [{\n            'v': 1039,\n            'f': \"1039\",\n        },\n\"dcc_grusin\",\n\"PRJNA605907\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8,\n            'f': \"8\",\n        },\n\"2020-02-24 10:05:49\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"2019 novel coronavirus\"],\n [{\n            'v': 1040,\n            'f': \"1040\",\n        },\n\"dcc_grusin\",\n\"PRJNA605983\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 9,\n            'f': \"9\",\n        },\n\"2020-02-18 10:56:55\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Raw sequence reads\"],\n [{\n            'v': 1041,\n            'f': \"1041\",\n        },\n\"dcc_grusin\",\n\"PRJNA606159\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 9,\n            'f': \"9\",\n        },\n\"2020-02-18 10:56:42\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Discovery of Bat Coronaviruses through Surveillance and Probe Capture-Based Next-Generation Sequencing.\"],\n [{\n            'v': 1042,\n            'f': \"1042\",\n        },\n\"dcc_grusin\",\n\"PRJNA606165\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 3,\n            'f': \"3\",\n        },\n\"2021-12-09 02:42:10\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Bat coronavirus RaTG13 Raw sequence reads\"],\n [{\n            'v': 1043,\n            'f': \"1043\",\n        },\n\"dcc_grusin\",\n\"PRJNA606875\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 7,\n            'f': \"7\",\n        },\n\"2020-03-28 01:35:49\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Pangolin coronavirus Raw sequence reads\"],\n [{\n            'v': 1044,\n            'f': \"1044\",\n        },\n\"dcc_grusin\",\n\"PRJNA607948\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 12,\n            'f': \"12\",\n        },\n\"2020-04-15 02:56:12\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 parallel sequencing by Illumina and Oxford Nanopore Technologies\"],\n [{\n            'v': 1045,\n            'f': \"1045\",\n        },\n\"dcc_grusin\",\n\"PRJNA608224\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 5,\n            'f': \"5\",\n        },\n\"2020-03-24 01:37:42\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Direct RNA sequencing and early evolution of SARS-CoV-2\"],\n [{\n            'v': 1046,\n            'f': \"1046\",\n        },\n\"dcc_grusin\",\n\"PRJNA608242\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 8,\n            'f': \"8\",\n        },\n\"2020-03-23 14:08:56\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Whole genome sequencing of Coronavirus samples from different patients\"],\n [{\n            'v': 1047,\n            'f': \"1047\",\n        },\n\"dcc_grusin\",\n\"PRJNA608651\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-02-26 21:07:02\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Raw reads and sequence alignment of the complete genome of the first case of SARS-CoV-2 in Nepal\"],\n [{\n            'v': 1048,\n            'f': \"1048\",\n        },\n\"dcc_grusin\",\n\"PRJNA608742\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 21,\n            'f': \"21\",\n        },\n\"2020-04-23 01:40:08\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Single-cell landscape of bronchoalveolar immune cells in COVID-19 patients\"],\n [{\n            'v': 1049,\n            'f': \"1049\",\n        },\n\"dcc_grusin\",\n\"PRJNA609134\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 72,\n            'f': \"72\",\n        },\n\"2020-02-29 02:46:50\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"LY6E blocks coronavirus fusion and confers immune control of viral disease\"],\n [{\n            'v': 1050,\n            'f': \"1050\",\n        },\n\"dcc_grusin\",\n\"PRJNA610248\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 162,\n            'f': \"162\",\n        },\n\"2020-04-30 01:54:33\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Raw sequence reads\"],\n [{\n            'v': 1051,\n            'f': \"1051\",\n        },\n\"dcc_grusin\",\n\"PRJNA610428\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1122,\n            'f': \"1122\",\n        },\n\"2022-08-15 06:49:40\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Washington SARS-CoV-2 isolate sequences\"],\n [{\n            'v': 1052,\n            'f': \"1052\",\n        },\n\"dcc_grusin\",\n\"PRJNA611496\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 1,\n            'f': \"1\",\n        },\n\"2020-03-23 14:10:01\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 sequencing\"],\n [{\n            'v': 1053,\n            'f': \"1053\",\n        },\n\"dcc_grusin\",\n\"PRJNA612578\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 50376,\n            'f': \"50376\",\n        },\n\"2023-10-26 05:24:24\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Genome sequencing\"],\n [{\n            'v': 1054,\n            'f': \"1054\",\n        },\n\"dcc_grusin\",\n\"PRJNA612766\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 241,\n            'f': \"241\",\n        },\n\"2020-03-23 14:13:09\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Nanopore targeted sequencing for SARS-CoV-2 and other respiratory viruses\"],\n [{\n            'v': 1055,\n            'f': \"1055\",\n        },\n\"dcc_grusin\",\n\"PRJNA613094\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2020-09-19 01:58:42\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Respiratory Tract Metagenome - SARS-CoV2 genome sequencing from the first patient identified with COVID-19 in Quito, Ecuador.\"],\n [{\n            'v': 1056,\n            'f': \"1056\",\n        },\n\"dcc_grusin\",\n\"PRJNA613297\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 10,\n            'f': \"10\",\n        },\n\"2020-03-23 14:14:53\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"The sequence of human ACE2 is suboptimal for binding the S spike protein of SARS coronavirus 2\"],\n [{\n            'v': 1057,\n            'f': \"1057\",\n        },\n\"dcc_grusin\",\n\"PRJNA613951\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 2,\n            'f': \"2\",\n        },\n\"2020-03-23 14:15:53\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Nasopharyngeal Swab Metatranscriptomics for identification of SARS-Cov-2\"],\n [{\n            'v': 1058,\n            'f': \"1058\",\n        },\n\"dcc_grusin\",\n\"PRJNA613958\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 14866,\n            'f': \"14866\",\n        },\n\"2020-12-04 12:11:04\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) genome sequencing\"],\n [{\n            'v': 1059,\n            'f': \"1059\",\n        },\n\"dcc_grusin\",\n\"PRJNA614504\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 10259,\n            'f': \"10259\",\n        },\n\"2023-06-05 16:08:08\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 Deep Sequencing University of Wisconsin-Madison\"],\n [{\n            'v': 1060,\n            'f': \"1060\",\n        },\n\"dcc_grusin\",\n\"PRJNA614546\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 45,\n            'f': \"45\",\n        },\n\"2020-05-07 11:15:11\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"High sensitivity detection of SARS-CoV-2 using multiplex PCR and a multiplex-PCR-based metagenomic method\"],\n [{\n            'v': 1061,\n            'f': \"1061\",\n        },\n\"dcc_grusin\",\n\"PRJNA614976\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 69,\n            'f': \"69\",\n        },\n\"2020-05-01 01:54:34\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Genome sequencing\"],\n [{\n            'v': 1062,\n            'f': \"1062\",\n        },\n\"dcc_grusin\",\n\"PRJNA614995\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 142844,\n            'f': \"142844\",\n        },\n\"2023-10-24 06:26:33\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"SARS-CoV-2 WGS\"],\n [{\n            'v': 1063,\n            'f': \"1063\",\n        },\n\"dcc_grusin\",\n\"PRJNA615032\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 329,\n            'f': \"329\",\n        },\n\"2020-04-24 01:39:41\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Transcriptional response to SARS-CoV-2 infection\"],\n [{\n            'v': 1064,\n            'f': \"1064\",\n        },\n\"dcc_grusin\",\n\"PRJNA615319\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 6,\n            'f': \"6\",\n        },\n\"2020-04-14 19:54:11\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Deep sequencing of recombinant SARS-CoV-2 (rSARS-CoV-2) and SARS-CoV-2-GFP (rSARS-CoV-2-GFP)\"],\n [{\n            'v': 1065,\n            'f': \"1065\",\n        },\n\"dcc_grusin\",\n\"PRJNA616147\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 7,\n            'f': \"7\",\n        },\n\"2020-04-15 05:26:29\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Severe acute respiratory syndrome coronavirus 2 Raw sequence reads\"],\n [{\n            'v': 1066,\n            'f': \"1066\",\n        },\n\"dcc_grusin\",\n\"PRJNA616446\",\n\"Webin-842\",\nNaN,\n{\n            'v': 4,\n            'f': \"4\",\n        },\n{\n            'v': 10,\n            'f': \"10\",\n        },\n\"2020-04-14 19:56:50\",\n{\n            'v': 0,\n            'f': \"0\",\n        },\nNaN,\n\"Homo sapiens Raw sequence reads\"]],\n        columns: [[\"number\", \"index\"], [\"string\", \"datahub\"], [\"string\", \"prj_id\"], [\"string\", \"prj_webin\"], [\"string\", \"hold_date\"], [\"number\", \"status_id\"], [\"number\", \"run_count\"], [\"string\", \"latest_run\"], [\"number\", \"analysis_count\"], [\"string\", \"latest_analysis\"], [\"string\", \"prj_title\"]],\n        columnOptions: [{\"width\": \"1px\", \"className\": \"index_column\"}],\n        rowsPerPage: 25,\n        helpUrl: \"https://colab.research.google.com/notebooks/data_table.ipynb\",\n        suppressOutputScrolling: true,\n        minimumWidth: undefined,\n      });\n\n      function appendQuickchartButton(parentElement) {\n        let quickchartButtonContainerElement = document.createElement('div');\n        quickchartButtonContainerElement.innerHTML = `\n<div id=\"df-8ab3f660-ec16-47af-bccd-350f21315259\">\n  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-8ab3f660-ec16-47af-bccd-350f21315259')\"\n            title=\"Suggest charts.\"\n            style=\"display:none;\">\n    \n<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n     width=\"24px\">\n    <g>\n        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n    </g>\n</svg>\n  </button>\n  \n<style>\n  .colab-df-quickchart {\n      --bg-color: #E8F0FE;\n      --fill-color: #1967D2;\n      --hover-bg-color: #E2EBFA;\n      --hover-fill-color: #174EA6;\n      --disabled-fill-color: #AAA;\n      --disabled-bg-color: #DDD;\n  }\n\n  [theme=dark] .colab-df-quickchart {\n      --bg-color: #3B4455;\n      --fill-color: #D2E3FC;\n      --hover-bg-color: #434B5C;\n      --hover-fill-color: #FFFFFF;\n      --disabled-bg-color: #3B4455;\n      --disabled-fill-color: #666;\n  }\n\n  .colab-df-quickchart {\n    background-color: var(--bg-color);\n    border: none;\n    border-radius: 50%;\n    cursor: pointer;\n    display: none;\n    fill: var(--fill-color);\n    height: 32px;\n    padding: 0;\n    width: 32px;\n  }\n\n  .colab-df-quickchart:hover {\n    background-color: var(--hover-bg-color);\n    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n    fill: var(--button-hover-fill-color);\n  }\n\n  .colab-df-quickchart-complete:disabled,\n  .colab-df-quickchart-complete:disabled:hover {\n    background-color: var(--disabled-bg-color);\n    fill: var(--disabled-fill-color);\n    box-shadow: none;\n  }\n\n  .colab-df-spinner {\n    border: 2px solid var(--fill-color);\n    border-color: transparent;\n    border-bottom-color: var(--fill-color);\n    animation:\n      spin 1s steps(1) infinite;\n  }\n\n  @keyframes spin {\n    0% {\n      border-color: transparent;\n      border-bottom-color: var(--fill-color);\n      border-left-color: var(--fill-color);\n    }\n    20% {\n      border-color: transparent;\n      border-left-color: var(--fill-color);\n      border-top-color: var(--fill-color);\n    }\n    30% {\n      border-color: transparent;\n      border-left-color: var(--fill-color);\n      border-top-color: var(--fill-color);\n      border-right-color: var(--fill-color);\n    }\n    40% {\n      border-color: transparent;\n      border-right-color: var(--fill-color);\n      border-top-color: var(--fill-color);\n    }\n    60% {\n      border-color: transparent;\n      border-right-color: var(--fill-color);\n    }\n    80% {\n      border-color: transparent;\n      border-right-color: var(--fill-color);\n      border-bottom-color: var(--fill-color);\n    }\n    90% {\n      border-color: transparent;\n      border-bottom-color: var(--fill-color);\n    }\n  }\n</style>\n\n  <script>\n    async function quickchart(key) {\n      const quickchartButtonEl =\n        document.querySelector('#' + key + ' button');\n      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n      quickchartButtonEl.classList.add('colab-df-spinner');\n      try {\n        const charts = await google.colab.kernel.invokeFunction(\n            'suggestCharts', [key], {});\n      } catch (error) {\n        console.error('Error during call to suggestCharts:', error);\n      }\n      quickchartButtonEl.classList.remove('colab-df-spinner');\n      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n    }\n    (() => {\n      let quickchartButtonEl =\n        document.querySelector('#df-8ab3f660-ec16-47af-bccd-350f21315259 button');\n      quickchartButtonEl.style.display =\n        google.colab.kernel.accessAllowed ? 'block' : 'none';\n    })();\n  </script>\n</div>`;\n        parentElement.appendChild(quickchartButtonContainerElement);\n      }\n\n      appendQuickchartButton(table);\n    "
          },
          "metadata": {},
          "execution_count": 14
        }
      ]
    }
  ]
}